export const mockPublications = [
  {
    _id: "32979835|None",
    id: "32979835",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "EBioMedicine; 2020 Sep 23 ; 60 102984. doi:10.1016/j.ebiom.2020.102984",
          year: "2020",
          type: "title",
          authors:
            "Sun J, LaRock DL, Skowronski EA, Kimmey JM, Olson J, Jiang Z, O'Donoghue AJ, Nizet V, LaRock CN, ",
          section: "Title",
        },
        offset: 0,
        text:
          "The Pseudomonas aeruginosa protease LasB directly activates IL-1beta.",
        sentences: [],
        annotations: [
          {
            id: "2",
            infons: { identifier: "287", type: "Species" },
            text: "Pseudomonas aeruginosa",
            locations: [{ offset: 4, length: 22 }],
          },
          {
            id: "3",
            infons: { identifier: "-", type: "Chemical" },
            text: "LasB",
            locations: [{ offset: 36, length: 4 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 70,
        text:
          "BACKGROUND: Pulmonary damage by Pseudomonas aeruginosa during cystic fibrosis lung infection and ventilator-associated pneumonia is mediated both by pathogen virulence factors and host inflammation. Impaired immune function due to tissue damage and inflammation, coupled with pathogen multidrug resistance, complicates the management of these deep-seated infections. Pathological inflammation during infection is driven by interleukin-1beta (IL-1beta), but the molecular processes involved are not fully understood. METHODS: We examined IL-1beta activation in a pulmonary model infection of Pseudomonas aeruginosa and in vitro using genetics, specific inhibitors, recombinant proteins, and targeted reporters of protease activity and IL-1beta bioactivity. FINDINGS: Caspase-family inflammasome proteases canonically regulate maturation of this proinflammatory cytokine, but we report that plasticity in IL-1beta proteolytic activation allows for its direct maturation by the pseudomonal protease LasB. LasB promotes IL-1beta activation, neutrophilic inflammation, and destruction of lung architecture characteristic of severe P. aeruginosa pulmonary infection. INTERPRETATION: Preservation of lung function and effective immune clearance may be enhanced by selectively controlling inflammation. Discovery of this IL-1beta regulatory mechanism provides a distinct target for anti-inflammatory therapeutics, such as matrix metalloprotease inhibitors that inhibit LasB and limit inflammation and pathology during P. aeruginosa pulmonary infections. FUNDING: Full details are provided in the Acknowledgements section.",
        sentences: [],
        annotations: [
          {
            id: "23",
            infons: { identifier: "MESH:D008171", type: "Disease" },
            text: "Pulmonary damage",
            locations: [{ offset: 82, length: 16 }],
          },
          {
            id: "24",
            infons: { identifier: "287", type: "Species" },
            text: "Pseudomonas aeruginosa",
            locations: [{ offset: 102, length: 22 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:D003550", type: "Disease" },
            text: "cystic fibrosis lung infection",
            locations: [{ offset: 132, length: 30 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D011014", type: "Disease" },
            text: "pneumonia",
            locations: [{ offset: 189, length: 9 }],
          },
          {
            id: "27",
            infons: { identifier: "MESH:D007249", type: "Disease" },
            text: "inflammation",
            locations: [{ offset: 255, length: 12 }],
          },
          {
            id: "28",
            infons: { identifier: "MESH:D007249", type: "Disease" },
            text: "inflammation",
            locations: [{ offset: 319, length: 12 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D007239", type: "Disease" },
            text: "infections",
            locations: [{ offset: 425, length: 10 }],
          },
          {
            id: "30",
            infons: { identifier: "MESH:D007249", type: "Disease" },
            text: "inflammation",
            locations: [{ offset: 450, length: 12 }],
          },
          {
            id: "31",
            infons: { identifier: "MESH:D007239", type: "Disease" },
            text: "infection",
            locations: [{ offset: 470, length: 9 }],
          },
          {
            id: "32",
            infons: { identifier: "MESH:D007239", type: "Disease" },
            text: "infection",
            locations: [{ offset: 648, length: 9 }],
          },
          {
            id: "33",
            infons: { identifier: "287", type: "Species" },
            text: "Pseudomonas aeruginosa",
            locations: [{ offset: 661, length: 22 }],
          },
          {
            id: "34",
            infons: { identifier: "-", type: "Chemical" },
            text: "LasB",
            locations: [{ offset: 1066, length: 4 }],
          },
          {
            id: "35",
            infons: { identifier: "-", type: "Chemical" },
            text: "LasB",
            locations: [{ offset: 1072, length: 4 }],
          },
          {
            id: "36",
            infons: { identifier: "MESH:D007249", type: "Disease" },
            text: "neutrophilic inflammation",
            locations: [{ offset: 1107, length: 25 }],
          },
          {
            id: "37",
            infons: { identifier: "MESH:D011552", type: "Disease" },
            text: "aeruginosa pulmonary infection",
            locations: [{ offset: 1199, length: 30 }],
          },
          {
            id: "38",
            infons: { identifier: "MESH:D007249", type: "Disease" },
            text: "inflammation",
            locations: [{ offset: 1351, length: 12 }],
          },
          {
            id: "39",
            infons: { identifier: "-", type: "Chemical" },
            text: "LasB",
            locations: [{ offset: 1531, length: 4 }],
          },
          {
            id: "40",
            infons: { identifier: "MESH:D007249", type: "Disease" },
            text: "inflammation",
            locations: [{ offset: 1546, length: 12 }],
          },
          {
            id: "41",
            infons: { identifier: "MESH:D011552", type: "Disease" },
            text: "aeruginosa pulmonary infections",
            locations: [{ offset: 1583, length: 31 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 32979835,
    pmcid: null,
    created: { $date: 1601341549653 },
    accessions: [
      "disease@MESH:D011014",
      "disease@MESH:D003550",
      "disease@MESH:D008171",
      "disease@MESH:D007249",
      "chemical@-",
      "disease@MESH:D007239",
      "disease@MESH:D011552",
      "species@287",
    ],
    journal: "EBioMedicine",
    year: 2020,
    authors: [
      "Sun J",
      "LaRock DL",
      "Skowronski EA",
      "Kimmey JM",
      "Olson J",
      "Jiang Z",
      "O'Donoghue AJ",
      "Nizet V",
      "LaRock CN",
    ],
  },
  {
    _id: "32973030|None",
    id: "32973030",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Science; 2020 Sep 25 ; 369 (6511) 1644-1649. doi:10.1126/science.abb7709",
          year: "2020",
          type: "title",
          authors:
            "Wendt G, Zhao L, Chen R, Liu C, O'Donoghue AJ, Caffrey CR, Reese ML, Collins JJ, ",
          section: "Title",
        },
        offset: 0,
        text:
          "A single-cell RNA-seq atlas of Schistosoma mansoni identifies a key regulator of blood feeding.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "6183", type: "Species" },
            text: "Schistosoma mansoni",
            locations: [{ offset: 31, length: 19 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 96,
        text:
          "Schistosomiasis is a neglected tropical disease that infects 240 million people. With no vaccines and only one drug available, new therapeutic targets are needed. The causative agents, schistosomes, are intravascular flatworm parasites that feed on blood and lay eggs, resulting in pathology. The function of the parasite's various tissues in successful parasitism are poorly understood, hindering identification of therapeutic targets. Using single-cell RNA sequencing (RNA-seq), we characterize 43,642 cells from the adult schistosome and identify 68 distinct cell populations, including specialized stem cells that maintain the parasite's blood-digesting gut. These stem cells express the gene hnf4, which is required for gut maintenance, blood feeding, and pathology in vivo. Together, these data provide molecular insights into the organ systems of this important pathogen and identify potential therapeutic targets.",
        sentences: [],
        annotations: [
          {
            id: "5",
            infons: { identifier: "9606", type: "Species" },
            text: "people",
            locations: [{ offset: 169, length: 6 }],
          },
          {
            id: "6",
            infons: { identifier: "MESH:D004211", type: "Disease" },
            text: "intravascular flatworm parasites",
            locations: [{ offset: 299, length: 32 }],
          },
          {
            id: "7",
            infons: { identifier: "MESH:D010272", type: "Disease" },
            text: "parasitism",
            locations: [{ offset: 450, length: 10 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 32973030,
    pmcid: null,
    created: { $date: 1601097685488 },
    accessions: [
      "species@6183",
      "species@9606",
      "disease@MESH:D004211",
      "disease@MESH:D010272",
    ],
    journal: "Science",
    year: 2020,
    authors: [
      "Wendt G",
      "Zhao L",
      "Chen R",
      "Liu C",
      "O'Donoghue AJ",
      "Caffrey CR",
      "Reese ML",
      "Collins JJ",
    ],
  },
  {
    _id: "32899982|None",
    id: "32899982",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Molecules; 2020 Sep 06 ; 25 (18) . doi:10.3390/molecules25184071",
          year: "2020",
          type: "title",
          authors:
            "Maffioli E, Jiang Z, Nonnis S, Negri A, Romeo V, Lietz CB, Hook V, Ristagno G, Baselli G, Kistler EB, Aletti F, O'Donoghue AJ, Tedeschi G, ",
          section: "Title",
        },
        offset: 0,
        text:
          "High-Resolution Mass Spectrometry-Based Approaches for the Detection and Quantification of Peptidase Activity in Plasma.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 121,
        text:
          "Proteomic technologies have identified 234 peptidases in plasma but little quantitative information about the proteolytic activity has been uncovered. In this study, the substrate profile of plasma proteases was evaluated using two nano-LC-ESI-MS/MS methods. Multiplex substrate profiling by mass spectrometry (MSP-MS) quantifies plasma protease activity in vitro using a global and unbiased library of synthetic peptide reporter substrates, and shotgun peptidomics quantifies protein degradation products that have been generated in vivo by proteases. The two approaches gave complementary results since they both highlight key peptidase activities in plasma including amino- and carboxypeptidases with different substrate specificity profiles. These assays provide a significant advantage over traditional approaches, such as fluorogenic peptide reporter substrates, because they can detect active plasma proteases in a global and unbiased manner, in comparison to detecting select proteases using specific reporter substrates. We discovered that plasma proteins are cleaved by endoproteases and these peptide products are subsequently degraded by amino- and carboxypeptidases. The exopeptidases are more active and stable in plasma and therefore were found to be the most active proteases in the in vitro assay. The protocols presented here set the groundwork for studies to evaluate changes in plasma proteolytic activity in shock.",
        sentences: [],
        annotations: [],
        relations: [],
      },
    ],
    relations: [],
    pmid: 32899982,
    pmcid: null,
    created: { $date: 1599736343110 },
    accessions: [],
    journal: "Molecules",
    year: 2020,
    authors: [
      "Maffioli E",
      "Jiang Z",
      "Nonnis S",
      "Negri A",
      "Romeo V",
      "Lietz CB",
      "Hook V",
      "Ristagno G",
      "Baselli G",
      "Kistler EB",
      "Aletti F",
      "O'Donoghue AJ",
      "Tedeschi G",
    ],
  },
  {
    _id: "32409675|None",
    id: "32409675",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Sci Rep; 2020 May 14 ; 10 (1) 7941. doi:10.1038/s41598-020-62910-2",
          year: "2020",
          type: "title",
          authors:
            "Adem S, Jain S, Sveiven M, Zhou X, O'Donoghue AJ, Hall DA, ",
          section: "Title",
        },
        offset: 0,
        text:
          "Giant magnetoresistive biosensors for real-time quantitative detection of protease activity.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 93,
        text:
          "Proteases are enzymes that cleave proteins and are crucial to physiological processes such as digestion, blood clotting, and wound healing. Unregulated protease activity is a biomarker of several human diseases. Synthetic peptides that are selectively hydrolyzed by a protease of interest can be used as reporter substrates of unregulated protease activity. We developed an activity-based protease sensor by immobilizing magnetic nanoparticles (MNPs) to the surface of a giant magnetoresistive spin-valve (GMR SV) sensor using peptides. Cleavage of these peptides by a protease releases the magnetic nanoparticles resulting in a time-dependent change in the local magnetic field. Using this approach, we detected a significant release of MNPs after 3.5 minutes incubation using just 4 nM of the cysteine protease, papain. In addition, we show that proteases in healthy human urine do not release the MNPs, however addition of 20 nM of papain to the urine samples resulted in a time-dependent change in magnetoresistance. This study lays the foundation for using GMR SV sensors as a platform for real-time, quantitative detection of protease activity in biological fluids.",
        sentences: [],
        annotations: [
          {
            id: "3",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 289, length: 5 }],
          },
          {
            id: "4",
            infons: {
              identifier: "1508",
              type: "Gene",
              ncbi_homologene: "37550",
            },
            text: "cysteine protease",
            locations: [{ offset: 888, length: 17 }],
          },
          {
            id: "5",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 962, length: 5 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 32409675,
    pmcid: null,
    created: { $date: 1592771113165 },
    accessions: ["gene@1508", "species@9606"],
    journal: "Sci Rep",
    year: 2020,
    authors: [
      "Adem S",
      "Jain S",
      "Sveiven M",
      "Zhou X",
      "O'Donoghue AJ",
      "Hall DA",
    ],
  },
  {
    _id: "32323409|None",
    id: "32323409",
    infons: {},
    passages: [
      {
        infons: {
          journal: "Pest Manag. Sci.; 2020 Apr 22 . doi:10.1002/ps.5866",
          year: "2020",
          type: "title",
          authors:
            "Guo Y, Wang Y, O'Donoghue AJ, Jiang Z, Carballar-Lejaraz\u00fa R, Liang G, Hu X, Wang R, Xu L, Guan X, Zhang F, Wu S, ",
          section: "Title",
        },
        offset: 0,
        text:
          "Engineering of multiple trypsin/chymotrypsin sites in Cry3A to enhance its activity against Monochamus alternatus Hope larvae.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "192382", type: "Species" },
            text: "Monochamus alternatus",
            locations: [{ offset: 92, length: 21 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 127,
        text:
          "BACKGROUND: Bacillus thuringiensis Cry3 toxins exhibit specific toxicity against several coleopteran larvae. However, owing to its low toxicity to M. alternatus, Cry3A toxin is not useful for managing M. alternatus larvae. Here we assessed the proteolytic activation of Cry3Aa toxin in M. alternatus larval midgut and increased its toxicity by molecular modification. RESULTS: Our results indicated that insufficient processing of Cry3Aa protoxin and non-specific enzymatic digestion of Cry3Aa toxin in the midgut of M. alternatus larvae led to low toxicity. The results of transcriptome analysis, enzymatic assay with fluorogenic substrates, and multiplex substrate profiling by mass spectrometry showed that the main digestive enzymes in M. alternatus larval midgut were trypsin-like proteases that preferentially cleaved peptides with arginine and lysine residues. Consequently, trypsin recognition sites were introduced into the domain I of Cry3Aa protoxin in the loop regions between alpha-helix 3 and alpha-helix 4 to facilitate proteolytic activation. Multiple potential trypsin cleavage sites away from the helix sheet and functional regions in Cry3Aa proteins were also mutated to alanine to prevent non-specific enzymatic digestion. Bioassays indicated that a modified Cry3Aa-T toxin (K65A, K70A, K231A, K468A, and K596A) showed a 9.5-fold (LC50 = 12.3 mug/mL) increase in toxicity to M. alternatus larvae when compared to native Cry3Aa toxin. CONCLUSION: This study highlights an effective way to increase the toxicity of Cry3Aa toxin to M. alternatus, which may be suitable for managing the resistance of transgenic plants to other pests, including some of the most important pests in agriculture. This article is protected by copyright. All rights reserved.",
        sentences: [],
        annotations: [
          {
            id: "20",
            infons: { identifier: "1428", type: "Species" },
            text: "Bacillus thuringiensis",
            locations: [{ offset: 139, length: 22 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "toxicity",
            locations: [{ offset: 191, length: 8 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "toxicity",
            locations: [{ offset: 262, length: 8 }],
          },
          {
            id: "23",
            infons: { identifier: "192382", type: "Species" },
            text: "M. alternatus",
            locations: [{ offset: 274, length: 13 }],
          },
          {
            id: "24",
            infons: { identifier: "192382", type: "Species" },
            text: "M. alternatus",
            locations: [{ offset: 328, length: 13 }],
          },
          {
            id: "25",
            infons: { identifier: "192382", type: "Species" },
            text: "M. alternatus",
            locations: [{ offset: 413, length: 13 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "toxicity",
            locations: [{ offset: 459, length: 8 }],
          },
          {
            id: "27",
            infons: { identifier: "192382", type: "Species" },
            text: "M. alternatus",
            locations: [{ offset: 644, length: 13 }],
          },
          {
            id: "28",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "toxicity",
            locations: [{ offset: 676, length: 8 }],
          },
          {
            id: "29",
            infons: { identifier: "192382", type: "Species" },
            text: "M. alternatus",
            locations: [{ offset: 867, length: 13 }],
          },
          {
            id: "30",
            infons: { identifier: "MESH:D010455", type: "Chemical" },
            text: "peptides",
            locations: [{ offset: 951, length: 8 }],
          },
          {
            id: "31",
            infons: { identifier: "MESH:D001120", type: "Chemical" },
            text: "arginine",
            locations: [{ offset: 965, length: 8 }],
          },
          {
            id: "32",
            infons: { identifier: "MESH:D008239", type: "Chemical" },
            text: "lysine",
            locations: [{ offset: 978, length: 6 }],
          },
          {
            id: "33",
            infons: { identifier: "MESH:D000409", type: "Chemical" },
            text: "alanine",
            locations: [{ offset: 1317, length: 7 }],
          },
          {
            id: "34",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "toxicity",
            locations: [{ offset: 1510, length: 8 }],
          },
          {
            id: "35",
            infons: { identifier: "192382", type: "Species" },
            text: "M. alternatus",
            locations: [{ offset: 1522, length: 13 }],
          },
          {
            id: "36",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "toxicity",
            locations: [{ offset: 1648, length: 8 }],
          },
          {
            id: "37",
            infons: { identifier: "192382", type: "Species" },
            text: "M. alternatus",
            locations: [{ offset: 1676, length: 13 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 32323409,
    pmcid: null,
    created: { $date: 1592771113165 },
    accessions: [
      "chemical@MESH:D000409",
      "species@192382",
      "species@1428",
      "chemical@MESH:D008239",
      "chemical@MESH:D001120",
      "chemical@MESH:D010455",
      "disease@MESH:D064420",
    ],
    journal: "Pest Manag. Sci.",
    year: 2020,
    authors: [
      "Guo Y",
      "Wang Y",
      "O'Donoghue AJ",
      "Jiang Z",
      "Carballar-Lejaraz\u00fa R",
      "Liang G",
      "Hu X",
      "Wang R",
      "Xu L",
      "Guan X",
      "Zhang F",
      "Wu S",
    ],
  },
  {
    _id: "32305689|None",
    id: "32305689",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Biochim Biophys Acta Proteins Proteom; 2020 Apr 16 140428. doi:10.1016/j.bbapap.2020.140428",
          year: "2020",
          type: "title",
          authors:
            "Hook V, Yoon M, Mosier C, Ito G, Podvin S, Head BP, Rissman R, O'Donoghue AJ, Hook G, ",
          section: "Title",
        },
        offset: 0,
        text:
          "Cathepsin B in neurodegeneration of Alzheimer's disease, traumatic brain injury, and related brain disorders.",
        sentences: [],
        annotations: [
          {
            id: "4",
            infons: {
              identifier: "1508",
              type: "Gene",
              ncbi_homologene: "37550",
            },
            text: "Cathepsin B",
            locations: [{ offset: 0, length: 11 }],
          },
          {
            id: "5",
            infons: { identifier: "MESH:D000544", type: "Disease" },
            text: "neurodegeneration of Alzheimer's disease",
            locations: [{ offset: 15, length: 40 }],
          },
          {
            id: "6",
            infons: { identifier: "MESH:D000070642", type: "Disease" },
            text: "traumatic brain injury",
            locations: [{ offset: 57, length: 22 }],
          },
          {
            id: "7",
            infons: { identifier: "MESH:D001927", type: "Disease" },
            text: "brain disorders",
            locations: [{ offset: 93, length: 15 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 110,
        text:
          "Investigations of Alzheimer's disease (AD), traumatic brain injury (TBI), and related brain disorders have provided extensive evidence for involvement of cathepsin B, a lysosomal cysteine protease, in mediating the behavioral deficits and neuropathology of these neurodegenerative diseases. This review integrates findings of cathepsin B regulation in clinical biomarker studies, animal model genetic and inhibitor evaluations, structural studies, and lysosomal cell biological mechanisms in AD, TBI, and related brain disorders. The results together indicate the role of cathepsin B in the behavioral deficits and neuropathology of these disorders. Lysosomal leakage occurs in AD and TBI, and related neurodegeneration, which leads to the hypothesis that cathepsin B is redistributed from the lysosome to the cytosol where it initiates cell death and inflammation processes associated with neurodegeneration. These results together implicate cathepsin B as a major contributor to these neuropathological changes and behavioral deficits. These findings support the investigation of cathepsin B as a potential drug target for therapeutic discovery and treatment of AD, TBI, and TBI-related brain disorders.",
        sentences: [],
        annotations: [
          {
            id: "35",
            infons: { identifier: "MESH:D000544", type: "Disease" },
            text: "Alzheimer's disease",
            locations: [{ offset: 128, length: 19 }],
          },
          {
            id: "36",
            infons: { identifier: "MESH:D000544", type: "Disease" },
            text: "AD",
            locations: [{ offset: 149, length: 2 }],
          },
          {
            id: "37",
            infons: { identifier: "MESH:D000070642", type: "Disease" },
            text: "traumatic brain injury",
            locations: [{ offset: 154, length: 22 }],
          },
          {
            id: "38",
            infons: { identifier: "MESH:D000070642", type: "Disease" },
            text: "TBI",
            locations: [{ offset: 178, length: 3 }],
          },
          {
            id: "39",
            infons: { identifier: "MESH:D001927", type: "Disease" },
            text: "brain disorders",
            locations: [{ offset: 196, length: 15 }],
          },
          {
            id: "40",
            infons: {
              identifier: "1508",
              type: "Gene",
              ncbi_homologene: "37550",
            },
            text: "cathepsin B",
            locations: [{ offset: 264, length: 11 }],
          },
          {
            id: "41",
            infons: { identifier: "MESH:D001523", type: "Disease" },
            text: "behavioral deficits",
            locations: [{ offset: 325, length: 19 }],
          },
          {
            id: "42",
            infons: { identifier: "MESH:D019636", type: "Disease" },
            text: "neurodegenerative diseases",
            locations: [{ offset: 373, length: 26 }],
          },
          {
            id: "43",
            infons: {
              identifier: "1508",
              type: "Gene",
              ncbi_homologene: "37550",
            },
            text: "cathepsin B",
            locations: [{ offset: 436, length: 11 }],
          },
          {
            id: "44",
            infons: { identifier: "MESH:D000544", type: "Disease" },
            text: "AD",
            locations: [{ offset: 602, length: 2 }],
          },
          {
            id: "45",
            infons: { identifier: "MESH:D000070642", type: "Disease" },
            text: "TBI",
            locations: [{ offset: 606, length: 3 }],
          },
          {
            id: "46",
            infons: { identifier: "MESH:D001927", type: "Disease" },
            text: "brain disorders",
            locations: [{ offset: 623, length: 15 }],
          },
          {
            id: "47",
            infons: {
              identifier: "1508",
              type: "Gene",
              ncbi_homologene: "37550",
            },
            text: "cathepsin B",
            locations: [{ offset: 682, length: 11 }],
          },
          {
            id: "48",
            infons: { identifier: "MESH:D001523", type: "Disease" },
            text: "behavioral deficits",
            locations: [{ offset: 701, length: 19 }],
          },
          {
            id: "49",
            infons: { identifier: "MESH:D000544", type: "Disease" },
            text: "AD",
            locations: [{ offset: 788, length: 2 }],
          },
          {
            id: "50",
            infons: { identifier: "MESH:D000070642", type: "Disease" },
            text: "TBI",
            locations: [{ offset: 795, length: 3 }],
          },
          {
            id: "51",
            infons: { identifier: "MESH:D019636", type: "Disease" },
            text: "neurodegeneration",
            locations: [{ offset: 812, length: 17 }],
          },
          {
            id: "52",
            infons: {
              identifier: "1508",
              type: "Gene",
              ncbi_homologene: "37550",
            },
            text: "cathepsin B",
            locations: [{ offset: 866, length: 11 }],
          },
          {
            id: "53",
            infons: { identifier: "MESH:D007249", type: "Disease" },
            text: "inflammation",
            locations: [{ offset: 962, length: 12 }],
          },
          {
            id: "54",
            infons: { identifier: "MESH:D019636", type: "Disease" },
            text: "neurodegeneration",
            locations: [{ offset: 1001, length: 17 }],
          },
          {
            id: "55",
            infons: {
              identifier: "1508",
              type: "Gene",
              ncbi_homologene: "37550",
            },
            text: "cathepsin B",
            locations: [{ offset: 1053, length: 11 }],
          },
          {
            id: "56",
            infons: { identifier: "MESH:D001523", type: "Disease" },
            text: "behavioral deficits",
            locations: [{ offset: 1127, length: 19 }],
          },
          {
            id: "57",
            infons: {
              identifier: "1508",
              type: "Gene",
              ncbi_homologene: "37550",
            },
            text: "cathepsin B",
            locations: [{ offset: 1192, length: 11 }],
          },
          {
            id: "58",
            infons: { identifier: "MESH:D000544", type: "Disease" },
            text: "AD",
            locations: [{ offset: 1274, length: 2 }],
          },
          {
            id: "59",
            infons: { identifier: "MESH:D000070642", type: "Disease" },
            text: "TBI",
            locations: [{ offset: 1278, length: 3 }],
          },
          {
            id: "60",
            infons: { identifier: "MESH:D000070642", type: "Disease" },
            text: "TBI",
            locations: [{ offset: 1287, length: 3 }],
          },
          {
            id: "61",
            infons: { identifier: "MESH:D001927", type: "Disease" },
            text: "brain disorders",
            locations: [{ offset: 1299, length: 15 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 32305689,
    pmcid: null,
    created: { $date: 1592771113165 },
    accessions: [
      "disease@MESH:D000544",
      "gene@1508",
      "disease@MESH:D000070642",
      "disease@MESH:D001523",
      "disease@MESH:D019636",
      "disease@MESH:D007249",
      "disease@MESH:D001927",
    ],
    journal: "Biochim Biophys Acta Proteins Proteom",
    year: 2020,
    authors: [
      "Hook V",
      "Yoon M",
      "Mosier C",
      "Ito G",
      "Podvin S",
      "Head BP",
      "Rissman R",
      "O'Donoghue AJ",
      "Hook G",
    ],
  },
  {
    _id: "32295833|None",
    id: "32295833",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Mol. Cell Proteomics; 2020 Apr 15 . doi:10.1074/mcp.RA120.002079",
          year: "2020",
          type: "title",
          authors:
            "Podvin S, Jones A, Liu Q, Aulston B, Ransom LS, Ames J, Shen G, Lietz CB, Jiang Z, O'Donoghue AJ, Winston C, Ikezu T, Rissman R, Yuan S, Hook V, ",
          section: "Title",
        },
        offset: 0,
        text:
          "Dysregulation of Exosome Cargo by Mutant Tau Expressed in Human-Induced Pluripotent Stem Cell (iPSC) Neurons Revealed by Proteomics Analyses.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "9606", type: "Species" },
            text: "Human",
            locations: [{ offset: 58, length: 5 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 142,
        text:
          "Accumulation and propagation of hyperphosphorylated tau (p-tau) is a common neuropathological hallmark associated with neurodegeneration of Alzheimer's disease (AD), frontotemporal dementia and parkinsonism linked to chromosome 17 (FTDP-17), and related tauopathies. Extracellular vesicles, specifically exosomes, have recently been demonstrated to participate in mediating tau propagation in brain. Exosomes produced by human induced pluripotent stem cell (iPSC)-derived neurons expressing mutant Tau (mTau), containing the P301L and V337M Tau mutations of FTDP-17, possess the ability to propagate p-tau pathology after injection into mouse brain.  To gain an understanding of the mTau exosome cargo involved in tau pathogenesis, these pathogenic exosomes were analyzed by proteomics and bioinformatics. The data showed that mTau expression dysregulates the exosome proteome to result in (1) proteins uniquely present only in mTau, and not control exosomes, (2) the absence of proteins in mTau exosomes, uniquely present in control exosomes, and (3) shared proteins which were significantly up-regulated or down-regulated in mTau compared to control exosomes. Notably, mTau exosomes (not control exosomes) contain ANP32A (also known as I1PP2A), an endogenous inhibitor of the PP2A phosphatase which regulates the phosphorylation state of p-tau.  Several of the mTau exosome-specific proteins have been shown to participate in AD mechanisms involving lysosomes, inflammation, secretases, and related processes.  Furthermore, the mTau exosomes lacked a substantial portion of proteins present in control exosomes involved in pathways of localization, vesicle transport, and protein binding functions. The shared proteins present in both mTau and control exosomes represented exosome functions of vesicle-mediated transport, exocytosis, and secretion processes. These data illustrate mTau as a dynamic regulator of the biogenesis of exosomes to result in acquisition, deletion, and up- or down-regulation of protein cargo to result in pathogenic mTau exosomes capable of in vivo propagation of p-tau neuropathology in mouse brain.",
        sentences: [],
        annotations: [
          {
            id: "19",
            infons: { identifier: "MESH:D009422", type: "Disease" },
            text: "neuropathological hallmark",
            locations: [{ offset: 218, length: 26 }],
          },
          {
            id: "20",
            infons: { identifier: "MESH:D000544", type: "Disease" },
            text: "neurodegeneration of Alzheimer's disease",
            locations: [{ offset: 261, length: 40 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D003704", type: "Disease" },
            text: "dementia",
            locations: [{ offset: 323, length: 8 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D010302", type: "Disease" },
            text: "parkinsonism",
            locations: [{ offset: 336, length: 12 }],
          },
          {
            id: "23",
            infons: {
              identifier: "4137",
              type: "Gene",
              ncbi_homologene: "74962",
            },
            text: "FTDP-17",
            locations: [{ offset: 374, length: 7 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D024801", type: "Disease" },
            text: "tauopathies",
            locations: [{ offset: 396, length: 11 }],
          },
          {
            id: "25",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 563, length: 5 }],
          },
          {
            id: "26",
            infons: {
              identifier: "rs63751273",
              type: "Mutation",
              subtype: "ProteinMutation",
              originalIdentifier: "p.P301L;RS#:63751273",
              hgvs: "p.P301L",
              identifiers: ["rs63751273"],
            },
            text: "P301L",
            locations: [{ offset: 667, length: 5 }],
          },
          {
            id: "27",
            infons: {
              identifier: "rs63750570",
              type: "Mutation",
              subtype: "ProteinMutation",
              originalIdentifier: "p.V337M;RS#:63750570",
              hgvs: "p.V337M",
              identifiers: ["rs63750570"],
            },
            text: "V337M",
            locations: [{ offset: 677, length: 5 }],
          },
          {
            id: "28",
            infons: {
              identifier: "4137",
              type: "Gene",
              ncbi_homologene: "74962",
            },
            text: "FTDP-17",
            locations: [{ offset: 700, length: 7 }],
          },
          {
            id: "29",
            infons: { identifier: "10090", type: "Species" },
            text: "mouse",
            locations: [{ offset: 779, length: 5 }],
          },
          {
            id: "30",
            infons: {
              identifier: "8125",
              type: "Gene",
              ncbi_homologene: "133783",
            },
            text: "ANP32A",
            locations: [{ offset: 1358, length: 6 }],
          },
          {
            id: "31",
            infons: {
              identifier: "8125",
              type: "Gene",
              ncbi_homologene: "133783",
            },
            text: "I1PP2A",
            locations: [{ offset: 1380, length: 6 }],
          },
          {
            id: "32",
            infons: {
              identifier: "5524",
              type: "Gene",
              ncbi_homologene: "6149",
            },
            text: "PP2A",
            locations: [{ offset: 1420, length: 4 }],
          },
          {
            id: "33",
            infons: { identifier: "-", type: "Chemical" },
            text: "p-tau",
            locations: [{ offset: 1482, length: 5 }],
          },
          {
            id: "34",
            infons: { identifier: "MESH:D007249", type: "Disease" },
            text: "inflammation",
            locations: [{ offset: 1605, length: 12 }],
          },
          {
            id: "35",
            infons: { identifier: "10090", type: "Species" },
            text: "mouse",
            locations: [{ offset: 2259, length: 5 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 32295833,
    pmcid: null,
    created: { $date: 1592771113165 },
    accessions: [
      "disease@MESH:D024801",
      "disease@MESH:D000544",
      "chemical@-",
      "disease@MESH:D003704",
      "disease@MESH:D007249",
      "mutation@rs63751273",
      "gene@5524",
      "mutation@rs63750570",
      "disease@MESH:D009422",
      "species@9606",
      "gene@4137",
      "disease@MESH:D010302",
      "species@10090",
      "gene@8125",
    ],
    journal: "Mol. Cell Proteomics",
    year: 2020,
    authors: [
      "Podvin S",
      "Jones A",
      "Liu Q",
      "Aulston B",
      "Ransom LS",
      "Ames J",
      "Shen G",
      "Lietz CB",
      "Jiang Z",
      "O'Donoghue AJ",
      "Winston C",
      "Ikezu T",
      "Rissman R",
      "Yuan S",
      "Hook V",
    ],
  },
  {
    _id: "32175287|None",
    id: "32175287",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Front Cell Infect Microbiol; 2020 ; 10 66. doi:10.3389/fcimb.2020.00066",
          year: "2020",
          "article-id_pmc": "PMC7054455",
          type: "title",
          authors:
            "Dvo\u0159\u00e1kov\u00e1 H, Leontovy\u010d R, Mach\u00e1\u010dek T, O'Donoghue AJ, \u0160edo O, Zdr\u00e1hal Z, Craik CS, Caffrey CR, Hor\u00e1k P, Mike\u0161 L, ",
          section: "Title",
        },
        offset: 0,
        text:
          "Isoforms of Cathepsin B1 in Neurotropic Schistosomula of Trichobilharzia regenti Differ in Substrate Preferences and a Highly Expressed Catalytically Inactive Paralog Binds Cystatin.",
        sentences: [],
        annotations: [
          {
            id: "2",
            infons: { identifier: "157069", type: "Species" },
            text: "Trichobilharzia regenti",
            locations: [{ offset: 57, length: 23 }],
          },
          {
            id: "3",
            infons: {
              identifier: "396497",
              type: "Gene",
              ncbi_homologene: "78",
            },
            text: "Cystatin",
            locations: [{ offset: 173, length: 8 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 183,
        text:
          "Schistosomula (the post-infective stages) of the neurotropic schistosome Trichobilharzia regenti possess multiple isoforms of cathepsin B1 peptidase (TrCB1.1-TrCB1.6) with involvement in nutrient digestion. The comparison of substrate preferences of TrCB1.1 and TrCB1.4 showed that TrCB1.4 had a very narrow substrate specificity and after processing it was less effective toward protein substrates when compared to TrCB1.1. Self-processing of both isoforms could be facilitated by sulfated polysaccharides due to a specific binding motif in the pro-sequence. Trans-activation by heterologous enzymes was also successfully employed. Expression profiling revealed a high level of transcription of genes encoding the enzymatically inactive paralogs TrCB1.5 and TrCB1.6. The transcription level of TrCB1.6 was comparable with that of TrCB1.1 and TrCB1.2, the most abundant active isoforms. Recombinant TrCB1.6wt, a wild type paralog with a Cys29-to-Gly substitution in the active site that renders the enzyme inactive, was processed by the active TrCB1 forms and by an asparaginyl endopeptidase. Although TrCB1.6wt lacked hydrolytic activity, endopeptidase, but not dipeptidase, activity could be restored by mutating Gly29 to Cys29. The lack of exopeptidase activity may be due to other mutations, such as His110-to-Asn in the occluding loop and Asp224-to-Gly in the main body of the mature TrCB1.6, which do not occur in the active isoforms TrCB1.1 and TrCB1.4 with exopeptidase activity. The catalytically active enzymes and the inactive TrCB1.6 paralog formed complexes with chicken cystatin, thus supporting experimentally the hypothesis that inactive paralogs could potentially regulate the activity of the active forms or protect them from being inhibited by host inhibitors. The effect on cell viability and nitric oxide production by selected immune cells observed for TrCB1.1 was not confirmed for TrCB1.6. We show here that the active isoforms of TrCB1 have different affinities for peptide substrates thereby facilitating diversity in protein-derived nutrition for the parasite. The inactive paralogs are unexpectedly highly expressed and one of them retains the ability to bind cystatins, likely due to specific mutations in the occluding loop and the enzyme body. This suggests a role in sequestration of inhibitors and protection of active cysteine peptidases.",
        sentences: [],
        annotations: [
          {
            id: "17",
            infons: { identifier: "157069", type: "Species" },
            text: "Trichobilharzia regenti",
            locations: [{ offset: 256, length: 23 }],
          },
          {
            id: "18",
            infons: { identifier: "MESH:D011134", type: "Chemical" },
            text: "polysaccharides",
            locations: [{ offset: 674, length: 15 }],
          },
          {
            id: "19",
            infons: { identifier: "-", type: "Chemical" },
            text: "Cys29",
            locations: [{ offset: 1120, length: 5 }],
          },
          {
            id: "20",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "Gly",
            locations: [{ offset: 1129, length: 3 }],
          },
          {
            id: "21",
            infons: { identifier: "-", type: "Chemical" },
            text: "Gly29",
            locations: [{ offset: 1398, length: 5 }],
          },
          {
            id: "22",
            infons: { identifier: "-", type: "Chemical" },
            text: "Cys29",
            locations: [{ offset: 1407, length: 5 }],
          },
          {
            id: "23",
            infons: { identifier: "-", type: "Chemical" },
            text: "His110",
            locations: [{ offset: 1487, length: 6 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D001216", type: "Chemical" },
            text: "Asn",
            locations: [{ offset: 1497, length: 3 }],
          },
          {
            id: "25",
            infons: { identifier: "-", type: "Chemical" },
            text: "Asp224",
            locations: [{ offset: 1527, length: 6 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "Gly",
            locations: [{ offset: 1537, length: 3 }],
          },
          {
            id: "27",
            infons: { identifier: "9031", type: "Species" },
            text: "chicken",
            locations: [{ offset: 1759, length: 7 }],
          },
          {
            id: "28",
            infons: {
              identifier: "396497",
              type: "Gene",
              ncbi_homologene: "78",
            },
            text: "cystatin",
            locations: [{ offset: 1767, length: 8 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D009569", type: "Chemical" },
            text: "nitric oxide",
            locations: [{ offset: 1996, length: 12 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 32175287,
    pmcid: null,
    created: { $date: 1592771113165 },
    accessions: [
      "chemical@-",
      "species@9031",
      "chemical@MESH:D005998",
      "chemical@MESH:D011134",
      "species@157069",
      "gene@396497",
      "chemical@MESH:D001216",
      "chemical@MESH:D009569",
    ],
    journal: "Front Cell Infect Microbiol",
    year: 2020,
    authors: [
      "Dvo\u0159\u00e1kov\u00e1 H",
      "Leontovy\u010d R",
      "Mach\u00e1\u010dek T",
      "O'Donoghue AJ",
      "\u0160edo O",
      "Zdr\u00e1hal Z",
      "Craik CS",
      "Caffrey CR",
      "Hor\u00e1k P",
      "Mike\u0161 L",
    ],
  },
  {
    _id: "31935054|None",
    id: "31935054",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "ACS Chem Biol. 2020 Mar 20;15(3):751-757. doi: 10.1021/acschembio.9b00992. Epub",
          year: "2020",
          "article-id_pmc": "PMC6984639",
          type: "title",
          authors:
            "Keller L, Canuto KM, Liu C, Suzuki BM, Almaliti J, Sikandar A, Naman CB, Glukhov E, Luo D, Duggan BM, Luesch H, Koehnke J, O'Donoghue AJ, Gerwick WH",
          section: "Title",
        },
        offset: 0,
        text:
          "Tutuilamides A-C: Vinyl-Chloride Containing Cyclodepsipeptides from Marine Cyanobacteria with Potent Elastase Inhibitory Properties.",
        sentences: [],
        annotations: [
          {
            id: "3",
            infons: { identifier: "MESH:D002244", type: "Chemical" },
            text: "Tutuilamides A-C",
            locations: [{ offset: 0, length: 16 }],
          },
          {
            id: "4",
            infons: { identifier: "MESH:D014752", type: "Chemical" },
            text: "Vinyl-Chloride",
            locations: [{ offset: 18, length: 14 }],
          },
          {
            id: "5",
            infons: { identifier: "MESH:D047630", type: "Chemical" },
            text: "Cyclodepsipeptides",
            locations: [{ offset: 44, length: 18 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 133,
        text:
          "Marine cyanobacteria (blue-green algae) have been shown to possess an enormous capacity to produce structurally diverse natural products that exhibit a broad spectrum of potent biological activities, including cytotoxic, antifungal, antiparasitic, antiviral and antibacterial activities. Using mass spectrometry-guided fractionation together with molecular networking, cyanobacterial field collections from American Samoa and Palmyra Atoll yielded three new cyclic peptides, tutuilamides A-C. Their structures were established by spectroscopic techniques including 1D and 2D NMR, HR-MS, and chemical derivatization. Structure elucidation was facilitated by employing advanced NMR techniques including non-uniform sampling in combination with the 1,1-ADEQUATE experiment. These cyclic peptides are characterized by the presence of several unusual residues including 3-amino-6-hydroxy-2-piperidone and 2-amino-2-butenoic acid, together with a novel vinyl chloride-containing residue. Tutuilamides A-C show potent elastase inhibitory activity together with moderate potency in H-460 lung cancer cell cytotoxicity assays. The binding mode to elastase was analyzed by X-ray crystallography revealing a reversible binding mode similar to the natural product lyngbyastatin 7. The presence of an additional hydrogen bond with the amino acid backbone of the flexible side chain of tutuilamide A, compared to lyngbyastatin 7, facilitates its stabilization in the elastase binding pocket and possibly explains its enhanced inhibitory potency.",
        sentences: [],
        annotations: [
          {
            id: "16",
            infons: { identifier: "1211", type: "Species" },
            text: "blue-green algae",
            locations: [{ offset: 155, length: 16 }],
          },
          {
            id: "17",
            infons: { identifier: "MESH:D002244", type: "Chemical" },
            text: "tutuilamides A-C",
            locations: [{ offset: 608, length: 16 }],
          },
          {
            id: "18",
            infons: { identifier: "MESH:C555955", type: "Chemical" },
            text: "3-amino-6-hydroxy-2-piperidone",
            locations: [{ offset: 998, length: 30 }],
          },
          {
            id: "19",
            infons: { identifier: "MESH:C006507", type: "Chemical" },
            text: "2-amino-2-butenoic acid",
            locations: [{ offset: 1033, length: 23 }],
          },
          {
            id: "20",
            infons: { identifier: "MESH:D014752", type: "Chemical" },
            text: "vinyl chloride",
            locations: [{ offset: 1080, length: 14 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D002244", type: "Chemical" },
            text: "Tutuilamides A-C",
            locations: [{ offset: 1115, length: 16 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D008175", type: "Disease" },
            text: "H-460 lung cancer cell cytotoxicity",
            locations: [{ offset: 1207, length: 35 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:C404389", type: "Chemical" },
            text: "lyngbyastatin",
            locations: [{ offset: 1385, length: 13 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D006859", type: "Chemical" },
            text: "hydrogen",
            locations: [{ offset: 1432, length: 8 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:C404389", type: "Chemical" },
            text: "lyngbyastatin",
            locations: [{ offset: 1532, length: 13 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 31935054,
    pmcid: null,
    created: { $date: 1592852772752 },
    accessions: [
      "chemical@MESH:D002244",
      "species@1211",
      "chemical@MESH:D047630",
      "chemical@MESH:C404389",
      "chemical@MESH:C006507",
      "disease@MESH:D008175",
      "chemical@MESH:D014752",
      "chemical@MESH:D006859",
      "chemical@MESH:C555955",
    ],
    journal: "ACS Chem Biol. 2020 Mar 20",
    year: 2020,
    authors: [
      "Keller L",
      "Canuto KM",
      "Liu C",
      "Suzuki BM",
      "Almaliti J",
      "Sikandar A",
      "Naman CB",
      "Glukhov E",
      "Luo D",
      "Duggan BM",
      "Luesch H",
      "Koehnke J",
      "O'Donoghue AJ",
      "Gerwick WH",
    ],
  },
  {
    _id: "31646247|None",
    id: "31646247",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "ACS Omega; 2019 Oct 151 4116) 16999-17008. doi:10.1021/acsomega.9b02491",
          year: "2019",
          "article-id_pmc": "PMC6796237",
          type: "title",
          authors:
            "Meewan I, Zhang X, Roy S, Ballatore C, O'Donoghue AJ, Schooley RT, Abagyan R, ",
          section: "Title",
        },
        offset: 0,
        text:
          "Discovery of New Inhibitors of Hepatitis C Virus NS3/4A Protease and Its D168A Mutant.",
        sentences: [],
        annotations: [
          {
            id: "2",
            infons: { identifier: "MESH:C537847", type: "Disease" },
            text: "Hepatitis C Virus NS3",
            locations: [{ offset: 31, length: 21 }],
          },
          {
            id: "3",
            infons: {
              identifier: "p.D168A",
              type: "Mutation",
              subtype: "ProteinMutation",
            },
            text: "D168A",
            locations: [{ offset: 73, length: 5 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 87,
        text:
          "Hepatitis C virus (HCV) is a human pathogen with high morbidity. The HCV NS3/4A protease is essential for viral replication and is one of the top three drug targets. Several drugs targeting the protease have been developed, but drug-resistant mutant strains emerged. Here, we screened a library and synthesized a novel class of small molecules based on a tryptophan derivative scaffold identified as HCV NS3/4A protease inhibitors that are active against both wild type and mutant form of the protease. Only the compounds with predicted binding poses not affected by the most frequent mutations in the active site were selected for experimental validation. The antiviral activities were evaluated by replicon and enzymatic assays. Twenty-two compounds were found to inhibit HCV with EC50 values ranging between 0.64 and 63 muM with compound 22 being the most active. In protease assays, 22 had a comparable inhibition profile for the common mutant HCV GT1b D168A and the wild-type enzyme. However, in the same assay, the potency of the approved drug, simeprevir, decreased 5.7-fold for the mutant enzyme relative to the wild type. The top three inhibitors were also tested against four human serine proteases and were shown to be specific to the viral protease. The fluorescence-based cell viability assay demonstrated a sufficient therapeutic range for the top three candidates.",
        sentences: [],
        annotations: [
          {
            id: "16",
            infons: { identifier: "11103", type: "Species" },
            text: "Hepatitis C virus",
            locations: [{ offset: 87, length: 17 }],
          },
          {
            id: "17",
            infons: { identifier: "MESH:D006526", type: "Disease" },
            text: "HCV",
            locations: [{ offset: 106, length: 3 }],
          },
          {
            id: "18",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 116, length: 5 }],
          },
          {
            id: "19",
            infons: { identifier: "MESH:D006526", type: "Disease" },
            text: "HCV",
            locations: [{ offset: 156, length: 3 }],
          },
          {
            id: "20",
            infons: { identifier: null, type: "Chemical", db: null },
            text: "tryptophan",
            locations: [{ offset: 442, length: 10 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D006526", type: "Disease" },
            text: "HCV",
            locations: [{ offset: 487, length: 3 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D006526", type: "Disease" },
            text: "HCV",
            locations: [{ offset: 861, length: 3 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:D006526", type: "Disease" },
            text: "HCV",
            locations: [{ offset: 1035, length: 3 }],
          },
          {
            id: "24",
            infons: {
              identifier: "p.D168A",
              type: "Mutation",
              subtype: "ProteinMutation",
            },
            text: "D168A",
            locations: [{ offset: 1044, length: 5 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:C532453", type: "Chemical" },
            text: "simeprevir",
            locations: [{ offset: 1138, length: 10 }],
          },
          {
            id: "26",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1273, length: 5 }],
          },
          {
            id: "27",
            infons: { identifier: "MESH:C047902", type: "Chemical" },
            text: "serine",
            locations: [{ offset: 1279, length: 6 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 31646247,
    pmcid: null,
    created: { $date: 1592771113165 },
    accessions: [
      "chemical@MESH:C532453",
      "chemical@MESH:C047902",
      "mutation@p.D168A",
      "species@11103",
      "species@9606",
      "disease@MESH:C537847",
      "disease@MESH:D006526",
    ],
    journal: "ACS Omega",
    year: 2019,
    authors: [
      "Meewan I",
      "Zhang X",
      "Roy S",
      "Ballatore C",
      "O'Donoghue AJ",
      "Schooley RT",
      "Abagyan R",
    ],
  },
  {
    _id: "31451503|None",
    id: "31451503",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Antimicrob Agents Chemother. 2019 Oct 22;63(11). pii: AAC.00448-19. doi:",
          year: "2019",
          "article-id_pmc": "PMC6811414",
          type: "title",
          authors:
            "O'Donoghue AJ, Bibo-Verdugo B, Miyamoto Y, Wang SC, Yang JZ, Zuill DE, Matsuka S, Jiang Z, Almaliti J, Caffrey CR, Gerwick WH, Eckmann L",
          section: "Title",
        },
        offset: 0,
        text: "20S proteasome as a drug target in Trichomonas vaginalis.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "5722", type: "Species" },
            text: "Trichomonas vaginalis",
            locations: [{ offset: 35, length: 21 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 58,
        text:
          "Trichomoniasis is a sexually transmitted disease with hundreds of millions of annual cases worldwide. Approved treatment options are limited to two related nitro-heterocyclic compounds, yet resistance to these drugs is an increasing concern. New antimicrobials against the causative agent, Trichomonas vaginalis are urgently needed. We show here that clinically approved anti-cancer drugs that inhibit the proteasome, a large protease complex with a critical role in degrading intracellular proteins in eukaryotes, have sub-micromolar activity against the parasite in vitro and on-target activity against the enriched T. vaginalis proteasome in cell-free assays. Proteomic analysis confirmed that the parasite has all seven alpha and seven beta subunits of the eukaryotic proteasome, although they have only modest sequence identities, ranging from 28-52%, relative to the respective human proteasome subunits. A screen of proteasome inhibitors derived from a marine natural product, carmaphycin, revealed one derivative, carmaphycin-17, with greater activity against T. vaginalis than the reference drug metronidazole, the ability to overcome metronidazole resistance, and reduced human cytotoxicity compared to the anti-cancer proteasome inhibitors. The increased selectivity of carmaphycin-17 for T. vaginalis was related to its &gt;5-fold greater potency against the beta1 and beta5 catalytic subunits of the T. vaginalis proteasome relative to the human proteasome subunits. In a murine model of vaginal trichomonad infection, proteasome inhibitors eliminated or significantly reduced parasite burden upon topical treatment without any apparent adverse effects. Together, these findings validate the proteasome of T. vaginalis as a therapeutic target for development of a novel class of trichomonacidal agents.",
        sentences: [],
        annotations: [
          {
            id: "21",
            infons: { identifier: "5722", type: "Species" },
            text: "Trichomonas vaginalis",
            locations: [{ offset: 348, length: 21 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "cancer",
            locations: [{ offset: 434, length: 6 }],
          },
          {
            id: "23",
            infons: { identifier: "5722", type: "Species" },
            text: "T. vaginalis",
            locations: [{ offset: 676, length: 12 }],
          },
          {
            id: "24",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 942, length: 5 }],
          },
          {
            id: "25",
            infons: { identifier: "-", type: "Chemical" },
            text: "carmaphycin",
            locations: [{ offset: 1042, length: 11 }],
          },
          {
            id: "26",
            infons: { identifier: "-", type: "Chemical" },
            text: "carmaphycin-17",
            locations: [{ offset: 1080, length: 14 }],
          },
          {
            id: "27",
            infons: { identifier: "5722", type: "Species" },
            text: "T. vaginalis",
            locations: [{ offset: 1126, length: 12 }],
          },
          {
            id: "28",
            infons: { identifier: "MESH:D008795", type: "Chemical" },
            text: "metronidazole",
            locations: [{ offset: 1163, length: 13 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D008795", type: "Chemical" },
            text: "metronidazole",
            locations: [{ offset: 1202, length: 13 }],
          },
          {
            id: "30",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1240, length: 5 }],
          },
          {
            id: "31",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "cytotoxicity",
            locations: [{ offset: 1246, length: 12 }],
          },
          {
            id: "32",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "cancer",
            locations: [{ offset: 1280, length: 6 }],
          },
          {
            id: "33",
            infons: { identifier: "-", type: "Chemical" },
            text: "carmaphycin-17",
            locations: [{ offset: 1339, length: 14 }],
          },
          {
            id: "34",
            infons: { identifier: "5722", type: "Species" },
            text: "T. vaginalis",
            locations: [{ offset: 1358, length: 12 }],
          },
          {
            id: "35",
            infons: { identifier: "5722", type: "Species" },
            text: "T. vaginalis",
            locations: [{ offset: 1471, length: 12 }],
          },
          {
            id: "36",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1511, length: 5 }],
          },
          {
            id: "37",
            infons: { identifier: "10090", type: "Species" },
            text: "murine",
            locations: [{ offset: 1543, length: 6 }],
          },
          {
            id: "38",
            infons: { identifier: "MESH:D007239", type: "Disease" },
            text: "infection",
            locations: [{ offset: 1579, length: 9 }],
          },
          {
            id: "39",
            infons: { identifier: "5722", type: "Species" },
            text: "T. vaginalis",
            locations: [{ offset: 1777, length: 12 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 31451503,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D007239",
      "disease@MESH:D064420",
      "chemical@-",
      "species@9606",
      "species@10090",
      "chemical@MESH:D008795",
      "species@5722",
      "disease@MESH:D009369",
    ],
    journal: "Antimicrob Agents Chemother. 2019 Oct 22",
    year: 2019,
    authors: [
      "O'Donoghue AJ",
      "Bibo-Verdugo B",
      "Miyamoto Y",
      "Wang SC",
      "Yang JZ",
      "Zuill DE",
      "Matsuka S",
      "Jiang Z",
      "Almaliti J",
      "Caffrey CR",
      "Gerwick WH",
      "Eckmann L",
    ],
  },
  {
    _id: "31362766|None",
    id: "31362766",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Parasit Vectors. 2019 Jul 30;12(1):383. doi: 10.1186/s13071-019-3639-9.",
          year: "2019",
          "article-id_pmc": "PMC6668146",
          type: "title",
          authors:
            "Chaimon S, Limpanont Y, Reamtong O, Ampawong S, Phuphisut O, Chusongsang P, Ruangsittichai J, Boonyuen U, Watthanakulpanich D, O'Donoghue AJ, Caffrey CR, Adisakwattana P",
          section: "Title",
        },
        offset: 0,
        text:
          "Molecular characterization and functional analysis of the Schistosoma mekongi Ca2+-dependent cysteine protease (calpain).",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "38744", type: "Species" },
            text: "Schistosoma mekongi",
            locations: [{ offset: 58, length: 19 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 122,
        text:
          "BACKGROUND: Schistosoma mekongi, which causes schistosomiasis in humans, is an important public health issue in Southeast Asia. Treatment with praziquantel is the primary method of control but emergence of praziquantel resistance requires the development of alternative drugs and vaccines. Calcium-dependent cysteine protease (calpain) is a novel vaccine candidate that has been studied in S. mansoni, S. japonicum, and protozoans including malaria, leishmania and trypanosomes. However, limited information is available on the properties and functions of calpain in other Schistosoma spp., including S. mekongi. In this study, we functionally characterized calpain 1 of S. mekongi (SmeCalp1). RESULTS: Calpain 1 of S. mekongi was obtained from transcriptomic analysis of S. mekongi; it had the highest expression level of all isoforms tested and was predominantly expressed in the adult male. SmeCalp1 cDNA is 2274 bp long and encodes 758 amino acids, with 85% to 90% homology with calpains in other Schistosoma species. Recombinant SmeCalp1 (rSmeCalp1), with a molecular weight of approximately 86.7 kDa, was expressed in bacteria and stimulated a marked antibody response in mice. Native SmeCalp1 was detected in crude worm extract and excretory-secretory product, and it was mainly localized in the tegument of the adult male; less signal was detected in the adult female worm. Thus, SmeCalp1 may play a role in surface membrane synthesis or host-parasite interaction. We assessed the protease activity of rSmeCalp1 and demonstrated that rSmeCalp1 could cleave the calpain substrate N-succinyl-Leu-Leu-Val-Tyr-7-amino-4-methylcoumarin, that was inhibited by calpain inhibitors (MDL28170 and E64c). Additionally, rSmeCalp1 could degrade the biological substrates fibronectin (blood clotting protein) and human complement C3, indicating important roles in the intravascular system and in host immune evasion. CONCLUSIONS: SmeCalp1 is expressed on the tegumental surface of the parasite and can cleave host defense molecules; thus, it might participate in growth, development and survival during the entire life-cycle of S. mekongi. Information on the properties and functions of SmeCalp1 reported herein will be advantageous in the development of effective drugs and vaccines against S. mekongi and other schistosomes.",
        sentences: [],
        annotations: [
          {
            id: "25",
            infons: { identifier: "38744", type: "Species" },
            text: "Schistosoma mekongi",
            locations: [{ offset: 134, length: 19 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D012552", type: "Disease" },
            text: "schistosomiasis",
            locations: [{ offset: 168, length: 15 }],
          },
          {
            id: "27",
            infons: { identifier: "9606", type: "Species" },
            text: "humans",
            locations: [{ offset: 187, length: 6 }],
          },
          {
            id: "28",
            infons: { identifier: "MESH:D011223", type: "Chemical" },
            text: "praziquantel",
            locations: [{ offset: 265, length: 12 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D011223", type: "Chemical" },
            text: "praziquantel",
            locations: [{ offset: 328, length: 12 }],
          },
          {
            id: "30",
            infons: { identifier: "6183", type: "Species" },
            text: "S. mansoni",
            locations: [{ offset: 512, length: 10 }],
          },
          {
            id: "31",
            infons: { identifier: "3897", type: "Species" },
            text: "S. japonicum",
            locations: [{ offset: 524, length: 12 }],
          },
          {
            id: "32",
            infons: { identifier: "MESH:D008288", type: "Disease" },
            text: "malaria",
            locations: [{ offset: 563, length: 7 }],
          },
          {
            id: "33",
            infons: { identifier: "MESH:D007896", type: "Disease" },
            text: "leishmania",
            locations: [{ offset: 572, length: 10 }],
          },
          {
            id: "34",
            infons: { identifier: "6183", type: "Species" },
            text: "Schistosoma",
            locations: [{ offset: 695, length: 11 }],
          },
          {
            id: "35",
            infons: { identifier: "38744", type: "Species" },
            text: "S. mekongi",
            locations: [{ offset: 723, length: 10 }],
          },
          {
            id: "36",
            infons: { identifier: "38744", type: "Species" },
            text: "S. mekongi",
            locations: [{ offset: 793, length: 10 }],
          },
          {
            id: "37",
            infons: { identifier: "38744", type: "Species" },
            text: "S. mekongi",
            locations: [{ offset: 838, length: 10 }],
          },
          {
            id: "38",
            infons: { identifier: "38744", type: "Species" },
            text: "S. mekongi",
            locations: [{ offset: 894, length: 10 }],
          },
          {
            id: "39",
            infons: { identifier: "6183", type: "Species" },
            text: "Schistosoma",
            locations: [{ offset: 1123, length: 11 }],
          },
          {
            id: "40",
            infons: { identifier: "10090", type: "Species" },
            text: "mice",
            locations: [{ offset: 1300, length: 4 }],
          },
          {
            id: "41",
            infons: { identifier: "-", type: "Chemical" },
            text: "N-succinyl-Leu-Leu-Val-Tyr-7-amino-4-methylcoumarin",
            locations: [{ offset: 1709, length: 51 }],
          },
          {
            id: "42",
            infons: { identifier: "MESH:C058076", type: "Chemical" },
            text: "MDL28170",
            locations: [{ offset: 1804, length: 8 }],
          },
          {
            id: "43",
            infons: { identifier: "MESH:C032425", type: "Chemical" },
            text: "E64c",
            locations: [{ offset: 1817, length: 4 }],
          },
          {
            id: "44",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1929, length: 5 }],
          },
          {
            id: "45",
            infons: {
              identifier: "718",
              type: "Gene",
              ncbi_homologene: "68031",
            },
            text: "complement C3",
            locations: [{ offset: 1935, length: 13 }],
          },
          {
            id: "46",
            infons: { identifier: "38744", type: "Species" },
            text: "S. mekongi",
            locations: [{ offset: 2244, length: 10 }],
          },
          {
            id: "47",
            infons: { identifier: "38744", type: "Species" },
            text: "S. mekongi",
            locations: [{ offset: 2408, length: 10 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 31362766,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:D011223",
      "disease@MESH:D008288",
      "disease@MESH:D007896",
      "chemical@-",
      "chemical@MESH:C032425",
      "species@9606",
      "species@3897",
      "species@10090",
      "disease@MESH:D012552",
      "gene@718",
      "chemical@MESH:C058076",
      "species@6183",
      "species@38744",
    ],
    journal: "Parasit Vectors. 2019 Jul 30",
    year: 2019,
    authors: [
      "Chaimon S",
      "Limpanont Y",
      "Reamtong O",
      "Ampawong S",
      "Phuphisut O",
      "Chusongsang P",
      "Ruangsittichai J",
      "Boonyuen U",
      "Watthanakulpanich D",
      "O'Donoghue AJ",
      "Caffrey CR",
      "Adisakwattana P",
    ],
  },
  {
    _id: "31355632|None",
    id: "31355632",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "ACS Infect Dis. 2019 Oct 11;5(10):1802-1812. doi: 10.1021/acsinfecdis.9b00237.",
          year: "2019",
          "article-id_pmc": "PMC6750318",
          type: "title",
          authors:
            "Bibo-Verdugo B, Wang SC, Almaliti J, Ta AP, Jiang Z, Wong DA, Lietz CB, Suzuki BM, El-Sakkary N, Hook V, Salvesen GS, Gerwick WH, Caffrey CR, O'Donoghue AJ",
          section: "Title",
        },
        offset: 0,
        text:
          "The Proteasome as a Drug Target in the Metazoan Pathogen, Schistosoma mansoni.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "6183", type: "Species" },
            text: "Schistosoma mansoni",
            locations: [{ offset: 58, length: 19 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 79,
        text:
          "Proteases are fundamental to successful parasitism, including that of the schistosome flatworm parasite, which causes the disease schistosomiasis in 200 million people worldwide. The proteasome is receiving attention as a potential drug target for treatment of a variety of infectious parasitic diseases, but it has been understudied in the schistosome. Adult Schistosoma mansoni were incubated with 1 muM concentrations of the proteasome inhibitors bortezomib, carfilzomib, and MG132. After 24 h, bortezomib and carfilzomib decreased worm motility by more than 85% and endogenous proteasome activity by &gt;75%, and after 72 h, they increased caspase activity by &gt;4.5-fold. The association between the engagement of the proteasome target and the phenotypic and biochemical effects recorded encouraged the chromatographic enrichment of the S. mansoni proteasome (Sm20S). Activity assays with fluorogenic proteasome substrates revealed that Sm20S contains caspase-type (beta1), trypsin-type (beta2), and chymotrypsin-type (beta5) activities. Sm20S was screened with 11 peptide epoxyketone inhibitors derived from the marine natural product carmaphycin B. Analogue 17 was 27.4-fold less cytotoxic to HepG2 cells than carmaphycin B and showed equal potency for the beta5 subunits of Sm20S, human constitutive proteasome, and human immunoproteasome. However, this analogue was 13.2-fold more potent at targeting Sm20S beta2 than it was at targeting the equivalent subunits of the human enzymes. Furthermore, 1 muM 17 decreased both worm motility and endogenous Sm20S activity by more than 90% after 24 h. We provide direct evidence of the proteasome's importance to schistosome viability and identify a lead for which future studies will aim to improve the potency, selectivity, and safety.",
        sentences: [],
        annotations: [
          {
            id: "19",
            infons: { identifier: "MESH:D010272", type: "Disease" },
            text: "parasitism",
            locations: [{ offset: 119, length: 10 }],
          },
          {
            id: "20",
            infons: { identifier: "MESH:D012552", type: "Disease" },
            text: "schistosomiasis",
            locations: [{ offset: 209, length: 15 }],
          },
          {
            id: "21",
            infons: { identifier: "9606", type: "Species" },
            text: "people",
            locations: [{ offset: 240, length: 6 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D010272", type: "Disease" },
            text: "infectious parasitic diseases",
            locations: [{ offset: 353, length: 29 }],
          },
          {
            id: "23",
            infons: { identifier: "6183", type: "Species" },
            text: "Schistosoma mansoni",
            locations: [{ offset: 439, length: 19 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D000069286", type: "Chemical" },
            text: "bortezomib",
            locations: [{ offset: 577, length: 10 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:C524865", type: "Chemical" },
            text: "carfilzomib",
            locations: [{ offset: 592, length: 11 }],
          },
          {
            id: "26",
            infons: { identifier: "6183", type: "Species" },
            text: "S. mansoni",
            locations: [{ offset: 922, length: 10 }],
          },
          {
            id: "27",
            infons: {
              identifier: "10242",
              type: "Gene",
              ncbi_homologene: "4257",
            },
            text: "beta2",
            locations: [{ offset: 1073, length: 5 }],
          },
          {
            id: "28",
            infons: { identifier: "-", type: "Chemical" },
            text: "epoxyketone",
            locations: [{ offset: 1158, length: 11 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:C000622250", type: "Chemical" },
            text: "carmaphycin B",
            locations: [{ offset: 1221, length: 13 }],
          },
          {
            id: "30",
            infons: { identifier: "MESH:C000622250", type: "Chemical" },
            text: "carmaphycin B",
            locations: [{ offset: 1297, length: 13 }],
          },
          {
            id: "31",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1369, length: 5 }],
          },
          {
            id: "32",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1404, length: 5 }],
          },
          {
            id: "33",
            infons: {
              identifier: "54583",
              type: "Gene",
              ncbi_homologene: "56936",
            },
            text: "Sm20",
            locations: [{ offset: 1490, length: 4 }],
          },
          {
            id: "34",
            infons: {
              identifier: "10242",
              type: "Gene",
              ncbi_homologene: "4257",
            },
            text: "beta2",
            locations: [{ offset: 1496, length: 5 }],
          },
          {
            id: "35",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1558, length: 5 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 31355632,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:C524865",
      "chemical@MESH:C000622250",
      "chemical@-",
      "gene@54583",
      "species@9606",
      "chemical@MESH:D000069286",
      "disease@MESH:D012552",
      "species@6183",
      "disease@MESH:D010272",
      "gene@10242",
    ],
    journal: "ACS Infect Dis. 2019 Oct 11",
    year: 2019,
    authors: [
      "Bibo-Verdugo B",
      "Wang SC",
      "Almaliti J",
      "Ta AP",
      "Jiang Z",
      "Wong DA",
      "Lietz CB",
      "Suzuki BM",
      "El-Sakkary N",
      "Hook V",
      "Salvesen GS",
      "Gerwick WH",
      "Caffrey CR",
      "O'Donoghue AJ",
    ],
  },
  {
    _id: "31256057|None",
    id: "31256057",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Biol Chem. 2019 Nov 26;400(12):1629-1638. doi: 10.1515/hsz-2019-0103.",
          year: "2019",
          "article-id_pmc": "PMC6609046",
          type: "title",
          authors:
            "Ivry SL, Knudsen GM, Caiazza F, Sharib JM, Jaradeh K, Ravalin M, O'Donoghue AJ, Kirkwood KS, Craik CS",
          section: "Title",
        },
        offset: 0,
        text:
          "The lysosomal aminopeptidase tripeptidyl peptidase 1 displays increased activity in malignant pancreatic cysts.",
        sentences: [],
        annotations: [
          {
            id: "3",
            infons: {
              identifier: "10404",
              type: "Gene",
              ncbi_homologene: "9401",
            },
            text: "aminopeptidase",
            locations: [{ offset: 14, length: 14 }],
          },
          {
            id: "4",
            infons: {
              identifier: "1200",
              type: "Gene",
              ncbi_homologene: "335",
            },
            text: "tripeptidyl peptidase 1",
            locations: [{ offset: 29, length: 23 }],
          },
          {
            id: "5",
            infons: { identifier: "MESH:D010181", type: "Disease" },
            text: "pancreatic cysts",
            locations: [{ offset: 94, length: 16 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 112,
        text:
          "Incidental detection of pancreatic cysts has increased dramatically over the last decade, but risk stratification and clinical management remain a challenge. Mucinous cysts are precursor lesions to pancreatic cancer, however, the majority are indolent. Current diagnostics cannot identify mucinous cysts that harbor cancer or reliably differentiate these lesions from nonmucinous cysts, which present minimal risk of malignant progression. We previously determined that activity of two aspartyl proteases was increased in mucinous cysts. Using a global protease activity profiling technology, termed multiplex substrate profiling by mass spectrometry (MSP-MS), we now show that aminopeptidase activity is also elevated in mucinous cysts. The serine aminopeptidase, tripeptidyl peptidase 1 (TPP1), was detected by proteomic analysis of cyst fluid samples and quantitation using targeted MS demonstrated that this protease was significantly more abundant in mucinous cysts. In a cohort of 110 cyst fluid samples, TPP1 activity was increased more than 3-fold in mucinous cysts relative to nonmucinous cysts. Moreover, TPP1 activity is primarily associated with mucinous cysts that harbor high-grade dysplasia or invasive carcinoma. Although only 59% accurate for differentiating these lesions, measurement of TPP1 activity may improve early detection and treatment of high-risk pancreatic cysts when used in conjunction with other promising biomarkers.",
        sentences: [],
        annotations: [
          {
            id: "20",
            infons: { identifier: "MESH:D010181", type: "Disease" },
            text: "pancreatic cysts",
            locations: [{ offset: 136, length: 16 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D002288", type: "Disease" },
            text: "Mucinous cysts",
            locations: [{ offset: 270, length: 14 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D010190", type: "Disease" },
            text: "pancreatic cancer",
            locations: [{ offset: 310, length: 17 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "cancer",
            locations: [{ offset: 428, length: 6 }],
          },
          {
            id: "24",
            infons: {
              identifier: "10404",
              type: "Gene",
              ncbi_homologene: "9401",
            },
            text: "aminopeptidase",
            locations: [{ offset: 790, length: 14 }],
          },
          {
            id: "25",
            infons: {
              identifier: "10404",
              type: "Gene",
              ncbi_homologene: "9401",
            },
            text: "aminopeptidase",
            locations: [{ offset: 861, length: 14 }],
          },
          {
            id: "26",
            infons: {
              identifier: "1200",
              type: "Gene",
              ncbi_homologene: "335",
            },
            text: "tripeptidyl peptidase 1",
            locations: [{ offset: 877, length: 23 }],
          },
          {
            id: "27",
            infons: {
              identifier: "1200",
              type: "Gene",
              ncbi_homologene: "335",
            },
            text: "TPP1",
            locations: [{ offset: 902, length: 4 }],
          },
          {
            id: "28",
            infons: {
              identifier: "1200",
              type: "Gene",
              ncbi_homologene: "335",
            },
            text: "TPP1",
            locations: [{ offset: 1123, length: 4 }],
          },
          {
            id: "29",
            infons: {
              identifier: "1200",
              type: "Gene",
              ncbi_homologene: "335",
            },
            text: "TPP1",
            locations: [{ offset: 1227, length: 4 }],
          },
          {
            id: "30",
            infons: { identifier: "MESH:D004476", type: "Disease" },
            text: "dysplasia",
            locations: [{ offset: 1308, length: 9 }],
          },
          {
            id: "31",
            infons: { identifier: "MESH:D009361", type: "Disease" },
            text: "invasive carcinoma",
            locations: [{ offset: 1321, length: 18 }],
          },
          {
            id: "32",
            infons: {
              identifier: "1200",
              type: "Gene",
              ncbi_homologene: "335",
            },
            text: "TPP1",
            locations: [{ offset: 1418, length: 4 }],
          },
          {
            id: "33",
            infons: { identifier: "MESH:D010181", type: "Disease" },
            text: "pancreatic cysts",
            locations: [{ offset: 1487, length: 16 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 31256057,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D010190",
      "disease@MESH:D010181",
      "gene@1200",
      "gene@10404",
      "disease@MESH:D004476",
      "disease@MESH:D009361",
      "disease@MESH:D002288",
      "disease@MESH:D009369",
    ],
    journal: "Biol Chem. 2019 Nov 26",
    year: 2019,
    authors: [
      "Ivry SL",
      "Knudsen GM",
      "Caiazza F",
      "Sharib JM",
      "Jaradeh K",
      "Ravalin M",
      "O'Donoghue AJ",
      "Kirkwood KS",
      "Craik CS",
    ],
  },
  {
    _id: "30760325|None",
    id: "30760325",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Microbiome. 2019 Feb 13;7(1):23. doi: 10.1186/s40168-019-0636-3.",
          year: "2019",
          "article-id_pmc": "PMC6375204",
          type: "title",
          authors:
            "Quinn RA, Adem S, Mills RH, Comstock W, DeRight Goldasich L, Humphrey G, Aksenov AA, Melnik AV, da Silva R, Ackermann G, Bandeira N, Gonzalez DJ, Conrad D, O'Donoghue AJ, Knight R, Dorrestein PC",
          section: "Title",
        },
        offset: 0,
        text:
          "Neutrophilic proteolysis in the cystic fibrosis lung correlates with a pathogenic microbiome.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "MESH:D003550", type: "Disease" },
            text: "Neutrophilic proteolysis in the cystic fibrosis lung",
            locations: [{ offset: 0, length: 52 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 94,
        text:
          "BACKGROUND: Studies of the cystic fibrosis (CF) lung microbiome have consistently shown that lung function decline is associated with decreased microbial diversity due to the dominance of opportunistic pathogens. However, how this phenomenon is reflected in the metabolites and chemical environment of lung secretions remains poorly understood. METHODS: Here we investigated the microbial and molecular composition of CF sputum samples using 16S rRNA gene amplicon sequencing and untargeted tandem mass spectrometry to determine their interrelationships and associations with clinical measures of disease severity. RESULTS: The CF metabolome was found to exist in two states: one from patients with more severe disease that had higher molecular diversity and more Pseudomonas aeruginosa and the other from patients with better lung function having lower metabolite diversity and fewer pathogenic bacteria. The two molecular states were differentiated by the abundance and diversity of peptides and amino acids. Patients with severe disease and more pathogenic bacteria had higher levels of peptides. Analysis of the carboxyl terminal residues of these peptides indicated that neutrophil elastase and cathepsin G were responsible for their generation, and accordingly, these patients had higher levels of proteolytic activity from these enzymes in their sputum. The CF pathogen Pseudomonas aeruginosa was correlated with the abundance of amino acids and is known to primarily feed on them in the lung. CONCLUSIONS: In cases of severe CF lung disease, proteolysis by host enzymes creates an amino acid-rich environment that P. aeruginosa comes to dominate, which may contribute to the pathogen's persistence by providing its preferred carbon source.",
        sentences: [],
        annotations: [
          {
            id: "16",
            infons: { identifier: "MESH:D003550", type: "Disease" },
            text: "cystic fibrosis",
            locations: [{ offset: 121, length: 15 }],
          },
          {
            id: "17",
            infons: { identifier: "9606", type: "Species" },
            text: "patients",
            locations: [{ offset: 779, length: 8 }],
          },
          {
            id: "18",
            infons: { identifier: "MESH:D056729", type: "Disease" },
            text: "severe disease",
            locations: [{ offset: 798, length: 14 }],
          },
          {
            id: "19",
            infons: { identifier: "287", type: "Species" },
            text: "Pseudomonas aeruginosa",
            locations: [{ offset: 858, length: 22 }],
          },
          {
            id: "20",
            infons: { identifier: "9606", type: "Species" },
            text: "patients",
            locations: [{ offset: 900, length: 8 }],
          },
          {
            id: "21",
            infons: { identifier: "9606", type: "Species" },
            text: "Patients",
            locations: [{ offset: 1105, length: 8 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D056729", type: "Disease" },
            text: "severe disease",
            locations: [{ offset: 1119, length: 14 }],
          },
          {
            id: "23",
            infons: {
              identifier: "1991",
              type: "Gene",
              ncbi_homologene: "20455",
            },
            text: "neutrophil elastase",
            locations: [{ offset: 1270, length: 19 }],
          },
          {
            id: "24",
            infons: {
              identifier: "1511",
              type: "Gene",
              ncbi_homologene: "105646",
            },
            text: "cathepsin G",
            locations: [{ offset: 1294, length: 11 }],
          },
          {
            id: "25",
            infons: { identifier: "9606", type: "Species" },
            text: "patients",
            locations: [{ offset: 1368, length: 8 }],
          },
          {
            id: "26",
            infons: { identifier: "287", type: "Species" },
            text: "Pseudomonas aeruginosa",
            locations: [{ offset: 1471, length: 22 }],
          },
          {
            id: "27",
            infons: { identifier: "MESH:D003550", type: "Disease" },
            text: "CF lung disease",
            locations: [{ offset: 1627, length: 15 }],
          },
          {
            id: "28",
            infons: { identifier: "287", type: "Species" },
            text: "P. aeruginosa",
            locations: [{ offset: 1716, length: 13 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D002244", type: "Chemical" },
            text: "carbon",
            locations: [{ offset: 1827, length: 6 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 30760325,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "gene@1511",
      "chemical@MESH:D002244",
      "disease@MESH:D056729",
      "species@9606",
      "gene@1991",
      "disease@MESH:D003550",
      "species@287",
    ],
    journal: "Microbiome. 2019 Feb 13",
    year: 2019,
    authors: [
      "Quinn RA",
      "Adem S",
      "Mills RH",
      "Comstock W",
      "DeRight Goldasich L",
      "Humphrey G",
      "Aksenov AA",
      "Melnik AV",
      "da Silva R",
      "Ackermann G",
      "Bandeira N",
      "Gonzalez DJ",
      "Conrad D",
      "O'Donoghue AJ",
      "Knight R",
      "Dorrestein PC",
    ],
  },
  {
    _id: "30705125|None",
    id: "30705125",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Mol Cell Proteomics. 2019 May;18(5):968-981. doi: 10.1074/mcp.TIR118.001099. Epub",
          year: "2019",
          "article-id_pmc": "PMC6495252",
          type: "title",
          authors:
            "Lapek JD Jr, Jiang Z, Wozniak JM, Arutyunova E, Wang SC, Lemieux MJ, Gonzalez DJ, O'Donoghue AJ",
          section: "Title",
        },
        offset: 0,
        text:
          "Quantitative Multiplex Substrate Profiling of Peptidases by Mass Spectrometry.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 79,
        text:
          "Proteolysis is an integral component of life and has been implicated in many disease processes. To improve our understanding of peptidase function, it is imperative to develop tools to uncover substrate specificity and cleavage efficiency. Here, we combine the quantitative power of tandem mass tags (TMTs) with an established peptide cleavage assay to yield quantitative Multiplex Substrate Profiling by Mass Spectrometry (qMSP-MS). This assay was validated with papain, a well-characterized cysteine peptidase, to generate cleavage efficiency values for hydrolysis of 275 unique peptide bonds in parallel. To demonstrate the breath of this assay, we show that qMSP-MS can uncover the substrate specificity of minimally characterized intramembrane rhomboid peptidases, as well as define hundreds of proteolytic activities in complex biological samples, including secretions from lung cancer cell lines. Importantly, our qMSP-MS library uses synthetic peptides whose termini are unmodified, allowing us to characterize not only endo- but also exo-peptidase activity. Each cleaved peptide sequence can be ranked by turnover rate, and the amino acid sequence of the best substrates can be used for designing fluorescent reporter substrates. Discovery of peptide substrates that are selectively cleaved by peptidases which are active at the site of disease highlights the potential for qMSP-MS to guide the development of peptidase-activating drugs for cancer and infectious disease.",
        sentences: [],
        annotations: [
          {
            id: "3",
            infons: { identifier: "MESH:D008175", type: "Disease" },
            text: "lung cancer",
            locations: [{ offset: 959, length: 11 }],
          },
          {
            id: "4",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "cancer",
            locations: [{ offset: 1529, length: 6 }],
          },
          {
            id: "5",
            infons: { identifier: "MESH:D003141", type: "Disease" },
            text: "infectious disease",
            locations: [{ offset: 1540, length: 18 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 30705125,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D008175",
      "disease@MESH:D003141",
      "disease@MESH:D009369",
    ],
    journal: "Mol Cell Proteomics. 2019 May",
    year: 2019,
    authors: [
      "Lapek JD Jr",
      "Jiang Z",
      "Wozniak JM",
      "Arutyunova E",
      "Wang SC",
      "Lemieux MJ",
      "Gonzalez DJ",
      "O'Donoghue AJ",
    ],
  },
  {
    _id: "30571119|None",
    id: "30571119",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "J Med Chem. 2019 Jan 24;62(2):480-490. doi: 10.1021/acs.jmedchem.8b01536. Epub",
          year: "2019",
          "article-id_pmc": "PMC6676893",
          type: "title",
          authors:
            "Damalanka VC, Han Z, Karmakar P, O'Donoghue AJ, La Greca F, Kim T, Pant SM, Helander J, Klefstrom J, Craik CS, Janetka JW",
          section: "Title",
        },
        offset: 0,
        text:
          "Discovery of Selective Matriptase and Hepsin Serine Protease Inhibitors: Useful Chemical Tools for Cancer Cell Biology.",
        sentences: [],
        annotations: [
          {
            id: "2",
            infons: {
              identifier: "3249",
              type: "Gene",
              ncbi_homologene: "20498",
            },
            text: "Hepsin",
            locations: [{ offset: 38, length: 6 }],
          },
          {
            id: "3",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "Cancer",
            locations: [{ offset: 99, length: 6 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 120,
        text:
          "Matriptase and hepsin belong to the family of type II transmembrane serine proteases (TTSPs). Increased activity of these and the plasma protease, hepatocyte growth factor activator (HGFA), is associated with unregulated cell signaling and tumor progression through increased MET and RON kinase signaling pathways. These proteases are highly expressed in multiple solid tumors and hematological malignancies. Herein, we detail the synthesis and structure-activity relationships (SAR) of a dipeptide library bearing Arg alpha-ketobenozothiazole (kbt) warheads as novel inhibitors of HGFA, matriptase, and hepsin. We elucidated the substrate specificity for HGFA using positional scanning of substrate combinatorial libraries (PS-SCL), which was used to discover selective inhibitors of matriptase and hepsin. Using these selective inhibitors, we have clarified the specific role of hepsin in maintaining epithelial cell membrane integrity, known to be lost in breast cancer progression. These selective compounds are useful as chemical biology tools and for future drug discovery efforts.",
        sentences: [],
        annotations: [
          {
            id: "21",
            infons: {
              identifier: "3249",
              type: "Gene",
              ncbi_homologene: "20498",
            },
            text: "hepsin",
            locations: [{ offset: 135, length: 6 }],
          },
          {
            id: "22",
            infons: {
              identifier: "3083",
              type: "Gene",
              ncbi_homologene: "1170",
            },
            text: "hepatocyte growth factor activator",
            locations: [{ offset: 267, length: 34 }],
          },
          {
            id: "23",
            infons: {
              identifier: "3083",
              type: "Gene",
              ncbi_homologene: "1170",
            },
            text: "HGFA",
            locations: [{ offset: 303, length: 4 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "tumor",
            locations: [{ offset: 360, length: 5 }],
          },
          {
            id: "25",
            infons: {
              identifier: "4486",
              type: "Gene",
              ncbi_homologene: "1835",
            },
            text: "RON",
            locations: [{ offset: 404, length: 3 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "tumors",
            locations: [{ offset: 490, length: 6 }],
          },
          {
            id: "27",
            infons: { identifier: "MESH:D019337", type: "Disease" },
            text: "hematological malignancies",
            locations: [{ offset: 501, length: 26 }],
          },
          {
            id: "28",
            infons: { identifier: "MESH:D004151", type: "Chemical" },
            text: "dipeptide",
            locations: [{ offset: 609, length: 9 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D001120", type: "Chemical" },
            text: "Arg",
            locations: [{ offset: 635, length: 3 }],
          },
          {
            id: "30",
            infons: { identifier: "-", type: "Chemical" },
            text: "alpha-ketobenozothiazole",
            locations: [{ offset: 639, length: 24 }],
          },
          {
            id: "31",
            infons: { identifier: "-", type: "Chemical" },
            text: "kbt",
            locations: [{ offset: 665, length: 3 }],
          },
          {
            id: "32",
            infons: {
              identifier: "3083",
              type: "Gene",
              ncbi_homologene: "1170",
            },
            text: "HGFA",
            locations: [{ offset: 702, length: 4 }],
          },
          {
            id: "33",
            infons: {
              identifier: "3249",
              type: "Gene",
              ncbi_homologene: "20498",
            },
            text: "hepsin",
            locations: [{ offset: 724, length: 6 }],
          },
          {
            id: "34",
            infons: {
              identifier: "3083",
              type: "Gene",
              ncbi_homologene: "1170",
            },
            text: "HGFA",
            locations: [{ offset: 776, length: 4 }],
          },
          {
            id: "35",
            infons: {
              identifier: "3249",
              type: "Gene",
              ncbi_homologene: "20498",
            },
            text: "hepsin",
            locations: [{ offset: 920, length: 6 }],
          },
          {
            id: "36",
            infons: {
              identifier: "3249",
              type: "Gene",
              ncbi_homologene: "20498",
            },
            text: "hepsin",
            locations: [{ offset: 1001, length: 6 }],
          },
          {
            id: "37",
            infons: { identifier: "MESH:D001943", type: "Disease" },
            text: "breast cancer",
            locations: [{ offset: 1079, length: 13 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 30571119,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:D004151",
      "disease@MESH:D019337",
      "chemical@-",
      "disease@MESH:D001943",
      "gene@3083",
      "gene@4486",
      "chemical@MESH:D001120",
      "gene@3249",
      "disease@MESH:D009369",
    ],
    journal: "J Med Chem. 2019 Jan 24",
    year: 2019,
    authors: [
      "Damalanka VC",
      "Han Z",
      "Karmakar P",
      "O'Donoghue AJ",
      "La Greca F",
      "Kim T",
      "Pant SM",
      "Helander J",
      "Klefstrom J",
      "Craik CS",
      "Janetka JW",
    ],
  },
  {
    _id: "30379807|None",
    id: "30379807",
    infons: {},
    passages: [
      {
        infons: {
          journal: "PLoS Negl Trop Dis. 2018 Oct 31;12(10):e0006923. doi:",
          year: "2018",
          "article-id_pmc": "PMC6231675",
          type: "title",
          authors:
            "Rebello KM, McKerrow JH, Mota EM, O Donoghue AJ, Neves-Ferreira AGC",
          section: "Title",
        },
        offset: 0,
        text:
          "Activity profiling of peptidases in Angiostrongylus costaricensis first-stage larvae and adult worms.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "334426", type: "Species" },
            text: "Angiostrongylus costaricensis",
            locations: [{ offset: 36, length: 29 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 102,
        text:
          "BACKGROUND: Angiostrongylus costaricensis is a relatively uncharacterized nematode that causes abdominal angiostrongyliasis in Latin America, a human parasitic disease. Currently, no effective pharmacological treatment for angiostrongyliasis exists. Peptidases are known to be druggable targets for a variety of diseases and are essential for several biological processes in parasites. Therefore, this study aimed to systematically characterize the peptidase activity of A. costaricensis in different developmental stages of this parasitic nematode. METHODOLOGY/PRINCIPAL FINDINGS: A library of diverse tetradecapeptides was incubated with cellular lysates from adult worms and from first-stage larvae (L1) and cleaved peptide products were identified by mass spectrometry. Lysates were also treated with class specific peptidase inhibitors to determine which enzyme class was responsible for the proteolytic activity. Peptidase activity from the four major mechanistic classes (aspartic, metallo, serine and cysteine) were detected in adult worm lysate, whereas aspartic, metallo and serine-peptidases were found in the larval lysates. In addition, the substrate specificity profile was found to vary at different pH values. CONCLUSIONS/SIGNIFICANCE: The proteolytic activities in adult worm and L1 lysates were characterized using a highly diversified library of peptide substrates and the activity was validated using a selection of fluorescent substrates. Taken together, peptidase signatures for different developmental stages of this parasite has improved our understanding of the disease pathogenesis and may be useful as potential drug targets or vaccine candidates.",
        sentences: [],
        annotations: [
          {
            id: "10",
            infons: { identifier: "334426", type: "Species" },
            text: "Angiostrongylus costaricensis",
            locations: [{ offset: 114, length: 29 }],
          },
          {
            id: "11",
            infons: { identifier: "MESH:C536369", type: "Disease" },
            text: "abdominal angiostrongyliasis",
            locations: [{ offset: 197, length: 28 }],
          },
          {
            id: "12",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 246, length: 5 }],
          },
          {
            id: "13",
            infons: { identifier: "MESH:D010272", type: "Disease" },
            text: "parasitic disease",
            locations: [{ offset: 252, length: 17 }],
          },
          {
            id: "14",
            infons: { identifier: "MESH:C536369", type: "Disease" },
            text: "angiostrongyliasis",
            locations: [{ offset: 325, length: 18 }],
          },
          {
            id: "15",
            infons: { identifier: "334426", type: "Species" },
            text: "A. costaricensis",
            locations: [{ offset: 573, length: 16 }],
          },
          {
            id: "16",
            infons: { identifier: "MESH:D012694", type: "Chemical" },
            text: "serine",
            locations: [{ offset: 1100, length: 6 }],
          },
          {
            id: "17",
            infons: { identifier: "MESH:D003545", type: "Chemical" },
            text: "cysteine",
            locations: [{ offset: 1111, length: 8 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 30379807,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "species@9606",
      "species@334426",
      "chemical@MESH:D003545",
      "chemical@MESH:D012694",
      "disease@MESH:D010272",
      "disease@MESH:C536369",
    ],
    journal: "PLoS Negl Trop Dis. 2018 Oct 31",
    year: 2018,
    authors: [
      "Rebello KM",
      "McKerrow JH",
      "Mota EM",
      "O Donoghue AJ",
      "Neves-Ferreira AGC",
    ],
  },
  {
    _id: "30367778|None",
    id: "30367778",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Biol Chem. 2018 Nov 27;399(12):1375-1388. doi: 10.1515/hsz-2018-0240.",
          year: "2018",
          "article-id_pmc": "PMC6356111",
          type: "title",
          authors:
            "Beekman C, Jiang Z, Suzuki BM, Palmer JM, Lindner DL, O'Donoghue AJ, Knudsen GM, Bennett RJ",
          section: "Title",
        },
        offset: 0,
        text:
          "Characterization of PdCP1, a serine carboxypeptidase from Pseudogymnoascus destructans, the causal agent of White-nose Syndrome.",
        sentences: [],
        annotations: [
          {
            id: "2",
            infons: { identifier: "655981", type: "Species" },
            text: "Pseudogymnoascus destructans",
            locations: [{ offset: 58, length: 28 }],
          },
          {
            id: "3",
            infons: { identifier: "MESH:D056784", type: "Disease" },
            text: "White-nose Syndrome",
            locations: [{ offset: 108, length: 19 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 129,
        text:
          "Pseudogymnoascus destructans is a pathogenic fungus responsible for White-nose Syndrome (WNS), a disease afflicting multiple species of North American bats. Pseudogymnoascus destructans infects susceptible bats during hibernation, invading dermal tissue and causing extensive tissue damage. In contrast, other Pseudogymnoascus species are non-pathogenic and cross-species comparisons may therefore reveal factors that contribute to virulence. In this study, we compared the secretome of P. destructans with that from several closely related Pseudogymnoascus species. A diverse set of hydrolytic enzymes were identified, including a putative serine peptidase, PdCP1, that was unique to the P. destructans secretome. A recombinant form of PdCP1 was purified and substrate preference determined using a multiplexed-substrate profiling method based on enzymatic degradation of a synthetic peptide library and analysis by mass spectrometry. Most peptide substrates were sequentially truncated from the carboxyl-terminus revealing that this enzyme is a bona fide carboxypeptidase. Peptides with arginine located close to the carboxyl-terminus were rapidly cleaved, and a fluorescent substrate containing arginine was therefore used to characterize PdCP1 activity and to screen a selection of peptidase inhibitors. Antipain and leupeptin were found to be the most potent inhibitors of PdCP1 activity.",
        sentences: [],
        annotations: [
          {
            id: "14",
            infons: { identifier: "655981", type: "Species" },
            text: "Pseudogymnoascus destructans",
            locations: [{ offset: 129, length: 28 }],
          },
          {
            id: "15",
            infons: { identifier: "MESH:D056784", type: "Disease" },
            text: "White-nose Syndrome",
            locations: [{ offset: 197, length: 19 }],
          },
          {
            id: "16",
            infons: { type: "Disease", identifier: null, db: null },
            text: "WNS",
            locations: [{ offset: 218, length: 3 }],
          },
          {
            id: "17",
            infons: { identifier: "655981", type: "Species" },
            text: "Pseudogymnoascus destructans",
            locations: [{ offset: 286, length: 28 }],
          },
          {
            id: "18",
            infons: { identifier: "655981", type: "Species" },
            text: "Pseudogymnoascus",
            locations: [{ offset: 439, length: 16 }],
          },
          {
            id: "19",
            infons: { identifier: "655981", type: "Species" },
            text: "P. destructans",
            locations: [{ offset: 616, length: 14 }],
          },
          {
            id: "20",
            infons: { identifier: "655981", type: "Species" },
            text: "Pseudogymnoascus",
            locations: [{ offset: 670, length: 16 }],
          },
          {
            id: "21",
            infons: { identifier: "655981", type: "Species" },
            text: "P. destructans",
            locations: [{ offset: 818, length: 14 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D001120", type: "Chemical" },
            text: "arginine",
            locations: [{ offset: 1218, length: 8 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:D001120", type: "Chemical" },
            text: "arginine",
            locations: [{ offset: 1327, length: 8 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 30367778,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D056784",
      "chemical@MESH:D001120",
      "species@655981",
    ],
    journal: "Biol Chem. 2018 Nov 27",
    year: 2018,
    authors: [
      "Beekman C",
      "Jiang Z",
      "Suzuki BM",
      "Palmer JM",
      "Lindner DL",
      "O'Donoghue AJ",
      "Knudsen GM",
      "Bennett RJ",
    ],
  },
  {
    _id: "30103207|None",
    id: "30103207",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Int J Parasitol Drugs Drug Resist. 2018 Dec;8(3):394-402. doi:",
          year: "2018",
          "article-id_pmc": "PMC6092455",
          type: "title",
          authors:
            "Jalovecka M, Hartmann D, Miyamoto Y, Eckmann L, Hajdusek O, O'Donoghue AJ, Sojka D",
          section: "Title",
        },
        offset: 0,
        text: "Validation of Babesia proteasome as a drug target.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "MESH:D001404", type: "Disease" },
            text: "Babesia",
            locations: [{ offset: 14, length: 7 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 51,
        text:
          "Babesiosis is a tick-transmitted zoonosis caused by apicomplexan parasites of the genus Babesia. Treatment of this emerging malaria-related disease has relied on antimalarial drugs and antibiotics. The proteasome of Plasmodium, the causative agent of malaria, has recently been validated as a target for anti-malarial drug development and therefore, in this study, we investigated the effect of epoxyketone (carfilzomib, ONX-0914 and epoxomicin) and boronic acid (bortezomib and ixazomib) proteasome inhibitors on the growth and survival of Babesia. Testing the compounds against Babesia divergens ex vivo revealed suppressive effects on parasite growth with activity that was higher than the cytotoxic effects on a non-transformed mouse macrophage cell line. Furthermore, we showed that the most-effective compound, carfilzomib, significantly reduces parasite multiplication in a Babesia microti infected mouse model without noticeable adverse effects. In addition, treatment with carfilzomib lead to an ex vivo and in vivo decrease in proteasome activity and accumulation of polyubiquitinated proteins compared to untreated control. Overall, our results demonstrate that the Babesia proteasome is a valid target for drug development and warrants the design of potent and selective B. divergens proteasome inhibitors for the treatment of babesiosis.",
        sentences: [],
        annotations: [
          {
            id: "23",
            infons: { identifier: "MESH:D001404", type: "Disease" },
            text: "Babesiosis",
            locations: [{ offset: 51, length: 10 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D001404", type: "Disease" },
            text: "genus Babesia",
            locations: [{ offset: 133, length: 13 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:D008288", type: "Disease" },
            text: "malaria",
            locations: [{ offset: 175, length: 7 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D008288", type: "Disease" },
            text: "malaria",
            locations: [{ offset: 302, length: 7 }],
          },
          {
            id: "27",
            infons: { identifier: "-", type: "Chemical" },
            text: "epoxyketone",
            locations: [{ offset: 446, length: 11 }],
          },
          {
            id: "28",
            infons: { identifier: "MESH:C524865", type: "Chemical" },
            text: "carfilzomib",
            locations: [{ offset: 459, length: 11 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:C542291", type: "Chemical" },
            text: "ONX-0914",
            locations: [{ offset: 472, length: 8 }],
          },
          {
            id: "30",
            infons: { identifier: "MESH:C078846", type: "Chemical" },
            text: "epoxomicin",
            locations: [{ offset: 485, length: 10 }],
          },
          {
            id: "31",
            infons: { identifier: "MESH:D001897", type: "Chemical" },
            text: "boronic acid",
            locations: [{ offset: 501, length: 12 }],
          },
          {
            id: "32",
            infons: { identifier: "MESH:D000069286", type: "Chemical" },
            text: "bortezomib",
            locations: [{ offset: 515, length: 10 }],
          },
          {
            id: "33",
            infons: { identifier: "MESH:C548400", type: "Chemical" },
            text: "ixazomib",
            locations: [{ offset: 530, length: 8 }],
          },
          {
            id: "34",
            infons: { identifier: "MESH:D001404", type: "Disease" },
            text: "Babesia",
            locations: [{ offset: 592, length: 7 }],
          },
          {
            id: "35",
            infons: { identifier: "32595", type: "Species" },
            text: "Babesia divergens",
            locations: [{ offset: 631, length: 17 }],
          },
          {
            id: "36",
            infons: { identifier: "10090", type: "Species" },
            text: "mouse",
            locations: [{ offset: 783, length: 5 }],
          },
          {
            id: "37",
            infons: { identifier: "MESH:C524865", type: "Chemical" },
            text: "carfilzomib",
            locations: [{ offset: 868, length: 11 }],
          },
          {
            id: "38",
            infons: { identifier: "MESH:D001404", type: "Disease" },
            text: "Babesia microti infected",
            locations: [{ offset: 932, length: 24 }],
          },
          {
            id: "39",
            infons: { identifier: "10090", type: "Species" },
            text: "mouse",
            locations: [{ offset: 957, length: 5 }],
          },
          {
            id: "40",
            infons: { identifier: "MESH:C524865", type: "Chemical" },
            text: "carfilzomib",
            locations: [{ offset: 1033, length: 11 }],
          },
          {
            id: "41",
            infons: { identifier: "MESH:D001404", type: "Disease" },
            text: "Babesia",
            locations: [{ offset: 1228, length: 7 }],
          },
          {
            id: "42",
            infons: { identifier: "32595", type: "Species" },
            text: "B. divergens",
            locations: [{ offset: 1334, length: 12 }],
          },
          {
            id: "43",
            infons: { identifier: "MESH:D001404", type: "Disease" },
            text: "babesiosis",
            locations: [{ offset: 1390, length: 10 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 30103207,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:C524865",
      "species@32595",
      "disease@MESH:D008288",
      "chemical@-",
      "species@10090",
      "chemical@MESH:D000069286",
      "chemical@MESH:D001897",
      "chemical@MESH:C542291",
      "disease@MESH:D001404",
      "chemical@MESH:C548400",
      "chemical@MESH:C078846",
    ],
    journal: "Int J Parasitol Drugs Drug Resist. 2018 Dec",
    year: 2018,
    authors: [
      "Jalovecka M",
      "Hartmann D",
      "Miyamoto Y",
      "Eckmann L",
      "Hajdusek O",
      "O'Donoghue AJ",
      "Sojka D",
    ],
  },
  {
    _id: "30085657|None",
    id: "30085657",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "ACS Chem Biol. 2018 Sep 21;13(9):2513-2521. doi: 10.1021/acschembio.8b00420. Epub",
          year: "2018",
          "article-id_pmc": "PMC6156775",
          type: "title",
          authors:
            "Xu JH, Jiang Z, Solania A, Chatterjee S, Suzuki B, Lietz CB, Hook VYH, O'Donoghue AJ, Wolan DW",
          section: "Title",
        },
        offset: 0,
        text:
          "A Commensal Dipeptidyl Aminopeptidase with Specificity for N-Terminal Glycine Degrades Human-Produced Antimicrobial Peptides in Vitro.",
        sentences: [],
        annotations: [
          {
            id: "2",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "Glycine",
            locations: [{ offset: 70, length: 7 }],
          },
          {
            id: "3",
            infons: { identifier: "9606", type: "Species" },
            text: "Human",
            locations: [{ offset: 87, length: 5 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 135,
        text:
          "Proteases within the C1B hydrolase family are encoded by many organisms. We subjected a putative C1B-like cysteine protease secreted by the human gut commensal Parabacteroides distasonis to mass spectrometry-based substrate profiling to find preferred peptide substrates. The P. distasonis protease, which we termed Pd_dinase, has a sequential diaminopeptidase activity with strong specificity for N-terminal glycine residues. Using the substrate sequence information, we verified the importance of the P2 glycine residue with a panel of fluorogenic substrates and calculated kcat and KM for the dipeptide glycine-arginine-AMC. A potent and irreversible dipeptide inhibitor with a C-terminal acyloxymethyl ketone warhead, glycine-arginine- AOMK, was then synthesized and demonstrated that the Pd_dinase active site requires a free N-terminal amine for potent and rapid inhibition. We next determined the homohexameric Pd_dinase structure in complex with glycine-arginine- AOMK and uncovered unexpected active site features that govern the strict substrate preferences and differentiate this protease from members of the C1B and broader papain-like C1 protease families. We finally showed that Pd_dinase hydrolyzes several human antimicrobial peptides and therefore posit that this P. distasonis enzyme may be secreted into the extracellular milieu to assist in gut colonization by inactivation of host antimicrobial peptides.",
        sentences: [],
        annotations: [
          {
            id: "19",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 275, length: 5 }],
          },
          {
            id: "20",
            infons: { identifier: "823", type: "Species" },
            text: "Parabacteroides distasonis",
            locations: [{ offset: 295, length: 26 }],
          },
          {
            id: "21",
            infons: { identifier: "823", type: "Species" },
            text: "P. distasonis",
            locations: [{ offset: 411, length: 13 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "glycine",
            locations: [{ offset: 544, length: 7 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "glycine",
            locations: [{ offset: 641, length: 7 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D004151", type: "Chemical" },
            text: "dipeptide",
            locations: [{ offset: 731, length: 9 }],
          },
          {
            id: "25",
            infons: { identifier: "-", type: "Chemical" },
            text: "glycine-arginine-AMC",
            locations: [{ offset: 741, length: 20 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D004151", type: "Chemical" },
            text: "dipeptide",
            locations: [{ offset: 789, length: 9 }],
          },
          {
            id: "27",
            infons: { identifier: "-", type: "Chemical" },
            text: "acyloxymethyl ketone",
            locations: [{ offset: 827, length: 20 }],
          },
          {
            id: "28",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "glycine",
            locations: [{ offset: 857, length: 7 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D000588", type: "Chemical" },
            text: "amine",
            locations: [{ offset: 977, length: 5 }],
          },
          {
            id: "30",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "glycine",
            locations: [{ offset: 1089, length: 7 }],
          },
          {
            id: "31",
            infons: { identifier: "MESH:D001120", type: "Chemical" },
            text: "arginine",
            locations: [{ offset: 1097, length: 8 }],
          },
          {
            id: "32",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1357, length: 5 }],
          },
          {
            id: "33",
            infons: { identifier: "823", type: "Species" },
            text: "P. distasonis",
            locations: [{ offset: 1416, length: 13 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 30085657,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:D004151",
      "chemical@MESH:D000588",
      "chemical@-",
      "chemical@MESH:D005998",
      "species@9606",
      "species@823",
      "chemical@MESH:D001120",
    ],
    journal: "ACS Chem Biol. 2018 Sep 21",
    year: 2018,
    authors: [
      "Xu JH",
      "Jiang Z",
      "Solania A",
      "Chatterjee S",
      "Suzuki B",
      "Lietz CB",
      "Hook VYH",
      "O'Donoghue AJ",
      "Wolan DW",
    ],
  },
  {
    _id: "30044760|None",
    id: "30044760",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Biol Chem. 2018 Nov 27;399(12):1389-1397. doi: 10.1515/hsz-2018-0255.",
          year: "2018",
          type: "title",
          authors:
            "Arutyunova E, Jiang Z, Yang J, Kulepa AN, Young HS, Verhelst S, O'Donoghue AJ, Lemieux MJ",
          section: "Title",
        },
        offset: 0,
        text:
          "An internally quenched peptide as a new model substrate for rhomboid intramembrane proteases.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 94,
        text:
          "Rhomboids are ubiquitous intramembrane serine proteases that cleave transmembrane substrates. Their functions include growth factor signaling, mitochondrial homeostasis, and parasite invasion. A recent study revealed that the Escherichia coli rhomboid protease EcGlpG is essential for its extraintestinal pathogenic colonization within the gut. Crystal structures of EcGlpG and the Haemophilus influenzae rhomboid protease HiGlpG have deciphered an active site that is buried within the lipid bilayer but exposed to the aqueous environment via a cavity at the periplasmic face. A lack of physiological transmembrane substrates has hampered progression for understanding their catalytic mechanism and screening inhibitor libraries. To identify a soluble substrate for use in the study of rhomboid proteases, an array of internally quenched peptides were assayed with HiGlpG, EcGlpG and PsAarA from Providencia stuartti. One substrate was identified that was cleaved by all three rhomboid proteases, with HiGlpG having the highest cleavage efficiency. Mass spectrometry analysis determined that all enzymes hydrolyze this substrate between norvaline and tryptophan. Kinetic analysis in both detergent and bicellular systems demonstrated that this substrate can be cleaved in solution and in the lipid environment. The substrate was subsequently used to screen a panel of benzoxazin-4-one inhibitors to validate its use in inhibitor discovery.",
        sentences: [],
        annotations: [
          {
            id: "9",
            infons: { identifier: "562", type: "Species" },
            text: "Escherichia coli",
            locations: [{ offset: 320, length: 16 }],
          },
          {
            id: "10",
            infons: { identifier: "MESH:D006192", type: "Disease" },
            text: "Haemophilus influenzae rhomboid",
            locations: [{ offset: 476, length: 31 }],
          },
          {
            id: "11",
            infons: { identifier: "MESH:D008055", type: "Chemical" },
            text: "lipid",
            locations: [{ offset: 581, length: 5 }],
          },
          {
            id: "12",
            infons: { identifier: "MESH:C566273", type: "Disease" },
            text: "rhomboid proteases",
            locations: [{ offset: 881, length: 18 }],
          },
          {
            id: "13",
            infons: { identifier: "MESH:C566273", type: "Disease" },
            text: "rhomboid proteases",
            locations: [{ offset: 1072, length: 18 }],
          },
          {
            id: "14",
            infons: { identifier: "MESH:C005313", type: "Chemical" },
            text: "norvaline",
            locations: [{ offset: 1232, length: 9 }],
          },
          {
            id: "15",
            infons: { identifier: "MESH:D014364", type: "Chemical" },
            text: "tryptophan",
            locations: [{ offset: 1246, length: 10 }],
          },
          {
            id: "16",
            infons: { identifier: "MESH:D008055", type: "Chemical" },
            text: "lipid",
            locations: [{ offset: 1387, length: 5 }],
          },
          {
            id: "17",
            infons: { identifier: "-", type: "Chemical" },
            text: "benzoxazin-4-one",
            locations: [{ offset: 1463, length: 16 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 30044760,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:D008055",
      "chemical@-",
      "disease@MESH:C566273",
      "disease@MESH:D006192",
      "chemical@MESH:C005313",
      "species@562",
      "chemical@MESH:D014364",
    ],
    journal: "Biol Chem. 2018 Nov 27",
    year: 2018,
    authors: [
      "Arutyunova E",
      "Jiang Z",
      "Yang J",
      "Kulepa AN",
      "Young HS",
      "Verhelst S",
      "O'Donoghue AJ",
      "Lemieux MJ",
    ],
  },
  {
    _id: "29725596|None",
    id: "29725596",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Front Mol Biosci. 2018 Apr 19;5:40. doi: 10.3389/fmolb.2018.00040. eCollection",
          year: "2018",
          "article-id_pmc": "PMC5917446",
          type: "title",
          authors:
            "Corvo I, Ferraro F, Merlino A, Zuberbuhler K, O'Donoghue AJ, Pastro L, Pi-Denis N, Basika T, Roche L, McKerrow JH, Craik CS, Caffrey CR, Tort JF",
          section: "Title",
        },
        offset: 0,
        text:
          "Substrate Specificity of Cysteine Proteases Beyond the S2 Pocket: Mutagenesis and Molecular Dynamics Investigation of Fasciola hepatica Cathepsins L.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "MESH:D005211", type: "Disease" },
            text: "Fasciola hepatica Cathepsins",
            locations: [{ offset: 118, length: 28 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 150,
        text:
          "Cysteine proteases are widespread in all life kingdoms, being central to diverse physiological processes based on a broad range of substrate specificity. Paralogous Fasciola hepatica cathepsin L proteases are essential to parasite invasion, tissue migration and reproduction. In spite of similarities in their overall sequence and structure, these enzymes often exhibit different substrate specificity. These preferences are principally determined by the amino acid composition of the active site's S2 subsite (pocket) of the enzyme that interacts with the substrate P2 residue (Schetcher and Berger nomenclature). Although secreted FhCL1 accommodates aliphatic residues in the S2 pocket, FhCL2 is also efficient in cleaving proline in that position. To understand these differences, we engineered the FhCL1 S2 subsite at three amino acid positions to render it identical to that present in FhCL2. The substitutions did not produce the expected increment in proline accommodation in P2. Rather, they decreased the enzyme's catalytic efficiency toward synthetic peptides. Nonetheless, a change in the P3 specificity was associated with the mutation of Leu67 to Tyr, a hinge residue between the S2 and S3 subsites that contributes to the accommodation of Gly in S3. Molecular dynamic simulations highlighted changes in the spatial distribution and secondary structure of the S2 and S3 pockets of the mutant FhCL1 enzymes. The reduced affinity and catalytic efficiency of the mutant enzymes may be due to a narrowing of the active site cleft that hinders the accommodation of substrates. Because the variations in the enzymatic activity measured could not be exclusively allocated to those residues lining the active site, other more external positions might modulate enzyme conformation, and, therefore, catalytic activity.",
        sentences: [],
        annotations: [
          {
            id: "8",
            infons: { identifier: "MESH:D005211", type: "Disease" },
            text: "Paralogous Fasciola hepatica",
            locations: [{ offset: 304, length: 28 }],
          },
          {
            id: "9",
            infons: { identifier: "MESH:D011392", type: "Chemical" },
            text: "proline",
            locations: [{ offset: 875, length: 7 }],
          },
          {
            id: "10",
            infons: { identifier: "MESH:D011392", type: "Chemical" },
            text: "proline",
            locations: [{ offset: 1108, length: 7 }],
          },
          {
            id: "11",
            infons: { identifier: "MESH:D010455", type: "Chemical" },
            text: "peptides",
            locations: [{ offset: 1211, length: 8 }],
          },
          {
            id: "12",
            infons: {
              identifier: "p.L67Y",
              type: "Mutation",
              subtype: "ProteinMutation",
            },
            text: "Leu67 to Tyr",
            locations: [{ offset: 1301, length: 12 }],
          },
          {
            id: "13",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "Gly",
            locations: [{ offset: 1403, length: 3 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 29725596,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "mutation@p.L67Y",
      "chemical@MESH:D005998",
      "disease@MESH:D005211",
      "chemical@MESH:D011392",
      "chemical@MESH:D010455",
    ],
    journal: "Front Mol Biosci. 2018 Apr 19",
    year: 2018,
    authors: [
      "Corvo I",
      "Ferraro F",
      "Merlino A",
      "Zuberbuhler K",
      "O'Donoghue AJ",
      "Pastro L",
      "Pi-Denis N",
      "Basika T",
      "Roche L",
      "McKerrow JH",
      "Craik CS",
      "Caffrey CR",
      "Tort JF",
    ],
  },
  {
    _id: "29677188|None",
    id: "29677188",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "PLoS Negl Trop Dis; 2018 Apr; 12(4) 0006446. doi:10.1371/journal.pntd.0006446",
          year: "2018",
          "article-id_pmc": "PMC5931690",
          type: "title",
          authors:
            "Leontovy\u010d A, Ulrychov\u00e1 L, O Donoghue AJ, Vondr\u00e1\u0161ek J, Mare\u0161ov\u00e1 L, Hub\u00e1lek M, Fajtov\u00e1 P, Chanov\u00e1 M, Jiang Z, Craik CS, Caffrey CR, Mare\u0161 M, Dvo\u0159\u00e1k J, Horn M",
          section: "Title",
        },
        offset: 0,
        text:
          "SmSP2: A serine protease secreted by the blood fluke pathogen Schistosoma mansoni with anti-hemostatic properties.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 115,
        text:
          "BACKGROUND: Serine proteases are important virulence factors for many pathogens. Recently, we discovered a group of trypsin-like serine proteases with domain organization unique to flatworm parasites and containing a thrombospondin type 1 repeat (TSR-1). These proteases are recognized as antigens during host infection and may prove useful as anthelminthic vaccines, however their molecular characteristics are under-studied. Here, we characterize the structural and proteolytic attributes of serine protease 2 (SmSP2) from Schistosoma mansoni, one of the major species responsible for the tropical infectious disease, schistosomiasis. METHODOLOGY/PRINCIPAL FINDINGS: SmSP2 comprises three domains: a histidine stretch, TSR-1 and a serine protease domain. The cleavage specificity of recombinant SmSP2 was determined using positional scanning and multiplex combinatorial libraries and the determinants of specificity were identified with 3D homology models, demonstrating a trypsin-like endopeptidase mode of action. SmSP2 displayed restricted proteolysis on protein substrates. It activated tissue plasminogen activator and plasminogen as key components of the fibrinolytic system, and released the vasoregulatory peptide, kinin, from kininogen. SmSP2 was detected in the surface tegument, esophageal glands and reproductive organs of the adult parasite by immunofluorescence microscopy, and in the excretory/secretory products by immunoblotting. CONCLUSIONS/SIGNIFICANCE: The data suggest that SmSP2 is secreted, functions at the host-parasite interface and contributes to the survival of the parasite by manipulating host vasodilatation and fibrinolysis. SmSP2 may be, therefore, a potential target for anti-schistosomal therapy.",
        sentences: [],
        annotations: [
          {
            id: "4",
            infons: { identifier: "MESH:D006086", type: "Disease" },
            text: "host infection",
            locations: [{ offset: 420, length: 14 }],
          },
          {
            id: "5",
            infons: { identifier: "MESH:D003141", type: "Disease" },
            text: "infectious disease",
            locations: [{ offset: 715, length: 18 }],
          },
          {
            id: "6",
            infons: { identifier: "MESH:D012552", type: "Disease" },
            text: "schistosomiasis",
            locations: [{ offset: 735, length: 15 }],
          },
          {
            id: "7",
            infons: { identifier: "MESH:D006639", type: "Chemical" },
            text: "histidine",
            locations: [{ offset: 817, length: 9 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 29677188,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D012552",
      "disease@MESH:D003141",
      "disease@MESH:D006086",
      "chemical@MESH:D006639",
    ],
    journal: "PLoS Negl Trop Dis",
    year: 2018,
    authors: [
      "Leontovy\u010d A",
      "Ulrychov\u00e1 L",
      "O Donoghue AJ",
      "Vondr\u00e1\u0161ek J",
      "Mare\u0161ov\u00e1 L",
      "Hub\u00e1lek M",
      "Fajtov\u00e1 P",
      "Chanov\u00e1 M",
      "Jiang Z",
      "Craik CS",
      "Caffrey CR",
      "Mare\u0161 M",
      "Dvo\u0159\u00e1k J",
      "Horn M",
    ],
  },
  {
    _id: "29556927|None",
    id: "29556927",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "J. Am. Soc. Mass Spectrom.; 2018 Mar 19. doi:10.1007/s13361-018-1915-0",
          year: "2018",
          "article-id_pmc": "PMC5943185",
          type: "title",
          authors:
            "Lietz CB, Toneff T, Mosier C, Podvin S, O Donoghue AJ, Hook V",
          section: "Title",
        },
        offset: 0,
        text:
          "Phosphopeptidomics Reveals Differential Phosphorylation States and Novel SxE Phosphosite Motifs of Neuropeptides in Dense Core Secretory Vesicles.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 147,
        text:
          "Neuropeptides are vital for cell-cell communication and function in the regulation of the nervous and endocrine systems. They are generated by post-translational modification (PTM) steps resulting in small active peptides generated from prohormone precursors. Phosphorylation is a significant PTM for the bioactivity of neuropeptides. From the known diversity of distinct neuropeptide functions, it is hypothesized that the extent of phosphorylation varies among different neuropeptides. To assess this hypothesis, neuropeptide-containing dense core secretory vesicles from bovine adrenal medullary chromaffin cells were subjected to global phosphopeptidomics analyses by liquid chromatography (LC)-mass spectrometry (MS/MS). Phosphopeptides were identified directly by LC-MS/MS and indirectly by phosphatase treatment followed by LC-MS/MS. The data identified numerous phosphorylated peptides derived from neuropeptide precursors such as chromogranins, secretogranins, proenkephalin and pro-NPY. Phosphosite occupancies were observed at high and low levels among identified peptides and many of the high occupancy phosphopeptides represent prohormone-derived peptides with currently unknown bioactivities. Peptide sequence analyses demonstrated SxE as the most prevalent phosphorylation site motif, corresponding to phosphorylation sites of the Fam20C protein kinase known to be present in the secretory pathway. The range of high to low phosphosite occupancies for neuropeptides demonstrates cellular regulation of neuropeptide phosphorylation. Graphical Abstract  .",
        sentences: [],
        annotations: [
          {
            id: "6",
            infons: { identifier: "9913", type: "Species" },
            text: "bovine",
            locations: [{ offset: 721, length: 6 }],
          },
          {
            id: "7",
            infons: { identifier: "-", type: "Chemical" },
            text: "chromaffin",
            locations: [{ offset: 746, length: 10 }],
          },
          {
            id: "8",
            infons: {
              identifier: "281387",
              type: "Gene",
              ncbi_homologene: "4528",
            },
            text: "proenkephalin",
            locations: [{ offset: 1117, length: 13 }],
          },
          {
            id: "9",
            infons: { identifier: "-", type: "Chemical" },
            text: "Phosphosite",
            locations: [{ offset: 1144, length: 11 }],
          },
          {
            id: "10",
            infons: {
              identifier: "534672",
              type: "Gene",
              ncbi_homologene: "56879",
            },
            text: "Fam20C",
            locations: [{ offset: 1493, length: 6 }],
          },
          {
            id: "11",
            infons: { identifier: "-", type: "Chemical" },
            text: "phosphosite",
            locations: [{ offset: 1586, length: 11 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 29556927,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: ["chemical@-", "gene@281387", "species@9913", "gene@534672"],
    journal: "J. Am. Soc. Mass Spectrom.",
    year: 2018,
    authors: [
      "Lietz CB",
      "Toneff T",
      "Mosier C",
      "Podvin S",
      "O Donoghue AJ",
      "Hook V",
    ],
  },
  {
    _id: "29246799|None",
    id: "29246799",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "J. Invest. Dermatol.; 2018 May; 138(5) 1137-1145. doi:10.1016/j.jid.2017.11.034",
          year: "2018",
          type: "title",
          authors:
            "Li H, Goh BN, Teh WK, Jiang Z, Goh JPZ, Goh A, Wu G, Hoon SS, Raida M, Camattari A, Yang L, O Donoghue AJ, Dawson TL",
          section: "Title",
        },
        offset: 0,
        text:
          "Skin Commensal Malassezia globosa Secreted Protease Attenuates Staphylococcus aureus Biofilm Formation.",
        sentences: [],
        annotations: [
          {
            id: "2",
            infons: { identifier: "MESH:D012871", type: "Disease" },
            text: "Skin Commensal Malassezia globosa Secreted",
            locations: [{ offset: 0, length: 42 }],
          },
          {
            id: "3",
            infons: { identifier: "1280", type: "Species" },
            text: "Staphylococcus aureus",
            locations: [{ offset: 63, length: 21 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 104,
        text:
          "Skin provides the first defense against pathogenic micro-organisms and is also colonized by a diverse microbiota. Phylogenetic analysis of whole skin microbiome at different skin sites in health and disease has generated important insights on possible microbial involvement in modulating skin health. However, functional roles of the skin microbial community remain unclear. The most common sebaceous skin commensal yeasts are the basidiomycetes, Malassezia. Here, we characterized the dominant secreted Malassezia globosa protease in culture and subsequently named it Malassezia globosa Secreted Aspartyl Protease 1 (MgSAP1). We defined recombinant MgSAP1's substrate cleavage profile using an unbiased, mass-spectrometry-based technique. We show that this enzyme is physiologically relevant as mgsap1 expression was detected on at least one facial skin site of 17 healthy human volunteers. In addition, we demonstrated that this protease rapidly hydrolyzes Staphylococcus aureus protein A, an important S. aureus virulence factor involved in immune evasion and biofilm formation. We further observed that MgSAP1 has anti-biofilm properties against S. aureus. Taken together, our study defines a role for the skin fungus Malassezia in inter-kingdom interactions and suggests that this fungus and the enzymes it produces may be beneficial for skin health.",
        sentences: [],
        annotations: [
          {
            id: "13",
            infons: { identifier: "4932", type: "Species" },
            text: "yeasts",
            locations: [{ offset: 520, length: 6 }],
          },
          {
            id: "14",
            infons: { type: "Disease", identifier: null, db: null },
            text: "Malassezia",
            locations: [{ offset: 551, length: 10 }],
          },
          {
            id: "15",
            infons: { identifier: "MESH:D002640", type: "Disease" },
            text: "secreted Malassezia globosa",
            locations: [{ offset: 599, length: 27 }],
          },
          {
            id: "16",
            infons: { identifier: "MESH:D002640", type: "Disease" },
            text: "Malassezia globosa Secreted",
            locations: [{ offset: 673, length: 27 }],
          },
          {
            id: "17",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 978, length: 5 }],
          },
          {
            id: "18",
            infons: { identifier: "1280", type: "Species" },
            text: "Staphylococcus aureus",
            locations: [{ offset: 1063, length: 21 }],
          },
          {
            id: "19",
            infons: { identifier: "1280", type: "Species" },
            text: "S. aureus",
            locations: [{ offset: 1109, length: 9 }],
          },
          {
            id: "20",
            infons: { identifier: "1280", type: "Species" },
            text: "S. aureus",
            locations: [{ offset: 1254, length: 9 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D009181", type: "Disease" },
            text: "skin fungus Malassezia",
            locations: [{ offset: 1314, length: 22 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 29246799,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "species@4932",
      "disease@MESH:D009181",
      "species@9606",
      "disease@MESH:D002640",
      "disease@MESH:D012871",
      "species@1280",
    ],
    journal: "J. Invest. Dermatol.",
    year: 2018,
    authors: [
      "Li H",
      "Goh BN",
      "Teh WK",
      "Jiang Z",
      "Goh JPZ",
      "Goh A",
      "Wu G",
      "Hoon SS",
      "Raida M",
      "Camattari A",
      "Yang L",
      "O Donoghue AJ",
      "Dawson TL",
    ],
  },
  {
    _id: "29182146|None",
    id: "29182146",
    infons: {},
    passages: [
      {
        infons: {
          journal: "Elife; 2017 Nov 28; 6. doi:10.7554/eLife.27364",
          year: "2017",
          "article-id_pmc": "PMC5705213",
          type: "title",
          authors:
            "Winter MB, La Greca F, Arastu-Kapur S, Caiazza F, Cimermancic P, Buchholz TJ, Anderl JL, Ravalin M, Bohn MF, Sali A, O Donoghue AJ, Craik CS",
          section: "Title",
        },
        offset: 0,
        text:
          "Immunoproteasome functions explained by divergence in cleavage specificity and regulation.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 91,
        text:
          "The immunoproteasome (iP) has been proposed to perform specialized roles in MHC class I antigen presentation, cytokine modulation, and T cell differentiation and has emerged as a promising therapeutic target for autoimmune disorders and cancer. However, divergence in function between the iP and the constitutive proteasome (cP) has been unclear. A global peptide library-based screening strategy revealed that the proteasomes have overlapping but distinct substrate specificities. Differing iP specificity alters the quantity of production of certain MHC I epitopes but does not appear to be preferentially suited for antigen presentation. Furthermore, iP specificity was found to have likely arisen through genetic drift from the ancestral cP. Specificity differences were exploited to develop isoform-selective substrates. Cellular profiling using these substrates revealed that divergence in regulation of the iP balances its relative contribution to proteasome capacity in immune cells, resulting in selective recovery from inhibition. These findings have implications for iP-targeted therapeutic development.",
        sentences: [],
        annotations: [
          {
            id: "2",
            infons: { identifier: "MESH:D001327", type: "Disease" },
            text: "autoimmune disorders",
            locations: [{ offset: 303, length: 20 }],
          },
          {
            id: "3",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "cancer",
            locations: [{ offset: 328, length: 6 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 29182146,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: ["disease@MESH:D001327", "disease@MESH:D009369"],
    journal: "Elife",
    year: 2017,
    authors: [
      "Winter MB",
      "La Greca F",
      "Arastu-Kapur S",
      "Caiazza F",
      "Cimermancic P",
      "Buchholz TJ",
      "Anderl JL",
      "Ravalin M",
      "Bohn MF",
      "Sali A",
      "O Donoghue AJ",
      "Craik CS",
    ],
  },
  {
    _id: "29168252|None",
    id: "29168252",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Protein Sci.; 2018 Mar; 27(3) 584-594. doi:10.1002/pro.3352",
          year: "2018",
          "article-id_pmc": "PMC5818756",
          type: "title",
          authors:
            "Ivry SL, Meyer NO, Winter MB, Bohn MF, Knudsen GM, O Donoghue AJ, Craik CS",
          section: "Title",
        },
        offset: 0,
        text:
          "Global substrate specificity profiling of post-translational modifying enzymes.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 80,
        text:
          "Enzymes that modify the proteome, referred to as post-translational modifying (PTM) enzymes, are central regulators of cellular signaling. Determining the substrate specificity of PTM enzymes is a critical step in unraveling their biological functions both in normal physiological processes and in disease states. Advances in peptide chemistry over the last century have enabled the rapid generation of peptide libraries for querying substrate recognition by PTM enzymes. In this article, we highlight various peptide-based approaches for analysis of PTM enzyme substrate specificity. We focus on the application of these technologies to proteases and also discuss specific examples in which they have been used to uncover the substrate specificity of other types of PTM enzymes, such as kinases. In particular, we highlight our multiplex substrate profiling by mass spectrometry (MSP-MS) assay, which uses a rationally designed, physicochemically diverse library of tetradecapeptides. We show how this method has been applied to PTM enzymes to uncover biological function, and guide substrate and inhibitor design. We also briefly discuss how this technique can be combined with other methods to gain a systems-level understanding of PTM enzyme regulation and function.",
        sentences: [],
        annotations: [],
        relations: [],
      },
    ],
    relations: [],
    pmid: 29168252,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [],
    journal: "Protein Sci.",
    year: 2018,
    authors: [
      "Ivry SL",
      "Meyer NO",
      "Winter MB",
      "Bohn MF",
      "Knudsen GM",
      "O Donoghue AJ",
      "Craik CS",
    ],
  },
  {
    _id: "28696697|None",
    id: "28696697",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "J. Med. Chem.; 2017 Aug 10; 60(15) 6721-6732. doi:10.1021/acs.jmedchem.7b00671",
          year: "2017",
          "article-id_pmc": "PMC5554889",
          type: "title",
          authors:
            "LaMonte GM, Almaliti J, Bibo-Verdugo B, Keller L, Zou BY, Yang J, Antonova-Koch Y, Orjuela-Sanchez P, Boyle CA, Vigil E, Wang L, Goldgof GM, Gerwick L, O Donoghue AJ, Winzeler EA, Gerwick WH, Ottilie S",
          section: "Title",
        },
        offset: 0,
        text:
          "Development of a Potent Inhibitor of the Plasmodium Proteasome with Reduced Mammalian Toxicity.",
        sentences: [],
        annotations: [
          {
            id: "3",
            infons: { identifier: "5833", type: "Species" },
            text: "Plasmodium",
            locations: [{ offset: 41, length: 10 }],
          },
          {
            id: "4",
            infons: { identifier: "9606", type: "Species" },
            text: "Mammalian",
            locations: [{ offset: 76, length: 9 }],
          },
          {
            id: "5",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "Toxicity",
            locations: [{ offset: 86, length: 8 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 96,
        text:
          "Naturally derived chemical compounds are the foundation of much of our pharmacopeia, especially in antiproliferative and anti-infective drug classes. Here, we report that a naturally derived molecule called carmaphycin B is a potent inhibitor against both the asexual and sexual blood stages of malaria infection. Using a combination of in silico molecular docking and in vitro directed evolution in a well-characterized drug-sensitive yeast model, we determined that these compounds target the beta5 subunit of the proteasome. These studies were validated using in vitro inhibition assays with proteasomes isolated from Plasmodium falciparum. As carmaphycin B is toxic to mammalian cells, we synthesized a series of chemical analogs that reduce host cell toxicity while maintaining blood-stage and gametocytocidal antimalarial activity and proteasome inhibition. This study describes a promising new class of antimalarial compound based on the carmaphycin B scaffold, as well as several chemical structural features that serve to enhance antimalarial specificity.",
        sentences: [],
        annotations: [
          {
            id: "14",
            infons: { identifier: "MESH:C000622250", type: "Chemical" },
            text: "carmaphycin B",
            locations: [{ offset: 303, length: 13 }],
          },
          {
            id: "15",
            infons: { identifier: "MESH:D008288", type: "Disease" },
            text: "sexual blood stages of malaria infection",
            locations: [{ offset: 368, length: 40 }],
          },
          {
            id: "16",
            infons: { identifier: "4932", type: "Species" },
            text: "yeast",
            locations: [{ offset: 532, length: 5 }],
          },
          {
            id: "17",
            infons: { identifier: "5833", type: "Species" },
            text: "Plasmodium falciparum",
            locations: [{ offset: 717, length: 21 }],
          },
          {
            id: "18",
            infons: { identifier: "MESH:C000622250", type: "Chemical" },
            text: "carmaphycin B",
            locations: [{ offset: 743, length: 13 }],
          },
          {
            id: "19",
            infons: { identifier: "9606", type: "Species" },
            text: "mammalian",
            locations: [{ offset: 769, length: 9 }],
          },
          {
            id: "20",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "toxicity",
            locations: [{ offset: 852, length: 8 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:C000622250", type: "Chemical" },
            text: "carmaphycin B",
            locations: [{ offset: 1041, length: 13 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 28696697,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:C000622250",
      "species@4932",
      "disease@MESH:D064420",
      "disease@MESH:D008288",
      "species@5833",
      "species@9606",
    ],
    journal: "J. Med. Chem.",
    year: 2017,
    authors: [
      "LaMonte GM",
      "Almaliti J",
      "Bibo-Verdugo B",
      "Keller L",
      "Zou BY",
      "Yang J",
      "Antonova-Koch Y",
      "Orjuela-Sanchez P",
      "Boyle CA",
      "Vigil E",
      "Wang L",
      "Goldgof GM",
      "Gerwick L",
      "O Donoghue AJ",
      "Winzeler EA",
      "Gerwick WH",
      "Ottilie S",
    ],
  },
  {
    _id: "28424202|None",
    id: "28424202",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Clin. Cancer Res.; 2017 Aug 15; 23(16) 4865-4874. doi:10.1158/1078-0432.CCR-16-2987",
          year: "2017",
          "article-id_pmc": "PMC5712228",
          type: "title",
          authors:
            "Ivry SL, Sharib JM, Dominguez DA, Roy N, Hatcher SE, Yip-Schneider MT, Schmidt CM, Brand RE, Park WG, Hebrok M, Kim GE, O Donoghue AJ, Kirkwood KS, Craik CS",
          section: "Title",
        },
        offset: 0,
        text:
          "Global Protease Activity Profiling Provides Differential Diagnosis of Pancreatic Cysts.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "MESH:D010181", type: "Disease" },
            text: "Pancreatic Cysts",
            locations: [{ offset: 70, length: 16 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 88,
        text:
          "Purpose: Pancreatic cysts are estimated to be present in 2%-3% of the adult population. Unfortunately, current diagnostics do not accurately distinguish benign cysts from those that can progress into invasive cancer. Misregulated pericellular proteolysis is a hallmark of malignancy, and therefore, we used a global approach to discover protease activities that differentiate benign nonmucinous cysts from premalignant mucinous cysts.Experimental Design: We employed an unbiased and global protease profiling approach to discover protease activities in 23 cyst fluid samples. The distinguishing activities of select proteases was confirmed in 110 samples using specific fluorogenic substrates and required less than 5 muL of cyst fluid.Results: We determined that the activities of the aspartyl proteases gastricsin and cathepsin E are highly increased in fluid from mucinous cysts. IHC analysis revealed that gastricsin expression was associated with regions of low-grade dysplasia, whereas cathepsin E expression was independent of dysplasia grade. Gastricsin activity differentiated mucinous from nonmucinous cysts with a specificity of 100% and a sensitivity of 93%, whereas cathepsin E activity was 92% specific and 70% sensitive. Gastricsin significantly outperformed the most widely used molecular biomarker, carcinoembryonic antigen (CEA), which demonstrated 94% specificity and 65% sensitivity. Combined analysis of gastricsin and CEA resulted in a near perfect classifier with 100% specificity and 98% sensitivity.Conclusions: Quantitation of gastricsin and cathepsin E activities accurately distinguished mucinous from nonmucinous pancreatic cysts and has the potential to replace current diagnostics for analysis of these highly prevalent lesions. Clin Cancer Res; 23(16); 4865-74.  2017 AACR.",
        sentences: [],
        annotations: [
          {
            id: "21",
            infons: { identifier: "MESH:D010181", type: "Disease" },
            text: "Pancreatic cysts",
            locations: [{ offset: 97, length: 16 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D009362", type: "Disease" },
            text: "invasive cancer",
            locations: [{ offset: 288, length: 15 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "hallmark of malignancy",
            locations: [{ offset: 348, length: 22 }],
          },
          {
            id: "24",
            infons: {
              identifier: "5225",
              type: "Gene",
              ncbi_homologene: "55669",
            },
            text: "gastricsin",
            locations: [{ offset: 893, length: 10 }],
          },
          {
            id: "25",
            infons: {
              identifier: "1510",
              type: "Gene",
              ncbi_homologene: "37551",
            },
            text: "cathepsin E",
            locations: [{ offset: 908, length: 11 }],
          },
          {
            id: "26",
            infons: {
              identifier: "5225",
              type: "Gene",
              ncbi_homologene: "55669",
            },
            text: "gastricsin",
            locations: [{ offset: 998, length: 10 }],
          },
          {
            id: "27",
            infons: { identifier: "MESH:D004476", type: "Disease" },
            text: "dysplasia",
            locations: [{ offset: 1061, length: 9 }],
          },
          {
            id: "28",
            infons: {
              identifier: "1510",
              type: "Gene",
              ncbi_homologene: "37551",
            },
            text: "cathepsin E",
            locations: [{ offset: 1080, length: 11 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D004476", type: "Disease" },
            text: "dysplasia",
            locations: [{ offset: 1122, length: 9 }],
          },
          {
            id: "30",
            infons: {
              identifier: "5225",
              type: "Gene",
              ncbi_homologene: "55669",
            },
            text: "Gastricsin",
            locations: [{ offset: 1139, length: 10 }],
          },
          {
            id: "31",
            infons: {
              identifier: "1510",
              type: "Gene",
              ncbi_homologene: "37551",
            },
            text: "cathepsin E",
            locations: [{ offset: 1267, length: 11 }],
          },
          {
            id: "32",
            infons: {
              identifier: "5225",
              type: "Gene",
              ncbi_homologene: "55669",
            },
            text: "Gastricsin",
            locations: [{ offset: 1324, length: 10 }],
          },
          {
            id: "33",
            infons: {
              identifier: "1084",
              type: "Gene",
              ncbi_homologene: "130497",
            },
            text: "carcinoembryonic antigen",
            locations: [{ offset: 1404, length: 24 }],
          },
          {
            id: "34",
            infons: {
              identifier: "1084",
              type: "Gene",
              ncbi_homologene: "130497",
            },
            text: "CEA",
            locations: [{ offset: 1430, length: 3 }],
          },
          {
            id: "35",
            infons: {
              identifier: "5225",
              type: "Gene",
              ncbi_homologene: "55669",
            },
            text: "gastricsin",
            locations: [{ offset: 1513, length: 10 }],
          },
          {
            id: "36",
            infons: {
              identifier: "1084",
              type: "Gene",
              ncbi_homologene: "130497",
            },
            text: "CEA",
            locations: [{ offset: 1528, length: 3 }],
          },
          {
            id: "37",
            infons: {
              identifier: "5225",
              type: "Gene",
              ncbi_homologene: "55669",
            },
            text: "gastricsin",
            locations: [{ offset: 1641, length: 10 }],
          },
          {
            id: "38",
            infons: {
              identifier: "1510",
              type: "Gene",
              ncbi_homologene: "37551",
            },
            text: "cathepsin E",
            locations: [{ offset: 1656, length: 11 }],
          },
          {
            id: "39",
            infons: { identifier: "MESH:D010181", type: "Disease" },
            text: "pancreatic cysts",
            locations: [{ offset: 1730, length: 16 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 28424202,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D010181",
      "gene@5225",
      "gene@1084",
      "gene@1510",
      "disease@MESH:D004476",
      "disease@MESH:D009362",
      "disease@MESH:D009369",
    ],
    journal: "Clin. Cancer Res.",
    year: 2017,
    authors: [
      "Ivry SL",
      "Sharib JM",
      "Dominguez DA",
      "Roy N",
      "Hatcher SE",
      "Yip-Schneider MT",
      "Schmidt CM",
      "Brand RE",
      "Park WG",
      "Hebrok M",
      "Kim GE",
      "O Donoghue AJ",
      "Kirkwood KS",
      "Craik CS",
    ],
  },
  {
    _id: "28414448|None",
    id: "28414448",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "ACS Chem. Biol.; 2017 06 16; 12(6) 1556-1565. doi:10.1021/acschembio.7b00143",
          year: "2017",
          "article-id_pmc": "PMC5575927",
          type: "title",
          authors:
            "Roncase EJ, Moon C, Chatterjee S, Gonz\u00e1lez-P\u00e1ez GE, Craik CS, O Donoghue AJ, Wolan DW",
          section: "Title",
        },
        offset: 0,
        text:
          "Substrate Profiling and High Resolution Co-complex Crystal Structure of a Secreted C11 Protease Conserved across Commensal Bacteria.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: {
              identifier: "51728",
              type: "Gene",
              ncbi_homologene: "5610",
            },
            text: "C11",
            locations: [{ offset: 83, length: 3 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 133,
        text:
          "Cysteine proteases are among the most abundant hydrolytic enzymes produced by bacteria, and this diverse family of proteins have significant biological roles in bacterial viability and environmental interactions. Members of the clostripain-like (C11) family of cysteine proteases from commensal gut bacterial strains have recently been shown to mediate immune responses by inducing neutrophil phagocytosis and activating bacterial pathogenic toxins. Development of substrates, inhibitors, and probes that target C11 proteases from enteric bacteria will help to establish the role of these proteins at the interface of the host and microbiome in health and disease. We employed a mass spectrometry-based substrate profiling method to identify an optimal peptide substrate of PmC11, a C11 protease secreted by the commensal bacterium Parabacteroides merdae. Using this substrate sequence information, we synthesized a panel of fluorogenic substrates to calculate kcat and KM and to evaluate the importance of the P2 amino acid for substrate turnover. A potent and irreversible tetrapeptide inhibitor with a C-terminal acyloxymethyl ketone warhead, Ac-VLTK-AOMK, was then synthesized. We determined the crystal structure of PmC11 in complex with this inhibitor and uncovered key active-site interactions that govern PmC11 substrate recognition and specificity. This is the first C11 protease structure in complex with a substrate mimetic and is also the highest resolution crystal structure of a C11 protease to date at 1.12 A resolution. Importantly, subjecting human epithelial cell lysates to PmC11 hydrolysis in combination with subtiligase-based N-terminal labeling and tandem mass spectrometry proteomics complemented the stringent substrate specificity observed in the in vitro substrate profiling experiment. The combination of chemical biological, biophysical, and biochemical techniques presented here to elucidate and characterize PmC11 substrate selectivity can be expanded to other proteases and the development of chemical tools to study these essential proteins in biologically relevant samples, such as the highly complex distal gut microbiome.",
        sentences: [],
        annotations: [
          {
            id: "11",
            infons: {
              identifier: "51728",
              type: "Gene",
              ncbi_homologene: "5610",
            },
            text: "C11",
            locations: [{ offset: 379, length: 3 }],
          },
          {
            id: "12",
            infons: {
              identifier: "51728",
              type: "Gene",
              ncbi_homologene: "5610",
            },
            text: "C11",
            locations: [{ offset: 645, length: 3 }],
          },
          {
            id: "13",
            infons: {
              identifier: "51728",
              type: "Gene",
              ncbi_homologene: "5610",
            },
            text: "C11",
            locations: [{ offset: 916, length: 3 }],
          },
          {
            id: "14",
            infons: { identifier: "46503", type: "Species" },
            text: "Parabacteroides merdae",
            locations: [{ offset: 965, length: 22 }],
          },
          {
            id: "15",
            infons: { identifier: "-", type: "Chemical" },
            text: "acyloxymethyl ketone",
            locations: [{ offset: 1249, length: 20 }],
          },
          {
            id: "16",
            infons: {
              identifier: "51728",
              type: "Gene",
              ncbi_homologene: "5610",
            },
            text: "C11",
            locations: [{ offset: 1509, length: 3 }],
          },
          {
            id: "17",
            infons: {
              identifier: "51728",
              type: "Gene",
              ncbi_homologene: "5610",
            },
            text: "C11",
            locations: [{ offset: 1626, length: 3 }],
          },
          {
            id: "18",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1693, length: 5 }],
          },
          {
            id: "19",
            infons: { identifier: "749906", type: "Species" },
            text: "gut microbiome",
            locations: [{ offset: 2275, length: 14 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 28414448,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "gene@51728",
      "species@749906",
      "chemical@-",
      "species@9606",
      "species@46503",
    ],
    journal: "ACS Chem. Biol.",
    year: 2017,
    authors: [
      "Roncase EJ",
      "Moon C",
      "Chatterjee S",
      "Gonz\u00e1lez-P\u00e1ez GE",
      "Craik CS",
      "O Donoghue AJ",
      "Wolan DW",
    ],
  },
  {
    _id: "28331908|None",
    id: "28331908",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Sci Immunol; 2016 Aug; 1(2). doi:10.1126/sciimmunol.aah3539",
          year: "2016",
          "article-id_pmc": "PMC5358671",
          type: "title",
          authors:
            "LaRock CN, Todd J, LaRock DL, Olson J, O Donoghue AJ, Robertson AA, Cooper MA, Hoffman HM, Nizet V",
          section: "Title",
        },
        offset: 0,
        text: "IL-1beta is an innate immune sensor of microbial proteolysis.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: {
              identifier: "16176",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 0, length: 8 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 62,
        text:
          "Interleukin-1beta (IL-1beta) is a key proinflammatory cytokine that drives antimicrobial immune responses. IL-1beta is aberrantly activated in autoimmune diseases, and IL-1beta inhibitors are used as therapeutic agents to treat patients with certain autoimmune disorders. Review of postmarketing surveillance of patients receiving IL-1beta inhibitors found a disproportionate reporting of invasive infections by group A Streptococcus (GAS). IL-1beta inhibition increased mouse susceptibility to GAS infection, but IL-1beta was produced independent of canonical inflammasomes. Newly synthesized IL-1beta has an amino-terminal prodomain that blocks signaling activity, which is usually proteolytically removed by caspase-1, a protease activated within the inflammasome structure. In place of host caspases, the secreted GAS cysteine protease SpeB generated mature IL-1beta. During invasive infection, GAS isolates may acquire pathoadaptive mutations eliminating SpeB expression to evade detection by IL-1beta. Pharmacological IL-1beta inhibition alleviates this selective pressure, allowing invasive infection by nonpathoadapted GAS. Thus, IL-1beta is a sensor that directly detects pathogen-associated proteolysis through an independent pathway operating in parallel with host inflammasomes. Because IL-1beta function is maintained across species, yet cleavage by caspases does not appear to be, detection of microbial proteases may represent an ancestral system of innate immune regulation.",
        sentences: [],
        annotations: [
          {
            id: "29",
            infons: {
              identifier: "16176",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 81, length: 8 }],
          },
          {
            id: "30",
            infons: {
              identifier: "3553",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 169, length: 8 }],
          },
          {
            id: "31",
            infons: { identifier: "MESH:D001327", type: "Disease" },
            text: "autoimmune diseases",
            locations: [{ offset: 205, length: 19 }],
          },
          {
            id: "32",
            infons: {
              identifier: "3553",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 230, length: 8 }],
          },
          {
            id: "33",
            infons: { identifier: "9606", type: "Species" },
            text: "patients",
            locations: [{ offset: 290, length: 8 }],
          },
          {
            id: "34",
            infons: { identifier: "MESH:D001327", type: "Disease" },
            text: "autoimmune disorders",
            locations: [{ offset: 312, length: 20 }],
          },
          {
            id: "35",
            infons: { identifier: "9606", type: "Species" },
            text: "patients",
            locations: [{ offset: 374, length: 8 }],
          },
          {
            id: "36",
            infons: {
              identifier: "3553",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 393, length: 8 }],
          },
          {
            id: "37",
            infons: { identifier: "MESH:D009362", type: "Disease" },
            text: "invasive infections",
            locations: [{ offset: 451, length: 19 }],
          },
          {
            id: "38",
            infons: { identifier: "36470", type: "Species" },
            text: "group A Streptococcus",
            locations: [{ offset: 474, length: 21 }],
          },
          {
            id: "39",
            infons: { identifier: "36470", type: "Species" },
            text: "GAS",
            locations: [{ offset: 497, length: 3 }],
          },
          {
            id: "40",
            infons: {
              identifier: "16176",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 503, length: 8 }],
          },
          {
            id: "41",
            infons: { identifier: "10090", type: "Species" },
            text: "mouse",
            locations: [{ offset: 533, length: 5 }],
          },
          {
            id: "42",
            infons: { identifier: "36470", type: "Species" },
            text: "GAS",
            locations: [{ offset: 557, length: 3 }],
          },
          {
            id: "43",
            infons: { identifier: "MESH:D007239", type: "Disease" },
            text: "infection",
            locations: [{ offset: 561, length: 9 }],
          },
          {
            id: "44",
            infons: {
              identifier: "16176",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 576, length: 8 }],
          },
          {
            id: "45",
            infons: {
              identifier: "16176",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 656, length: 8 }],
          },
          {
            id: "46",
            infons: { identifier: "36470", type: "Species" },
            text: "GAS",
            locations: [{ offset: 880, length: 3 }],
          },
          {
            id: "47",
            infons: {
              identifier: "16176",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 924, length: 8 }],
          },
          {
            id: "48",
            infons: { identifier: "MESH:D009362", type: "Disease" },
            text: "invasive infection",
            locations: [{ offset: 941, length: 18 }],
          },
          {
            id: "49",
            infons: { identifier: "36470", type: "Species" },
            text: "GAS",
            locations: [{ offset: 961, length: 3 }],
          },
          {
            id: "50",
            infons: {
              identifier: "16176",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 1060, length: 8 }],
          },
          {
            id: "51",
            infons: {
              identifier: "16176",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 1086, length: 8 }],
          },
          {
            id: "52",
            infons: { identifier: "MESH:D009362", type: "Disease" },
            text: "invasive infection",
            locations: [{ offset: 1151, length: 18 }],
          },
          {
            id: "53",
            infons: { identifier: "36470", type: "Species" },
            text: "GAS",
            locations: [{ offset: 1189, length: 3 }],
          },
          {
            id: "54",
            infons: {
              identifier: "16176",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 1200, length: 8 }],
          },
          {
            id: "55",
            infons: {
              identifier: "16176",
              type: "Gene",
              ncbi_homologene: "481",
            },
            text: "IL-1beta",
            locations: [{ offset: 1361, length: 8 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 28331908,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D007239",
      "disease@MESH:D001327",
      "species@9606",
      "gene@3553",
      "species@10090",
      "gene@16176",
      "species@36470",
      "disease@MESH:D009362",
    ],
    journal: "Sci Immunol",
    year: 2016,
    authors: [
      "LaRock CN",
      "Todd J",
      "LaRock DL",
      "Olson J",
      "O Donoghue AJ",
      "Robertson AA",
      "Cooper MA",
      "Hoffman HM",
      "Nizet V",
    ],
  },
  {
    _id: "28322550|None",
    id: "28322550",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Anal. Chem.; 2017 Apr 18; 89(8) 4550-4558. doi:10.1021/acs.analchem.6b05002",
          year: "2017",
          "article-id_pmc": "PMC5500290",
          type: "title",
          authors:
            "Meyer NO, O Donoghue AJ, Schulze-Gahmen U, Ravalin M, Moss SM, Winter MB, Knudsen GM, Craik CS",
          section: "Title",
        },
        offset: 0,
        text:
          "Multiplex Substrate Profiling by Mass Spectrometry for Kinases as a Method for Revealing Quantitative Substrate Motifs.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 120,
        text:
          "The more than 500 protein kinases comprising the human kinome catalyze hundreds of thousands of phosphorylation events to regulate a diversity of cellular functions; however, the extended substrate specificity is still unknown for many of these kinases. We report here a method for quantitatively describing kinase substrate specificity using an unbiased peptide library-based approach with direct measurement of phosphorylation by tandem liquid chromatography-tandem mass spectrometry (LC-MS/MS) peptide sequencing (multiplex substrate profiling by mass spectrometry, MSP-MS). This method can be deployed with as low as 10 nM enzyme to determine activity against S/T/Y-containing peptides; additionally, label-free quantitation is used to ascertain catalytic efficiency values for individual peptide substrates in the multiplex assay. Using this approach we developed quantitative motifs for a selection of kinases from each branch of the kinome, with and without known substrates, highlighting the applicability of the method. The sensitivity of this approach is evidenced by its ability to detect phosphorylation events from nanogram quantities of immunoprecipitated material, which allows for wider applicability of this method. To increase the information content of the quantitative kinase motifs, a sublibrary approach was used to expand the testable sequence space within a peptide library of approximately 100 members for CDK1, CDK7, and CDK9. Kinetic analysis of the HIV-1 Tat (transactivator of transcription)-positive transcription elongation factor b (P-TEFb) interaction allowed for localization of the P-TEFb phosphorylation site as well as characterization of the stimulatory effect of Tat on P-TEFb catalytic efficiency.",
        sentences: [],
        annotations: [
          {
            id: "7",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 169, length: 5 }],
          },
          {
            id: "8",
            infons: { identifier: "MESH:D010455", type: "Chemical" },
            text: "peptides",
            locations: [{ offset: 801, length: 8 }],
          },
          {
            id: "9",
            infons: {
              identifier: "983",
              type: "Gene",
              ncbi_homologene: "68203",
            },
            text: "CDK1",
            locations: [{ offset: 1551, length: 4 }],
          },
          {
            id: "10",
            infons: {
              identifier: "1022",
              type: "Gene",
              ncbi_homologene: "1363",
            },
            text: "CDK7",
            locations: [{ offset: 1557, length: 4 }],
          },
          {
            id: "11",
            infons: {
              identifier: "1025",
              type: "Gene",
              ncbi_homologene: "55566",
            },
            text: "CDK9",
            locations: [{ offset: 1567, length: 4 }],
          },
          {
            id: "12",
            infons: { identifier: "11676", type: "Species" },
            text: "HIV-1",
            locations: [{ offset: 1597, length: 5 }],
          },
          {
            id: "13",
            infons: { identifier: "MESH:C000656865", type: "Disease" },
            text: "P-TEFb catalytic efficiency",
            locations: [{ offset: 1829, length: 27 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 28322550,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "species@11676",
      "species@9606",
      "gene@983",
      "disease@MESH:C000656865",
      "gene@1025",
      "gene@1022",
      "chemical@MESH:D010455",
    ],
    journal: "Anal. Chem.",
    year: 2017,
    authors: [
      "Meyer NO",
      "O Donoghue AJ",
      "Schulze-Gahmen U",
      "Ravalin M",
      "Moss SM",
      "Winter MB",
      "Knudsen GM",
      "Craik CS",
    ],
  },
  {
    _id: "28122162|None",
    id: "28122162",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "FEBS J.; 2017 05; 284(10) 1503-1517. doi:10.1111/febs.14029",
          year: "2017",
          type: "title",
          authors: "Bibo-Verdugo B, Jiang Z, Caffrey CR, O Donoghue AJ",
          section: "Title",
        },
        offset: 0,
        text:
          "Targeting proteasomes in infectious organisms to combat disease.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 65,
        text:
          "Proteasomes are multisubunit, energy-dependent, proteolytic complexes that play an essential role in intracellular protein turnover. They are present in eukaryotes, archaea, and in some actinobacteria species. Inhibition of proteasome activity has emerged as a powerful strategy for anticancer therapy and three drugs have been approved for treatment of multiple myeloma. These compounds react covalently with a threonine residue located in the active site of a proteasome subunit to block protein degradation. Proteasomes in pathogenic organisms such as Mycobacterium tuberculosis and Plasmodium falciparum also have a nucleophilic threonine residue in the proteasome active site and are therefore sensitive to these anticancer drugs. This review summarizes efforts to validate the proteasome in pathogenic organisms as a therapeutic target. We describe several strategies that have been used to develop inhibitors with increased potency and selectivity for the pathogen proteasome relative to the human proteasome. In addition, we highlight a cell-based chemical screening approach that identified a potent, allosteric inhibitor of proteasomes found in Leishmania and Trypanosoma species. Finally, we discuss the development of proteasome inhibitors as anti-infective agents.",
        sentences: [],
        annotations: [
          {
            id: "7",
            infons: { identifier: "MESH:D009101", type: "Disease" },
            text: "multiple myeloma",
            locations: [{ offset: 419, length: 16 }],
          },
          {
            id: "8",
            infons: { identifier: "MESH:D013912", type: "Chemical" },
            text: "threonine",
            locations: [{ offset: 477, length: 9 }],
          },
          {
            id: "9",
            infons: { identifier: "1773", type: "Species" },
            text: "Mycobacterium tuberculosis",
            locations: [{ offset: 620, length: 26 }],
          },
          {
            id: "10",
            infons: { identifier: "5833", type: "Species" },
            text: "Plasmodium falciparum",
            locations: [{ offset: 651, length: 21 }],
          },
          {
            id: "11",
            infons: { identifier: "MESH:D013912", type: "Chemical" },
            text: "threonine",
            locations: [{ offset: 698, length: 9 }],
          },
          {
            id: "12",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1064, length: 5 }],
          },
          {
            id: "13",
            infons: { identifier: "MESH:D007896", type: "Disease" },
            text: "Leishmania and Trypanosoma",
            locations: [{ offset: 1220, length: 26 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 28122162,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D009101",
      "disease@MESH:D007896",
      "chemical@MESH:D013912",
      "species@5833",
      "species@1773",
      "species@9606",
    ],
    journal: "FEBS J.",
    year: 2017,
    authors: ["Bibo-Verdugo B", "Jiang Z", "Caffrey CR", "O Donoghue AJ"],
  },
  {
    _id: "27977806|None",
    id: "27977806",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "PLoS Pathog.; 2016 Dec; 12(12) 1006051. doi:10.1371/journal.ppat.1006051",
          year: "2016",
          "article-id_pmc": "PMC5158083",
          type: "title",
          authors:
            "Clarke SC, Dumesic PA, Homer CM, O Donoghue AJ, La Greca F, Pallova L, Majer P, Madhani HD, Craik CS",
          section: "Title",
        },
        offset: 0,
        text:
          "Integrated Activity and Genetic Profiling of Secreted Peptidases in Cryptococcus neoformans Reveals an Aspartyl Peptidase Required for Low pH Survival and Virulence.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "5207", type: "Species" },
            text: "Cryptococcus neoformans",
            locations: [{ offset: 68, length: 23 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 166,
        text:
          "The opportunistic fungal pathogen Cryptococcus neoformans is a major cause of mortality in immunocompromised individuals, resulting in more than 600,000 deaths per year. Many human fungal pathogens secrete peptidases that influence virulence, but in most cases the substrate specificity and regulation of these enzymes remains poorly understood. The paucity of such information is a roadblock to our understanding of the biological functions of peptidases and whether or not these enzymes are viable therapeutic targets. We report here an unbiased analysis of secreted peptidase activity and specificity in C. neoformans using a mass spectrometry-based substrate profiling strategy and subsequent functional investigations. Our initial studies revealed that global peptidase activity and specificity are dramatically altered by environmental conditions. To uncover the substrate preferences of individual enzymes and interrogate their biological functions, we constructed and profiled a ten-member gene deletion collection of candidate secreted peptidases. Through this deletion approach, we characterized the substrate specificity of three peptidases within the context of the C. neoformans secretome, including an enzyme known to be important for fungal entry into the brain. We selected a previously uncharacterized peptidase, which we term Major aspartyl peptidase 1 (May1), for detailed study due to its substantial contribution to extracellular proteolytic activity. Based on the preference of May1 for proteolysis between hydrophobic amino acids, we screened a focused library of aspartyl peptidase inhibitors and identified four high-affinity antagonists. Finally, we tested may1Delta strains in a mouse model of C. neoformans infection and found that strains lacking this enzyme are significantly attenuated for virulence. Our study reveals the secreted peptidase activity and specificity of an important human fungal pathogen, identifies responsible enzymes through genetic tests of their function, and demonstrates how this information can guide the development of high affinity small molecule inhibitors.",
        sentences: [],
        annotations: [
          {
            id: "10",
            infons: { identifier: "5207", type: "Species" },
            text: "Cryptococcus neoformans",
            locations: [{ offset: 200, length: 23 }],
          },
          {
            id: "11",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 341, length: 5 }],
          },
          {
            id: "12",
            infons: { identifier: "5207", type: "Species" },
            text: "C. neoformans",
            locations: [{ offset: 773, length: 13 }],
          },
          {
            id: "13",
            infons: { identifier: "5207", type: "Species" },
            text: "C. neoformans",
            locations: [{ offset: 1344, length: 13 }],
          },
          {
            id: "14",
            infons: { identifier: "10090", type: "Species" },
            text: "mouse",
            locations: [{ offset: 1872, length: 5 }],
          },
          {
            id: "15",
            infons: { identifier: "5207", type: "Species" },
            text: "C. neoformans",
            locations: [{ offset: 1887, length: 13 }],
          },
          {
            id: "16",
            infons: { identifier: "MESH:D007239", type: "Disease" },
            text: "infection",
            locations: [{ offset: 1901, length: 9 }],
          },
          {
            id: "17",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 2080, length: 5 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 27977806,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D007239",
      "species@5207",
      "species@9606",
      "species@10090",
    ],
    journal: "PLoS Pathog.",
    year: 2016,
    authors: [
      "Clarke SC",
      "Dumesic PA",
      "Homer CM",
      "O Donoghue AJ",
      "La Greca F",
      "Pallova L",
      "Majer P",
      "Madhani HD",
      "Craik CS",
    ],
  },
  {
    _id: "27739665|None",
    id: "27739665",
    infons: {},
    passages: [
      {
        infons: {
          journal: "ACS Infect Dis; 2016 Nov 11; 2(11) 807-815",
          year: "2016",
          "article-id_pmc": "PMC5109297",
          type: "title",
          authors:
            "Lentz CS, Ordonez AA, Kasperkiewicz P, La Greca F, O Donoghue AJ, Schulze CJ, Powers JC, Craik CS, Drag M, Jain SK, Bogyo M",
          section: "Title",
        },
        offset: 0,
        text:
          "Design of Selective Substrates and Activity-Based Probes for Hydrolase Important for Pathogenesis 1 (HIP1) from Mycobacterium tuberculosis.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "1773", type: "Species" },
            text: "Mycobacterium tuberculosis",
            locations: [{ offset: 112, length: 26 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 140,
        text:
          'Although serine proteases are important mediators of Mycobacterium tuberculosis (Mtb) virulence, there are currently no tools to selectively block or visualize members of this family of enzymes. Selective reporter substrates or activity-based probes (ABPs) could provide a means to monitor infection and response to therapy using imaging methods. Here, we use a combination of substrate selectivity profiling and focused screening to identify optimized reporter substrates and ABPs for the Mtb "Hydrolase important for pathogenesis 1" (Hip1) serine protease. Hip1 is a cell-envelope-associated enzyme with minimal homology to host proteases, making it an ideal target for probe development. We identified substituted 7-amino-4-chloro-3-(2-bromoethoxy)isocoumarins as irreversible inhibitor scaffolds. Furthermore, we used specificity data to generate selective reporter substrates and to further optimize a selective chloroisocoumarin inhibitor. These new reagents are potentially useful in delineating the roles of Hip1 during pathogenesis or as diagnostic imaging tools for specifically monitoring Mtb infections.',
        sentences: [],
        annotations: [
          {
            id: "9",
            infons: { identifier: "1773", type: "Species" },
            text: "Mycobacterium tuberculosis",
            locations: [{ offset: 193, length: 26 }],
          },
          {
            id: "10",
            infons: { identifier: "1773", type: "Species" },
            text: "Mtb",
            locations: [{ offset: 221, length: 3 }],
          },
          {
            id: "11",
            infons: { identifier: "MESH:D007239", type: "Disease" },
            text: "infection",
            locations: [{ offset: 430, length: 9 }],
          },
          {
            id: "12",
            infons: { identifier: "1773", type: "Species" },
            text: "Mtb",
            locations: [{ offset: 630, length: 3 }],
          },
          {
            id: "13",
            infons: { identifier: "-", type: "Chemical" },
            text: "7-amino-4-chloro-3-(2-bromoethoxy)isocoumarins",
            locations: [{ offset: 857, length: 46 }],
          },
          {
            id: "14",
            infons: { identifier: "-", type: "Chemical" },
            text: "chloroisocoumarin",
            locations: [{ offset: 1057, length: 17 }],
          },
          {
            id: "15",
            infons: { identifier: "1773", type: "Species" },
            text: "Mtb",
            locations: [{ offset: 1240, length: 3 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 27739665,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: ["chemical@-", "disease@MESH:D007239", "species@1773"],
    journal: "ACS Infect Dis",
    year: 2016,
    authors: [
      "Lentz CS",
      "Ordonez AA",
      "Kasperkiewicz P",
      "La Greca F",
      "O Donoghue AJ",
      "Schulze CJ",
      "Powers JC",
      "Craik CS",
      "Drag M",
      "Jain SK",
      "Bogyo M",
    ],
  },
  {
    _id: "27501047|None",
    id: "27501047",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "PLoS Negl Trop Dis; 2016 Aug; 10(8) 0004893. doi:10.1371/journal.pntd.0004893",
          year: "2016",
          "article-id_pmc": "PMC4976874",
          type: "title",
          authors:
            "Goupil LS, Ivry SL, Hsieh I, Suzuki BM, Craik CS, O Donoghue AJ, McKerrow JH",
          section: "Title",
        },
        offset: 0,
        text:
          "Cysteine and Aspartyl Proteases Contribute to Protein Digestion in the Gut of Freshwater Planaria.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "MESH:D003545", type: "Chemical" },
            text: "Cysteine",
            locations: [{ offset: 0, length: 8 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 99,
        text:
          "Proteases perform numerous vital functions in flatworms, many of which are likely to be conserved throughout the phylum Platyhelminthes. Within this phylum are several parasitic worms that are often poorly characterized due to their complex life-cycles and lack of responsiveness to genetic manipulation. The flatworm Schmidtea mediterranea, or planaria, is an ideal model organism to study the complex role of protein digestion due to its simple life cycle and amenability to techniques like RNA interference (RNAi). In this study, we were interested in deconvoluting the digestive protease system that exists in the planarian gut. To do this, we developed an alcohol-induced regurgitation technique to enrich for the gut enzymes in S. mediterranea. Using a panel of fluorescent substrates, we show that this treatment produces a sharp increase in proteolytic activity. These enzymes have broad yet diverse substrate specificity profiles. Proteomic analysis of the gut contents revealed the presence of cysteine and metallo-proteases. However, treatment with class-specific inhibitors showed that aspartyl and cysteine proteases are responsible for the majority of protein digestion. Specific RNAi knockdown of the cathepsin B-like cysteine protease (SmedCB) reduced protein degradation in vivo. Immunohistochemistry and whole-mount in situ hybridization (WISH) confirmed that the full-length and active forms of SmedCB are found in secretory cells surrounding the planaria intestinal lumen. Finally, we show that the knockdown of SmedCB reduces the speed of tissue regeneration. Defining the roles of proteases in planaria can provide insight to functions of conserved proteases in parasitic flatworms, potentially uncovering drug targets in parasites.",
        sentences: [],
        annotations: [
          {
            id: "6",
            infons: { identifier: "79327", type: "Species" },
            text: "Schmidtea mediterranea",
            locations: [{ offset: 417, length: 22 }],
          },
          {
            id: "7",
            infons: { identifier: "MESH:D000438", type: "Chemical" },
            text: "alcohol",
            locations: [{ offset: 760, length: 7 }],
          },
          {
            id: "8",
            infons: { identifier: "79327", type: "Species" },
            text: "S. mediterranea",
            locations: [{ offset: 833, length: 15 }],
          },
          {
            id: "9",
            infons: { identifier: "MESH:D003545", type: "Chemical" },
            text: "cysteine",
            locations: [{ offset: 1103, length: 8 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 27501047,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:D003545",
      "species@79327",
      "chemical@MESH:D000438",
    ],
    journal: "PLoS Negl Trop Dis",
    year: 2016,
    authors: [
      "Goupil LS",
      "Ivry SL",
      "Hsieh I",
      "Suzuki BM",
      "Craik CS",
      "O Donoghue AJ",
      "McKerrow JH",
    ],
  },
  {
    _id: "27149201|None",
    id: "27149201",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Biol. Chem.; 2016 09 01; 397(9) 871-81. doi:10.1515/hsz-2016-0138",
          year: "2016",
          "article-id_pmc": "PMC5712230",
          type: "title",
          authors:
            "O Donoghue AJ, Ivry SL, Chaudhury C, Hostetter DR, Hanahan D, Craik CS",
          section: "Title",
        },
        offset: 0,
        text:
          "Procathepsin E is highly abundant but minimally active in pancreatic ductal adenocarcinoma tumors.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "MESH:D021441", type: "Disease" },
            text: "pancreatic ductal adenocarcinoma tumors",
            locations: [{ offset: 58, length: 39 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 99,
        text:
          "The cathepsin family of lysosomal proteases is increasingly being recognized for their altered expression in cancer and role in facilitating tumor progression. The aspartyl protease cathepsin E is overexpressed in several cancers and has been investigated as a biomarker for pancreatic ductal adenocarcinoma (PDAC). Here we show that cathepsin E expression in mouse PDAC tumors is increased by more than 400-fold when compared to healthy pancreatic tissue. Cathepsin E accumulates over the course of disease progression and accounts for more than 3% of the tumor protein in mice with end-stage disease. Through immunoblot analysis we determined that only procathepsin E exists in mouse PDAC tumors and cell lines derived from these tumors. By decreasing the pH, this procathepsion E is converted to the mature form, resulting in an increase in proteolytic activity. Although active site inhibitors can bind procathepsin E, treatment of PDAC mice with the aspartyl protease inhibitor ritonavir did not decrease tumor burden. Lastly, we used multiplex substrate profiling by mass spectrometry to identify two synthetic peptides that are hydrolyzed by procathepsin E near neutral pH. This work represents a comprehensive analysis of procathepsin E in PDAC and could facilitate the development of improved biomarkers for disease detection.",
        sentences: [],
        annotations: [
          {
            id: "22",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "cancer",
            locations: [{ offset: 208, length: 6 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "tumor",
            locations: [{ offset: 240, length: 5 }],
          },
          {
            id: "24",
            infons: {
              identifier: "13034",
              type: "Gene",
              ncbi_homologene: "37551",
            },
            text: "cathepsin E",
            locations: [{ offset: 281, length: 11 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "cancers",
            locations: [{ offset: 321, length: 7 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D021441", type: "Disease" },
            text: "pancreatic ductal adenocarcinoma",
            locations: [{ offset: 374, length: 32 }],
          },
          {
            id: "27",
            infons: {
              identifier: "13034",
              type: "Gene",
              ncbi_homologene: "37551",
            },
            text: "cathepsin E",
            locations: [{ offset: 433, length: 11 }],
          },
          {
            id: "28",
            infons: { identifier: "10090", type: "Species" },
            text: "mouse",
            locations: [{ offset: 459, length: 5 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "tumors",
            locations: [{ offset: 470, length: 6 }],
          },
          {
            id: "30",
            infons: { identifier: "MESH:D010195", type: "Disease" },
            text: "pancreatic",
            locations: [{ offset: 537, length: 10 }],
          },
          {
            id: "31",
            infons: {
              identifier: "13034",
              type: "Gene",
              ncbi_homologene: "37551",
            },
            text: "Cathepsin E",
            locations: [{ offset: 556, length: 11 }],
          },
          {
            id: "32",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "tumor",
            locations: [{ offset: 656, length: 5 }],
          },
          {
            id: "33",
            infons: { identifier: "10090", type: "Species" },
            text: "mice",
            locations: [{ offset: 673, length: 4 }],
          },
          {
            id: "34",
            infons: { identifier: "MESH:D058625", type: "Disease" },
            text: "end-stage disease",
            locations: [{ offset: 683, length: 17 }],
          },
          {
            id: "35",
            infons: { identifier: "10090", type: "Species" },
            text: "mouse",
            locations: [{ offset: 779, length: 5 }],
          },
          {
            id: "36",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "tumors",
            locations: [{ offset: 790, length: 6 }],
          },
          {
            id: "37",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "tumors",
            locations: [{ offset: 831, length: 6 }],
          },
          {
            id: "38",
            infons: { identifier: "10090", type: "Species" },
            text: "mice",
            locations: [{ offset: 1040, length: 4 }],
          },
          {
            id: "39",
            infons: { identifier: "MESH:D019438", type: "Chemical" },
            text: "ritonavir",
            locations: [{ offset: 1082, length: 9 }],
          },
          {
            id: "40",
            infons: { identifier: "MESH:D009369", type: "Disease" },
            text: "tumor",
            locations: [{ offset: 1109, length: 5 }],
          },
          {
            id: "41",
            infons: { identifier: "MESH:D010455", type: "Chemical" },
            text: "peptides",
            locations: [{ offset: 1216, length: 8 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 27149201,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "gene@13034",
      "chemical@MESH:D019438",
      "disease@MESH:D058625",
      "species@10090",
      "disease@MESH:D021441",
      "disease@MESH:D010195",
      "chemical@MESH:D010455",
      "disease@MESH:D009369",
    ],
    journal: "Biol. Chem.",
    year: 2016,
    authors: [
      "O Donoghue AJ",
      "Ivry SL",
      "Chaudhury C",
      "Hostetter DR",
      "Hanahan D",
      "Craik CS",
    ],
  },
  {
    _id: "27006500|None",
    id: "27006500",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Proc. Natl. Acad. Sci. U.S.A.; 2016 Apr 05; 113(14) 2001-10. doi:10.1073/pnas.1524900113",
          year: "2016",
          "article-id_pmc": "PMC4833263",
          type: "title",
          authors:
            "Julien O, Zhuang M, Wiita AP, O Donoghue AJ, Knudsen GM, Craik CS, Wells JA",
          section: "Title",
        },
        offset: 0,
        text:
          "Quantitative MS-based enzymology of caspases reveals distinct protein substrate specificities, hierarchies, and cellular roles.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "835;839;841", type: "Gene" },
            text: "caspases",
            locations: [{ offset: 36, length: 8 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 128,
        text:
          "Proteases constitute the largest enzyme family, yet their biological roles are obscured by our rudimentary understanding of their cellular substrates. There are 12 human caspases that play crucial roles in inflammation and cell differentiation and drive the terminal stages of cell death. Recent N-terminomics technologies have begun to enumerate the diverse substrates individual caspases can cleave in complex cell lysates. It is clear that many caspases have shared substrates; however, few data exist about the catalytic efficiencies (kcat/KM) of these substrates, which is critical to understanding their true substrate preferences. In this study, we use quantitative MS to determine the catalytic efficiencies for hundreds of natural protease substrates in cellular lysate for two understudied members: caspase-2 and caspase-6. Most substrates are new, and the cleavage rates vary up to 500-fold. We compare the cleavage rates for common substrates with those found for caspase-3, caspase-7, and caspase-8, involved in apoptosis. There is little correlation in catalytic efficiencies among the five caspases, suggesting each has a unique set of preferred substrates, and thus more specialized roles than previously understood. We synthesized peptide substrates on the basis of protein cleavage sites and found similar catalytic efficiencies between the protein and peptide substrates. These data suggest the rates of proteolysis are dominated more by local primary sequence, and less by the tertiary protein fold. Our studies highlight that global quantitative rate analysis for posttranslational modification enzymes in complex milieus for native substrates is critical to better define their functions and relative sequence of events. ",
        sentences: [],
        annotations: [
          {
            id: "13",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 292, length: 5 }],
          },
          {
            id: "14",
            infons: { identifier: "835;839;841", type: "Gene" },
            text: "caspases",
            locations: [{ offset: 298, length: 8 }],
          },
          {
            id: "15",
            infons: { identifier: "MESH:D007249", type: "Disease" },
            text: "inflammation",
            locations: [{ offset: 334, length: 12 }],
          },
          {
            id: "16",
            infons: { identifier: "835;839;841", type: "Gene" },
            text: "caspases",
            locations: [{ offset: 509, length: 8 }],
          },
          {
            id: "17",
            infons: { identifier: "835;839;841", type: "Gene" },
            text: "caspases",
            locations: [{ offset: 576, length: 8 }],
          },
          {
            id: "18",
            infons: {
              identifier: "835",
              type: "Gene",
              ncbi_homologene: "7254",
            },
            text: "caspase-2",
            locations: [{ offset: 937, length: 9 }],
          },
          {
            id: "19",
            infons: {
              identifier: "839",
              type: "Gene",
              ncbi_homologene: "37455",
            },
            text: "caspase-6",
            locations: [{ offset: 951, length: 9 }],
          },
          {
            id: "20",
            infons: {
              identifier: "836",
              type: "Gene",
              ncbi_homologene: "37912",
            },
            text: "caspase-3",
            locations: [{ offset: 1104, length: 9 }],
          },
          {
            id: "21",
            infons: {
              identifier: "840",
              type: "Gene",
              ncbi_homologene: "11168",
            },
            text: "caspase-7",
            locations: [{ offset: 1115, length: 9 }],
          },
          {
            id: "22",
            infons: {
              identifier: "841",
              type: "Gene",
              ncbi_homologene: "7657",
            },
            text: "caspase-8",
            locations: [{ offset: 1130, length: 9 }],
          },
          {
            id: "23",
            infons: { identifier: "835;839;841", type: "Gene" },
            text: "caspases",
            locations: [{ offset: 1233, length: 8 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 27006500,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "gene@836",
      "disease@MESH:D007249",
      "gene@840",
      "species@9606",
      "gene@839",
      "gene@835",
      "gene@841",
      "gene@835;839;841",
    ],
    journal: "Proc. Natl. Acad. Sci. U.S.A.",
    year: 2016,
    authors: [
      "Julien O",
      "Zhuang M",
      "Wiita AP",
      "O Donoghue AJ",
      "Knudsen GM",
      "Craik CS",
      "Wells JA",
    ],
  },
  {
    _id: "26863983|None",
    id: "26863983",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Nature; 2016 Feb 11; 530(7589) 233-6. doi:10.1038/nature16936",
          year: "2016",
          "article-id_pmc": "PMC4755332",
          type: "title",
          authors:
            "Li H, O Donoghue AJ, van der Linden WA, Xie SC, Yoo E, Foe IT, Tilley L, Craik CS, da Fonseca PC, Bogyo M",
          section: "Title",
        },
        offset: 0,
        text:
          "Structure- and function-based design of Plasmodium-selective proteasome inhibitors.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "5833", type: "Species" },
            text: "Plasmodium",
            locations: [{ offset: 40, length: 10 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 84,
        text:
          "The proteasome is a multi-component protease complex responsible for regulating key processes such as the cell cycle and antigen presentation. Compounds that target the proteasome are potentially valuable tools for the treatment of pathogens that depend on proteasome function for survival and replication. In particular, proteasome inhibitors have been shown to be toxic for the malaria parasite Plasmodium falciparum at all stages of its life cycle. Most compounds that have been tested against the parasite also inhibit the mammalian proteasome, resulting in toxicity that precludes their use as therapeutic agents. Therefore, better definition of the substrate specificity and structural properties of the Plasmodium proteasome could enable the development of compounds with sufficient selectivity to allow their use as anti-malarial agents. To accomplish this goal, here we use a substrate profiling method to uncover differences in the specificities of the human and P. falciparum proteasome. We design inhibitors based on amino-acid preferences specific to the parasite proteasome, and find that they preferentially inhibit the beta2-subunit. We determine the structure of the P. falciparum 20S proteasome bound to the inhibitor using cryo-electron microscopy and single-particle analysis, to a resolution of 3.6 A. These data reveal the unusually open P. falciparum beta2 active site and provide valuable information about active-site architecture that can be used to further refine inhibitor design. Furthermore, consistent with the recent finding that the proteasome is important for stress pathways associated with resistance of artemisinin family anti-malarials, we observe growth inhibition synergism with low doses of this beta2-selective inhibitor in artemisinin-sensitive and -resistant parasites. Finally, we demonstrate that a parasite-selective inhibitor could be used to attenuate parasite growth in vivo without appreciable toxicity to the host. Thus, the Plasmodium proteasome is a chemically tractable target that could be exploited by next-generation anti-malarial agents. ",
        sentences: [],
        annotations: [
          {
            id: "15",
            infons: { identifier: "MESH:D008288", type: "Disease" },
            text: "malaria",
            locations: [{ offset: 464, length: 7 }],
          },
          {
            id: "16",
            infons: { identifier: "5833", type: "Species" },
            text: "Plasmodium falciparum",
            locations: [{ offset: 481, length: 21 }],
          },
          {
            id: "17",
            infons: { identifier: "9606", type: "Species" },
            text: "mammalian",
            locations: [{ offset: 611, length: 9 }],
          },
          {
            id: "18",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "toxicity",
            locations: [{ offset: 646, length: 8 }],
          },
          {
            id: "19",
            infons: { identifier: "5833", type: "Species" },
            text: "Plasmodium",
            locations: [{ offset: 794, length: 10 }],
          },
          {
            id: "20",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 1047, length: 5 }],
          },
          {
            id: "21",
            infons: { identifier: "5833", type: "Species" },
            text: "P. falciparum",
            locations: [{ offset: 1057, length: 13 }],
          },
          {
            id: "22",
            infons: { identifier: "5833", type: "Species" },
            text: "P. falciparum",
            locations: [{ offset: 1268, length: 13 }],
          },
          {
            id: "23",
            infons: { identifier: "5833", type: "Species" },
            text: "P. falciparum",
            locations: [{ offset: 1444, length: 13 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:C031327", type: "Chemical" },
            text: "artemisinin",
            locations: [{ offset: 1724, length: 11 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:C031327", type: "Chemical" },
            text: "artemisinin",
            locations: [{ offset: 1850, length: 11 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D064420", type: "Disease" },
            text: "toxicity",
            locations: [{ offset: 2029, length: 8 }],
          },
          {
            id: "27",
            infons: { identifier: "5833", type: "Species" },
            text: "Plasmodium",
            locations: [{ offset: 2061, length: 10 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 26863983,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D064420",
      "disease@MESH:D008288",
      "species@5833",
      "species@9606",
      "chemical@MESH:C031327",
    ],
    journal: "Nature",
    year: 2016,
    authors: [
      "Li H",
      "O Donoghue AJ",
      "van der Linden WA",
      "Xie SC",
      "Yoo E",
      "Foe IT",
      "Tilley L",
      "Craik CS",
      "da Fonseca PC",
      "Bogyo M",
    ],
  },
  {
    _id: "26613762|None",
    id: "26613762",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Mar. Biotechnol.; 2016 Apr; 18(2) 201-14. doi:10.1007/s10126-015-9681-5",
          year: "2016",
          type: "title",
          authors:
            "Bibo-Verdugo B, O Donoghue AJ, Rojo-Arreola L, Craik CS, Garc\u00eda-Carre\u00f1o F",
          section: "Title",
        },
        offset: 0,
        text:
          "Complementary Proteomic and Biochemical Analysis of Peptidases in Lobster Gastric Juice Uncovers the Functional Role of Individual Enzymes in Food Digestion.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "6693", type: "Species" },
            text: "Lobster",
            locations: [{ offset: 66, length: 7 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 158,
        text:
          "Crustaceans are a diverse group, distributed in widely variable environmental conditions for which they show an equally extensive range of biochemical adaptations. Some digestive enzymes have been studied by purification/characterization approaches. However, global analysis is crucial to understand how digestive enzymes interplay. Here, we present the first proteomic analysis of the digestive fluid from a crustacean (Homarus americanus) and identify glycosidases and peptidases as the most abundant classes of hydrolytic enzymes. The digestion pathway of complex carbohydrates was predicted by comparing the lobster enzymes to similar enzymes from other crustaceans. A novel and unbiased substrate profiling approach was used to uncover the global proteolytic specificity of gastric juice and determine the contribution of cysteine and aspartic acid peptidases. These enzymes were separated by gel electrophoresis and their individual substrate specificities uncovered from the resulting gel bands. This new technique is called zymoMSP. Each cysteine peptidase cleaves a set of unique peptide bonds and the S2 pocket determines their substrate specificity. Finally, affinity chromatography was used to enrich for a digestive cathepsin D1 to compare its substrate specificity and cold-adapted enzymatic properties to mammalian enzymes. We conclude that the H. americanus digestive peptidases may have useful therapeutic applications, due to their cold-adaptation properties and ability to hydrolyze collagen. ",
        sentences: [],
        annotations: [
          {
            id: "8",
            infons: { identifier: "6706", type: "Species" },
            text: "Homarus americanus",
            locations: [{ offset: 579, length: 18 }],
          },
          {
            id: "9",
            infons: { identifier: "MESH:D002241", type: "Chemical" },
            text: "carbohydrates",
            locations: [{ offset: 725, length: 13 }],
          },
          {
            id: "10",
            infons: { identifier: "6693", type: "Species" },
            text: "lobster",
            locations: [{ offset: 770, length: 7 }],
          },
          {
            id: "11",
            infons: { identifier: "MESH:D003545", type: "Chemical" },
            text: "cysteine",
            locations: [{ offset: 985, length: 8 }],
          },
          {
            id: "12",
            infons: { identifier: "9606", type: "Species" },
            text: "mammalian",
            locations: [{ offset: 1478, length: 9 }],
          },
          {
            id: "13",
            infons: { identifier: "6706", type: "Species" },
            text: "H. americanus",
            locations: [{ offset: 1518, length: 13 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 26613762,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "species@6693",
      "species@9606",
      "chemical@MESH:D003545",
      "chemical@MESH:D002241",
      "species@6706",
    ],
    journal: "Mar. Biotechnol.",
    year: 2016,
    authors: [
      "Bibo-Verdugo B",
      "O Donoghue AJ",
      "Rojo-Arreola L",
      "Craik CS",
      "Garc\u00eda-Carre\u00f1o F",
    ],
  },
  {
    _id: "26409899|None",
    id: "26409899",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Biochimie; 2016 Mar; 12299-109. doi:10.1016/j.biochi.2015.09.025",
          year: "2016",
          "article-id_pmc": "PMC4747843",
          type: "title",
          authors:
            "Dvo\u0159\u00e1k J, Fajtov\u00e1 P, Ulrychov\u00e1 L, Leontovy\u010d A, Rojo-Arreola L, Suzuki BM, Horn M, Mare\u0161 M, Craik CS, Caffrey CR, O Donoghue AJ",
          section: "Title",
        },
        offset: 0,
        text:
          "Excretion/secretion products from Schistosoma mansoni adults, eggs and schistosomula have unique peptidase specificity profiles.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 129,
        text:
          "Schistosomiasis is one of a number of chronic helminth diseases of poverty that severely impact personal and societal well-being and productivity. Peptidases (proteases) are vital to successful parasitism, and can modulate host physiology and immunology. Interference of peptidase action by specific drugs or vaccines can be therapeutically beneficial. To date, research on peptidases in the schistosome parasite has focused on either the functional characterization of individual peptidases or their annotation as part of global genome or transcriptome studies. We were interested in functionally characterizing the complexity of peptidase activity operating at the host-parasite interface, therefore the excretory-secretory products of key developmental stages of Schistosoma mansoni that parasitize the human were examined. Using class specific peptidase inhibitors in combination with a multiplex substrate profiling assay, a number of unique activities derived from endo- and exo-peptidases were revealed in the excretory-secretory products of schistosomula (larval migratory worms), adults and eggs. The data highlight the complexity of the functional degradome for each developmental stage of this parasite and facilitate further enquiry to establish peptidase identity, physiological and immunological function, and utility as drug or vaccine candidates. ",
        sentences: [],
        annotations: [
          {
            id: "4",
            infons: { identifier: "MESH:D012552", type: "Disease" },
            text: "Schistosomiasis",
            locations: [{ offset: 129, length: 15 }],
          },
          {
            id: "5",
            infons: { identifier: "MESH:D004194", type: "Disease" },
            text: "helminth diseases",
            locations: [{ offset: 175, length: 17 }],
          },
          {
            id: "6",
            infons: { identifier: "MESH:D010272", type: "Disease" },
            text: "parasitism",
            locations: [{ offset: 323, length: 10 }],
          },
          {
            id: "7",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 935, length: 5 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 26409899,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D012552",
      "disease@MESH:D004194",
      "disease@MESH:D010272",
      "species@9606",
    ],
    journal: "Biochimie",
    year: 2016,
    authors: [
      "Dvo\u0159\u00e1k J",
      "Fajtov\u00e1 P",
      "Ulrychov\u00e1 L",
      "Leontovy\u010d A",
      "Rojo-Arreola L",
      "Suzuki BM",
      "Horn M",
      "Mare\u0161 M",
      "Craik CS",
      "Caffrey CR",
      "O Donoghue AJ",
    ],
  },
  {
    _id: "25944934|None",
    id: "25944934",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Proc. Natl. Acad. Sci. U.S.A.; 2015 Jun 16; 112(24) 7478-83. doi:10.1073/pnas.1507082112",
          year: "2015",
          "article-id_pmc": "PMC4475985",
          type: "title",
          authors:
            "O Donoghue AJ, Knudsen GM, Beekman C, Perry JA, Johnson AD, DeRisi JL, Craik CS, Bennett RJ",
          section: "Title",
        },
        offset: 0,
        text:
          "Destructin-1 is a collagen-degrading endopeptidase secreted by Pseudogymnoascus destructans, the causative agent of white-nose syndrome.",
        sentences: [],
        annotations: [
          {
            id: "2",
            infons: { identifier: "655981", type: "Species" },
            text: "Pseudogymnoascus destructans",
            locations: [{ offset: 63, length: 28 }],
          },
          {
            id: "3",
            infons: { identifier: "MESH:D056784", type: "Disease" },
            text: "white-nose syndrome",
            locations: [{ offset: 116, length: 19 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 137,
        text:
          "Pseudogymnoascus destructans is the causative agent of white-nose syndrome, a disease that has caused the deaths of millions of bats in North America. This psychrophilic fungus proliferates at low temperatures and targets hibernating bats, resulting in their premature arousal from stupor with catastrophic consequences. Despite the impact of white-nose syndrome, little is known about the fungus itself or how it infects its mammalian host. P. destructans is not amenable to genetic manipulation, and therefore understanding the proteins involved in infection requires alternative approaches. Here, we identify hydrolytic enzymes secreted by P. destructans, and use a novel and unbiased substrate profiling technique to define active peptidases. These experiments revealed that endopeptidases are the major proteolytic activities secreted by P. destructans, and that collagen, the major structural protein in mammals, is actively degraded by the secretome. A serine endopeptidase, hereby-named Destructin-1, was subsequently identified, and a recombinant form overexpressed and purified. Biochemical analysis of Destructin-1 showed that it mediated collagen degradation, and a potent inhibitor of peptidase activity was identified. Treatment of P. destructans-conditioned media with this antagonist blocked collagen degradation and facilitated the detection of additional secreted proteolytic activities, including aminopeptidases and carboxypeptidases. These results provide molecular insights into the secretome of P. destructans, and identify serine endopeptidases that have the clear potential to facilitate tissue invasion and pathogenesis in the mammalian host. ",
        sentences: [],
        annotations: [
          {
            id: "16",
            infons: { identifier: "655981", type: "Species" },
            text: "Pseudogymnoascus destructans",
            locations: [{ offset: 137, length: 28 }],
          },
          {
            id: "17",
            infons: { identifier: "MESH:D056784", type: "Disease" },
            text: "white-nose syndrome",
            locations: [{ offset: 192, length: 19 }],
          },
          {
            id: "18",
            infons: { identifier: "MESH:D004194", type: "Disease" },
            text: "a disease",
            locations: [{ offset: 213, length: 9 }],
          },
          {
            id: "19",
            infons: { identifier: "MESH:D056784", type: "Disease" },
            text: "white-nose syndrome",
            locations: [{ offset: 480, length: 19 }],
          },
          {
            id: "20",
            infons: { identifier: "9606", type: "Species" },
            text: "mammalian",
            locations: [{ offset: 563, length: 9 }],
          },
          {
            id: "21",
            infons: { identifier: "655981", type: "Species" },
            text: "P. destructans",
            locations: [{ offset: 579, length: 14 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D007239", type: "Disease" },
            text: "infection",
            locations: [{ offset: 688, length: 9 }],
          },
          {
            id: "23",
            infons: { identifier: "655981", type: "Species" },
            text: "P. destructans",
            locations: [{ offset: 780, length: 14 }],
          },
          {
            id: "24",
            infons: { identifier: "655981", type: "Species" },
            text: "P. destructans",
            locations: [{ offset: 980, length: 14 }],
          },
          {
            id: "25",
            infons: { identifier: "655981", type: "Species" },
            text: "P. destructans",
            locations: [{ offset: 1383, length: 14 }],
          },
          {
            id: "26",
            infons: { identifier: "655981", type: "Species" },
            text: "P. destructans",
            locations: [{ offset: 1655, length: 14 }],
          },
          {
            id: "27",
            infons: { identifier: "9606", type: "Species" },
            text: "mammalian",
            locations: [{ offset: 1790, length: 9 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 25944934,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D007239",
      "species@655981",
      "species@9606",
      "disease@MESH:D004194",
      "disease@MESH:D056784",
    ],
    journal: "Proc. Natl. Acad. Sci. U.S.A.",
    year: 2015,
    authors: [
      "O Donoghue AJ",
      "Knudsen GM",
      "Beekman C",
      "Perry JA",
      "Johnson AD",
      "DeRisi JL",
      "Craik CS",
      "Bennett RJ",
    ],
  },
  {
    _id: "25279554|None",
    id: "25279554",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Biochem. J.; 2015 Jan 01; 465(1) 163-73. doi:10.1042/BJ20140809",
          year: "2015",
          type: "title",
          authors:
            "Sharma V, Panwar P, O Donoghue AJ, Cui H, Guido RV, Craik CS, Br\u00f6mme D",
          section: "Title",
        },
        offset: 0,
        text:
          "Structural requirements for the collagenase and elastase activity of cathepsin K and its selective inhibition by an exosite inhibitor.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: {
              identifier: "1513",
              type: "Gene",
              ncbi_homologene: "68053",
            },
            text: "cathepsin K",
            locations: [{ offset: 69, length: 11 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 135,
        text:
          "Human cathepsin K (CatK) is a major drug target for the treatment of osteoporosis. Although its collagenase activity is unique, CatK also exerts a potent elastolytic activity that is shared with human cathepsins V and S. Other members of the cysteine cathepsin family, which are structurally similar, do not exhibit significant collagen and elastin degrading activities. This raises the question of the presence of specific structural elements, exosites, that are required for these activities. CatK has two exosites that control its collagenolytic and elastolytic activity. Modifications of exosites 1 and 2 block the elastase activity of CatK, whereas only exosite-1 alterations prevent collagenolysis. Neither exosite affects the catalytic activity, protease stability, subsite specificity of CatK or the degradation of other biological substrates by this protease. A low-molecular-mass inhibitor that docks into exosite-1 inhibits the elastase and collagenase activity of CatK without interfering with the degradation of other protein substrates. The identification of CatK exosites opens up the prospect of designing highly potent inhibitors that selectively inhibit the degradation of therapeutically relevant substrates by this multifunctional protease. ",
        sentences: [],
        annotations: [
          {
            id: "15",
            infons: { identifier: "9606", type: "Species" },
            text: "Human",
            locations: [{ offset: 135, length: 5 }],
          },
          {
            id: "16",
            infons: {
              identifier: "1513",
              type: "Gene",
              ncbi_homologene: "68053",
            },
            text: "cathepsin K",
            locations: [{ offset: 141, length: 11 }],
          },
          {
            id: "17",
            infons: {
              identifier: "1513",
              type: "Gene",
              ncbi_homologene: "68053",
            },
            text: "CatK",
            locations: [{ offset: 154, length: 4 }],
          },
          {
            id: "18",
            infons: { identifier: "MESH:D010024", type: "Disease" },
            text: "osteoporosis",
            locations: [{ offset: 204, length: 12 }],
          },
          {
            id: "19",
            infons: {
              identifier: "1513",
              type: "Gene",
              ncbi_homologene: "68053",
            },
            text: "CatK",
            locations: [{ offset: 263, length: 4 }],
          },
          {
            id: "20",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 330, length: 5 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D003545", type: "Chemical" },
            text: "cysteine",
            locations: [{ offset: 377, length: 8 }],
          },
          {
            id: "22",
            infons: { identifier: "2006", type: "Gene" },
            text: "elastin",
            locations: [{ offset: 476, length: 7 }],
          },
          {
            id: "23",
            infons: {
              identifier: "1513",
              type: "Gene",
              ncbi_homologene: "68053",
            },
            text: "CatK",
            locations: [{ offset: 630, length: 4 }],
          },
          {
            id: "24",
            infons: {
              identifier: "1513",
              type: "Gene",
              ncbi_homologene: "68053",
            },
            text: "CatK",
            locations: [{ offset: 775, length: 4 }],
          },
          {
            id: "25",
            infons: {
              identifier: "1513",
              type: "Gene",
              ncbi_homologene: "68053",
            },
            text: "CatK",
            locations: [{ offset: 931, length: 4 }],
          },
          {
            id: "26",
            infons: {
              identifier: "1513",
              type: "Gene",
              ncbi_homologene: "68053",
            },
            text: "CatK",
            locations: [{ offset: 1111, length: 4 }],
          },
          {
            id: "27",
            infons: {
              identifier: "1513",
              type: "Gene",
              ncbi_homologene: "68053",
            },
            text: "CatK",
            locations: [{ offset: 1208, length: 4 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 25279554,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "gene@1513",
      "gene@2006",
      "species@9606",
      "chemical@MESH:D003545",
      "disease@MESH:D010024",
    ],
    journal: "Biochem. J.",
    year: 2015,
    authors: [
      "Sharma V",
      "Panwar P",
      "O Donoghue AJ",
      "Cui H",
      "Guido RV",
      "Craik CS",
      "Br\u00f6mme D",
    ],
  },
  {
    _id: "24073241|None",
    id: "24073241",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "PLoS ONE; 2013; 8(9) 75141. doi:10.1371/journal.pone.0075141",
          year: "2013",
          "article-id_pmc": "PMC3779220",
          type: "title",
          authors:
            "O Donoghue AJ, Jin Y, Knudsen GM, Perera NC, Jenne DE, Murphy JE, Craik CS, Hermiston TW",
          section: "Title",
        },
        offset: 0,
        text:
          "Global substrate profiling of proteases in human neutrophil extracellular traps reveals consensus motif predominantly contributed by elastase.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 43, length: 5 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 143,
        text:
          'Neutrophil extracellular traps (NETs) consist of antimicrobial molecules embedded in a web of extracellular DNA. Formation of NETs is considered to be a defense mechanism utilized by neutrophils to ensnare and kill invading pathogens, and has been recently termed NETosis. Neutrophils can be stimulated to undergo NETosis ex vivo, and are predicted to contain high levels of serine proteases, such as neutrophil elastase (NE), cathepsin G (CG) and proteinase 3 (PR3). Serine proteases are important effectors of neutrophil-mediated immunity, which function directly by degrading pathogenic virulent factors and indirectly via proteolytic activation or deactivation of cytokines, chemokines and receptors. In this study, we utilized a diverse and unbiased peptide library to detect and profile protease activity associated with NETs induced by phorbol-12-myristate-13-acetate (PMA). We obtained a "proteolytic signature" from NETs derived from healthy donor neutrophils and used proteomics to assist in the identification of the source of this proteolytic activity. In addition, we profiled each neutrophil serine protease and included the newly identified enzyme, neutrophil serine protease 4 (NSP4). Each enzyme had overlapping yet distinct endopeptidase activities and often cleaved at unique sites within the same peptide substrate. The dominant proteolytic activity in NETs was attributed to NE; however, cleavage sites corresponding to CG and PR3 activity were evident. When NE was immunodepleted, the remaining activity was attributed to CG and to a lesser extent PR3 and NSP4. Our results suggest that blocking NE activity would abrogate the major protease activity associated with NETs. In addition, the newly identified substrate specificity signatures will guide the design of more specific probes and inhibitors that target NET-associated proteases. ',
        sentences: [],
        annotations: [
          {
            id: "18",
            infons: {
              identifier: "1991",
              type: "Gene",
              ncbi_homologene: "20455",
            },
            text: "neutrophil elastase",
            locations: [{ offset: 544, length: 19 }],
          },
          {
            id: "19",
            infons: {
              identifier: "1991",
              type: "Gene",
              ncbi_homologene: "20455",
            },
            text: "NE",
            locations: [{ offset: 565, length: 2 }],
          },
          {
            id: "20",
            infons: {
              identifier: "1511",
              type: "Gene",
              ncbi_homologene: "105646",
            },
            text: "cathepsin G",
            locations: [{ offset: 570, length: 11 }],
          },
          {
            id: "21",
            infons: {
              identifier: "5657",
              type: "Gene",
              ncbi_homologene: "120934",
            },
            text: "proteinase 3",
            locations: [{ offset: 591, length: 12 }],
          },
          {
            id: "22",
            infons: {
              identifier: "5657",
              type: "Gene",
              ncbi_homologene: "120934",
            },
            text: "PR3",
            locations: [{ offset: 605, length: 3 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:D013755", type: "Chemical" },
            text: "phorbol-12-myristate-13-acetate",
            locations: [{ offset: 986, length: 31 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D013755", type: "Chemical" },
            text: "PMA",
            locations: [{ offset: 1019, length: 3 }],
          },
          {
            id: "25",
            infons: { identifier: "9606", type: "Species" },
            text: "donor",
            locations: [{ offset: 1094, length: 5 }],
          },
          {
            id: "26",
            infons: {
              identifier: "400668",
              type: "Gene",
              ncbi_homologene: "133812",
            },
            text: "neutrophil serine protease 4",
            locations: [{ offset: 1307, length: 28 }],
          },
          {
            id: "27",
            infons: {
              identifier: "400668",
              type: "Gene",
              ncbi_homologene: "133812",
            },
            text: "NSP4",
            locations: [{ offset: 1337, length: 4 }],
          },
          {
            id: "28",
            infons: {
              identifier: "1991",
              type: "Gene",
              ncbi_homologene: "20455",
            },
            text: "NE",
            locations: [{ offset: 1539, length: 2 }],
          },
          {
            id: "29",
            infons: {
              identifier: "5657",
              type: "Gene",
              ncbi_homologene: "120934",
            },
            text: "PR3",
            locations: [{ offset: 1591, length: 3 }],
          },
          {
            id: "30",
            infons: {
              identifier: "1991",
              type: "Gene",
              ncbi_homologene: "20455",
            },
            text: "NE",
            locations: [{ offset: 1623, length: 2 }],
          },
          {
            id: "31",
            infons: {
              identifier: "5657",
              type: "Gene",
              ncbi_homologene: "120934",
            },
            text: "PR3",
            locations: [{ offset: 1713, length: 3 }],
          },
          {
            id: "32",
            infons: {
              identifier: "400668",
              type: "Gene",
              ncbi_homologene: "133812",
            },
            text: "NSP4",
            locations: [{ offset: 1721, length: 4 }],
          },
          {
            id: "33",
            infons: {
              identifier: "1991",
              type: "Gene",
              ncbi_homologene: "20455",
            },
            text: "NE",
            locations: [{ offset: 1761, length: 2 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 24073241,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "gene@1511",
      "gene@400668",
      "gene@5657",
      "species@9606",
      "chemical@MESH:D013755",
      "gene@1991",
    ],
    journal: "PLoS ONE",
    year: 2013,
    authors: [
      "O Donoghue AJ",
      "Jin Y",
      "Knudsen GM",
      "Perera NC",
      "Jenne DE",
      "Murphy JE",
      "Craik CS",
      "Hermiston TW",
    ],
  },
  {
    _id: "23875031|None",
    id: "23875031",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "PLoS Negl Trop Dis; 2013; 7(7) 2269. doi:10.1371/journal.pntd.0002269",
          year: "2013",
          "article-id_pmc": "PMC3708847",
          type: "title",
          authors:
            "Corvo I, O Donoghue AJ, Pastro L, Pi-Denis N, Eroy-Reveles A, Roche L, McKerrow JH, Dalton JP, Craik CS, Caffrey CR, Tort JF",
          section: "Title",
        },
        offset: 0,
        text:
          "Dissecting the active site of the collagenolytic cathepsin L3 protease of the invasive stage of Fasciola hepatica.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "6192", type: "Species" },
            text: "Fasciola hepatica",
            locations: [{ offset: 96, length: 17 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 115,
        text:
          "BACKGROUND: A family of secreted cathepsin L proteases with differential activities is essential for host colonization and survival in the parasitic flatworm Fasciola hepatica. While the blood feeding adult secretes predominantly FheCL1, an enzyme with a strong preference for Leu at the S2 pocket of the active site, the infective stage produces FheCL3, a unique enzyme with collagenolytic activity that favours Pro at P2. METHODOLOGY/PRINCIPAL FINDINGS: Using a novel unbiased multiplex substrate profiling and mass spectrometry methodology (MSP-MS), we compared the preferences of FheCL1 and FheCL3 along the complete active site cleft and confirm that while the S2 imposes the greatest influence on substrate selectivity, preferences can be indicated on other active site subsites. Notably, we discovered that the activity of FheCL1 and FheCL3 enzymes is very different, sharing only 50% of the cleavage sites, supporting the idea of functional specialization. We generated variants of FheCL1 and FheCL3 with S2 and S3 residues by mutagenesis and evaluated their substrate specificity using positional scanning synthetic combinatorial libraries (PS-SCL). Besides the rare P2 Pro preference, FheCL3 showed a distinctive specificity at the S3 pocket, accommodating preferentially the small Gly residue. Both P2 Pro and P3 Gly preferences were strongly reduced when Trp67 of FheCL3 was replaced by Leu, rendering the enzyme incapable of digesting collagen. In contrast, the inverse Leu67Trp substitution in FheCL1 only slightly reduced its Leu preference and improved Pro acceptance in P2, but greatly increased accommodation of Gly at S3. CONCLUSIONS/SIGNIFICANCE: These data reveal the significance of S2 and S3 interactions in substrate binding emphasizing the role for residue 67 in modulating both sites, providing a plausible explanation for the FheCL3 collagenolytic activity essential to host invasion. The unique specificity of FheCL3 could be exploited in the design of specific inhibitors selectively directed to specific infective stage parasite proteinases.",
        sentences: [],
        annotations: [
          {
            id: "14",
            infons: { identifier: "6192", type: "Species" },
            text: "Fasciola hepatica",
            locations: [{ offset: 273, length: 17 }],
          },
          {
            id: "15",
            infons: { identifier: "MESH:D007930", type: "Chemical" },
            text: "Leu",
            locations: [{ offset: 392, length: 3 }],
          },
          {
            id: "16",
            infons: { identifier: "MESH:D002972", type: "Disease" },
            text: "cleft",
            locations: [{ offset: 748, length: 5 }],
          },
          {
            id: "17",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "Gly",
            locations: [{ offset: 1407, length: 3 }],
          },
          {
            id: "18",
            infons: { identifier: "MESH:D011392", type: "Chemical" },
            text: "Pro",
            locations: [{ offset: 1428, length: 3 }],
          },
          {
            id: "19",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "Gly",
            locations: [{ offset: 1439, length: 3 }],
          },
          {
            id: "20",
            infons: { identifier: "-", type: "Chemical" },
            text: "Trp67",
            locations: [{ offset: 1482, length: 5 }],
          },
          {
            id: "21",
            infons: { identifier: "-", type: "Chemical" },
            text: "FheCL3",
            locations: [{ offset: 1491, length: 6 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D007930", type: "Chemical" },
            text: "Leu",
            locations: [{ offset: 1514, length: 3 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:D007930", type: "Chemical" },
            text: "Leu",
            locations: [{ offset: 1656, length: 3 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D011392", type: "Chemical" },
            text: "Pro",
            locations: [{ offset: 1684, length: 3 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "Gly",
            locations: [{ offset: 1745, length: 3 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 23875031,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "species@6192",
      "chemical@-",
      "chemical@MESH:D005998",
      "chemical@MESH:D007930",
      "disease@MESH:D002972",
      "chemical@MESH:D011392",
    ],
    journal: "PLoS Negl Trop Dis",
    year: 2013,
    authors: [
      "Corvo I",
      "O Donoghue AJ",
      "Pastro L",
      "Pi-Denis N",
      "Eroy-Reveles A",
      "Roche L",
      "McKerrow JH",
      "Dalton JP",
      "Craik CS",
      "Caffrey CR",
      "Tort JF",
    ],
  },
  {
    _id: "23822125|None",
    id: "23822125",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Bioanalysis; 2013 Jul; 5(13) 1617-21. doi:10.4155/bio.13.98",
          year: "2013",
          type: "title",
          authors: "O Donoghue AJ, Hua S, Meyer B, Ros A",
          section: "Title",
        },
        offset: 0,
        text: "Bioanalysis Young Investigator Award 2013.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 43,
        text: "",
        sentences: [],
        annotations: [],
        relations: [],
      },
    ],
    relations: [],
    pmid: 23822125,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [],
    journal: "Bioanalysis",
    year: 2013,
    authors: ["O Donoghue AJ", "Hua S", "Meyer B", "Ros A"],
  },
  {
    _id: "23504313|None",
    id: "23504313",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "J. Biol. Chem.; 2013 May 03; 288(18) 12489-99. doi:10.1074/jbc.M113.456541",
          year: "2013",
          "article-id_pmc": "PMC3642297",
          type: "title",
          authors:
            "Small JL, O Donoghue AJ, Boritsch EC, Tsodikov OV, Knudsen GM, Vandal O, Craik CS, Ehrt S",
          section: "Title",
        },
        offset: 0,
        text:
          "Substrate specificity of MarP, a periplasmic protease required for resistance to acid and oxidative stress in Mycobacterium tuberculosis.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "1773", type: "Species" },
            text: "Mycobacterium tuberculosis",
            locations: [{ offset: 110, length: 26 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 138,
        text:
          "The transmembrane serine protease MarP is important for pH homeostasis in Mycobacterium tuberculosis (Mtb). Previous structural studies revealed that MarP contains a chymotrypsin fold and a disulfide bond that stabilizes the protease active site in the substrate-bound conformation. Here, we determined that MarP is located in the Mtb periplasm and showed that this localization is essential for function. Using the recombinant protease domain of MarP, we identified its substrate specificity using two independent assays: positional-scanning synthetic combinatorial library profiling and multiplex substrate profiling by mass spectrometry. These methods revealed that MarP prefers bulky residues at P4, tryptophan or leucine at P2, arginine or hydrophobic residues at P1, and alanine or asparagine at P1'. Guided by these data, we designed fluorogenic peptide substrates and characterized the kinetic properties of MarP. Finally, we tested the impact of mutating MarP cysteine residues on the peptidolytic activity of recombinant MarP and its ability to complement phenotypes of Mtb DeltaMarP. Taken together, our studies provide insight into the enzymatic properties of MarP, its substrate preference, and the importance of its transmembrane helices and disulfide bond.",
        sentences: [],
        annotations: [
          {
            id: "14",
            infons: { identifier: "1773", type: "Species" },
            text: "Mycobacterium tuberculosis",
            locations: [{ offset: 212, length: 26 }],
          },
          {
            id: "15",
            infons: { identifier: "1773", type: "Species" },
            text: "Mtb",
            locations: [{ offset: 240, length: 3 }],
          },
          {
            id: "16",
            infons: { identifier: "MESH:D004220", type: "Chemical" },
            text: "disulfide",
            locations: [{ offset: 328, length: 9 }],
          },
          {
            id: "17",
            infons: { identifier: "1773", type: "Species" },
            text: "Mtb",
            locations: [{ offset: 469, length: 3 }],
          },
          {
            id: "18",
            infons: { identifier: "MESH:D014364", type: "Chemical" },
            text: "tryptophan",
            locations: [{ offset: 842, length: 10 }],
          },
          {
            id: "19",
            infons: { identifier: "MESH:D007930", type: "Chemical" },
            text: "leucine",
            locations: [{ offset: 856, length: 7 }],
          },
          {
            id: "20",
            infons: { identifier: "MESH:D001120", type: "Chemical" },
            text: "arginine",
            locations: [{ offset: 871, length: 8 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D000409", type: "Chemical" },
            text: "alanine",
            locations: [{ offset: 915, length: 7 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D001216", type: "Chemical" },
            text: "asparagine",
            locations: [{ offset: 926, length: 10 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:D003545", type: "Chemical" },
            text: "cysteine",
            locations: [{ offset: 1107, length: 8 }],
          },
          {
            id: "24",
            infons: { identifier: "1773", type: "Species" },
            text: "Mtb",
            locations: [{ offset: 1218, length: 3 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:D004220", type: "Chemical" },
            text: "disulfide",
            locations: [{ offset: 1394, length: 9 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 23504313,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:D000409",
      "species@1773",
      "chemical@MESH:D003545",
      "chemical@MESH:D007930",
      "chemical@MESH:D001216",
      "chemical@MESH:D014364",
      "chemical@MESH:D001120",
      "chemical@MESH:D004220",
    ],
    journal: "J. Biol. Chem.",
    year: 2013,
    authors: [
      "Small JL",
      "O Donoghue AJ",
      "Boritsch EC",
      "Tsodikov OV",
      "Knudsen GM",
      "Vandal O",
      "Craik CS",
      "Ehrt S",
    ],
  },
  {
    _id: "23023596|None",
    id: "23023596",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Nat. Methods; 2012 Nov; 9(11) 1095-100. doi:10.1038/nmeth.2182",
          year: "2012",
          "article-id_pmc": "PMC3707110",
          type: "title",
          authors:
            "O Donoghue AJ, Eroy-Reveles AA, Knudsen GM, Ingram J, Zhou M, Statnekov JB, Greninger AL, Hostetter DR, Qu G, Maltby DA, Anderson MO, Derisi JL, McKerrow JH, Burlingame AL, Craik CS",
          section: "Title",
        },
        offset: 0,
        text:
          "Global identification of peptidase specificity by multiplex substrate profiling.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 81,
        text:
          "We developed a simple and rapid multiplex substrate-profiling method to reveal the substrate specificity of any endo- or exopeptidase using liquid chromatography-tandem mass spectrometry sequencing. We generated a physicochemically diverse library of peptides by incorporating all combinations of neighbor and near-neighbor amino acid pairs into decapeptide sequences that are flanked by unique dipeptides at each terminus. Addition of a panel of evolutionarily diverse peptidases to a mixture of these tetradecapeptides generated information on prime and nonprime sites as well as on substrate specificity that matched or expanded upon known substrate motifs. This method biochemically confirmed the activity of the klassevirus 3C protein responsible for polypeptide processing and allowed granzyme B substrates to be ranked by enzymatic turnover efficiency using label-free quantitation of precursor-ion abundance. Additionally, the proteolytic secretions from schistosome parasitic flatworm larvae and a pancreatic cancer cell line were deconvoluted in a subtractive strategy using class-specific peptidase inhibitors.",
        sentences: [],
        annotations: [
          {
            id: "3",
            infons: { identifier: "MESH:D004151", type: "Chemical" },
            text: "dipeptides",
            locations: [{ offset: 476, length: 10 }],
          },
          {
            id: "4",
            infons: {
              identifier: "3002",
              type: "Gene",
              ncbi_homologene: "108184",
            },
            text: "granzyme B",
            locations: [{ offset: 872, length: 10 }],
          },
          {
            id: "5",
            infons: { identifier: "MESH:D010190", type: "Disease" },
            text: "pancreatic cancer",
            locations: [{ offset: 1088, length: 17 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 23023596,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: ["disease@MESH:D010190", "chemical@MESH:D004151", "gene@3002"],
    journal: "Nat. Methods",
    year: 2012,
    authors: [
      "O Donoghue AJ",
      "Eroy-Reveles AA",
      "Knudsen GM",
      "Ingram J",
      "Zhou M",
      "Statnekov JB",
      "Greninger AL",
      "Hostetter DR",
      "Qu G",
      "Maltby DA",
      "Anderson MO",
      "Derisi JL",
      "McKerrow JH",
      "Burlingame AL",
      "Craik CS",
    ],
  },
  {
    _id: "22539347|None",
    id: "22539347",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "J. Biol. Chem.; 2012 Jun 15; 287(25) 21152-63. doi:10.1074/jbc.M112.347922",
          year: "2012",
          "article-id_pmc": "PMC3375538",
          type: "title",
          authors:
            "Sojka D, Franta Z, Frantov\u00e1 H, Bartosov\u00e1 P, Horn M, V\u00e1chov\u00e1 J, O Donoghue AJ, Eroy-Reveles AA, Craik CS, Knudsen GM, Caffrey CR, McKerrow JH, Mares M, Kop\u00e1cek P",
          section: "Title",
        },
        offset: 0,
        text:
          "Characterization of gut-associated cathepsin D hemoglobinase from tick Ixodes ricinus (IrCD1).",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "34613", type: "Species" },
            text: "Ixodes ricinus",
            locations: [{ offset: 71, length: 14 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 95,
        text:
          "To identify the gut-associated tick aspartic hemoglobinase, this work focuses on the functional diversity of multiple Ixodes ricinus cathepsin D forms (IrCDs). Out of three encoding genes representing Ixodes scapularis genome paralogs, IrCD1 is the most distinct enzyme with a shortened propeptide region and a unique pattern of predicted post-translational modifications. IrCD1 gene transcription is induced by tick feeding and is restricted to the gut tissue. The hemoglobinolytic role of IrCD1 was further supported by immunolocalization of IrCD1 in the vesicles of tick gut cells. Properties of recombinantly expressed rIrCD1 are consistent with the endo-lysosomal environment because the zymogen is autoactivated and remains optimally active in acidic conditions. Hemoglobin cleavage pattern of rIrCD1 is identical to that produced by the native enzyme. The preference for hydrophobic residues at the P1 and P1' position was confirmed by screening a novel synthetic tetradecapeptidyl substrate library. Outside the S1-S1' regions, rIrCD1 tolerates most amino acids but displays a preference for tyrosine at P3 and alanine at P2'. Further analysis of the cleavage site location within the peptide substrate indicated that IrCD1 is a true endopeptidase. The role in hemoglobinolysis was verified with RNAi knockdown of IrCD1 that decreased gut extract cathepsin D activity by &gt;90%. IrCD1 was newly characterized as a unique hemoglobinolytic cathepsin D contributing to the complex intestinal proteolytic network of mainly cysteine peptidases in ticks.",
        sentences: [],
        annotations: [
          {
            id: "7",
            infons: { identifier: "34613", type: "Species" },
            text: "Ixodes ricinus",
            locations: [{ offset: 213, length: 14 }],
          },
          {
            id: "8",
            infons: { identifier: "6945", type: "Species" },
            text: "Ixodes scapularis",
            locations: [{ offset: 296, length: 17 }],
          },
          {
            id: "9",
            infons: { identifier: "-", type: "Chemical" },
            text: "propeptide",
            locations: [{ offset: 382, length: 10 }],
          },
          {
            id: "10",
            infons: { identifier: "MESH:D014443", type: "Chemical" },
            text: "tyrosine",
            locations: [{ offset: 1195, length: 8 }],
          },
          {
            id: "11",
            infons: { identifier: "MESH:D000409", type: "Chemical" },
            text: "alanine",
            locations: [{ offset: 1214, length: 7 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 22539347,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "species@34613",
      "chemical@MESH:D014443",
      "chemical@MESH:D000409",
      "chemical@-",
      "species@6945",
    ],
    journal: "J. Biol. Chem.",
    year: 2012,
    authors: [
      "Sojka D",
      "Franta Z",
      "Frantov\u00e1 H",
      "Bartosov\u00e1 P",
      "Horn M",
      "V\u00e1chov\u00e1 J",
      "O Donoghue AJ",
      "Eroy-Reveles AA",
      "Craik CS",
      "Knudsen GM",
      "Caffrey CR",
      "McKerrow JH",
      "Mares M",
      "Kop\u00e1cek P",
    ],
  },
  {
    _id: "22190034|None",
    id: "22190034",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Nature; 2011 Dec 21; 481(7381) 365-70. doi:10.1038/nature10719",
          year: "2011",
          "article-id_pmc": "PMC3310911",
          type: "title",
          authors:
            "J\u00e4ger S, Cimermancic P, Gulbahce N, Johnson JR, McGovern KE, Clarke SC, Shales M, Mercenne G, Pache L, Li K, Hernandez H, Jang GM, Roth SL, Akiva E, Marlett J, Stephens M, D Orso I, Fernandes J, Fahey M, Mahon C, O Donoghue AJ, Todorovic A, Morris JH, Maltby DA, Alber T, Cagney G, Bushman FD, Young JA, Chanda SK, Sundquist WI, Kortemme T, Hernandez RD, Craik CS, Burlingame A, Sali A, Frankel AD, Krogan NJ",
          section: "Title",
        },
        offset: 0,
        text: "Global landscape of HIV-human protein complexes.",
        sentences: [],
        annotations: [
          {
            id: "2",
            infons: { identifier: "12721", type: "Species" },
            text: "HIV",
            locations: [{ offset: 20, length: 3 }],
          },
          {
            id: "3",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 24, length: 5 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 49,
        text:
          "Human immunodeficiency virus (HIV) has a small genome and therefore relies heavily on the host cellular machinery to replicate. Identifying which host proteins and complexes come into physical contact with the viral proteins is crucial for a comprehensive understanding of how HIV rewires the host's cellular machinery during the course of infection. Here we report the use of affinity tagging and purification mass spectrometry to determine systematically the physical interactions of all 18 HIV-1 proteins and polyproteins with host proteins in two different human cell lines (HEK293 and Jurkat). Using a quantitative scoring system that we call MiST, we identified with high confidence 497 HIV-human protein-protein interactions involving 435 individual human proteins, with ~40% of the interactions being identified in both cell types. We found that the host proteins hijacked by HIV, especially those found interacting in both cell types, are highly conserved across primates. We uncovered a number of host complexes targeted by viral proteins, including the finding that HIV protease cleaves eIF3d, a subunit of eukaryotic translation initiation factor 3. This host protein is one of eleven identified in this analysis that act to inhibit HIV replication. This data set facilitates a more comprehensive and detailed understanding of how the host machinery is manipulated during the course of HIV infection.",
        sentences: [],
        annotations: [
          {
            id: "17",
            infons: { identifier: "12721", type: "Species" },
            text: "Human immunodeficiency virus",
            locations: [{ offset: 49, length: 28 }],
          },
          {
            id: "18",
            infons: { identifier: "12721", type: "Species" },
            text: "HIV",
            locations: [{ offset: 79, length: 3 }],
          },
          {
            id: "19",
            infons: { identifier: "12721", type: "Species" },
            text: "HIV",
            locations: [{ offset: 326, length: 3 }],
          },
          {
            id: "20",
            infons: { identifier: "MESH:D007239", type: "Disease" },
            text: "infection",
            locations: [{ offset: 389, length: 9 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D015658", type: "Disease" },
            text: "HIV-1 proteins",
            locations: [{ offset: 542, length: 14 }],
          },
          {
            id: "22",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 610, length: 5 }],
          },
          {
            id: "23",
            infons: { identifier: "12721", type: "Species" },
            text: "HIV",
            locations: [{ offset: 742, length: 3 }],
          },
          {
            id: "24",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 746, length: 5 }],
          },
          {
            id: "25",
            infons: { identifier: "9606", type: "Species" },
            text: "human",
            locations: [{ offset: 806, length: 5 }],
          },
          {
            id: "26",
            infons: { identifier: "12721", type: "Species" },
            text: "HIV",
            locations: [{ offset: 933, length: 3 }],
          },
          {
            id: "27",
            infons: { identifier: "12721", type: "Species" },
            text: "HIV",
            locations: [{ offset: 1126, length: 3 }],
          },
          {
            id: "28",
            infons: { identifier: "12721", type: "Species" },
            text: "HIV",
            locations: [{ offset: 1294, length: 3 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D015658", type: "Disease" },
            text: "HIV infection",
            locations: [{ offset: 1447, length: 13 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 22190034,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D007239",
      "disease@MESH:D015658",
      "species@9606",
      "species@12721",
    ],
    journal: "Nature",
    year: 2011,
    authors: [
      "J\u00e4ger S",
      "Cimermancic P",
      "Gulbahce N",
      "Johnson JR",
      "McGovern KE",
      "Clarke SC",
      "Shales M",
      "Mercenne G",
      "Pache L",
      "Li K",
      "Hernandez H",
      "Jang GM",
      "Roth SL",
      "Akiva E",
      "Marlett J",
      "Stephens M",
      "D Orso I",
      "Fernandes J",
      "Fahey M",
      "Mahon C",
      "O Donoghue AJ",
      "Todorovic A",
      "Morris JH",
      "Maltby DA",
      "Alber T",
      "Cagney G",
      "Bushman FD",
      "Young JA",
      "Chanda SK",
      "Sundquist WI",
      "Kortemme T",
      "Hernandez RD",
      "Craik CS",
      "Burlingame A",
      "Sali A",
      "Frankel AD",
      "Krogan NJ",
    ],
  },
  {
    _id: "19556294|None",
    id: "19556294",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "Microbiology(Reading, Engl.); 2009 Nov; 155(Pt 11) 3673-82. doi:10.1099/mic.0.030940-0",
          year: "2009",
          "article-id_pmc": "PMC2888130",
          type: "title",
          authors:
            "Mahon CS, O Donoghue AJ, Goetz DH, Murray PG, Craik CS, Tuohy MG",
          section: "Title",
        },
        offset: 0,
        text:
          "Characterization of a multimeric, eukaryotic prolyl aminopeptidase: an inducible and highly specific intracellular peptidase from the non-pathogenic fungus Talaromyces emersonii.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "68825", type: "Species" },
            text: "Talaromyces emersonii",
            locations: [{ offset: 156, length: 21 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 179,
        text:
          "Fungi are capable of degrading proteins in their environment by secreting peptidases. However, the link between extracellular digestion and intracellular proteolysis has scarcely been investigated. Mycelial lysates of the filamentous fungus Talaromyces emersonii were screened for intracellular peptidase production. Five distinct proteolytic activities with specificity for the p-nitroanilide (pNA) peptides Suc-AAPF-pNA, Suc-AAA-pNA, K-pNA, F-pNA and P-pNA were identified. The native enzyme responsible for the removal of N-terminal proline residues was purified to homogeneity by ammonium sulfate fractionation followed by five successive chromatographic steps. The enzyme, termed Talaromyces emersonii prolyl aminopeptidase (TePAP), displayed a 50-fold specificity for cleaving N-terminal Pro-X (k(cat)/K(m)=2.1 x 10(6) M(-1) s(-1)) compared with Ala-X or Val-X bonds. This intracellular aminopeptidase was optimally active at pH 7.4 and 50 degrees C. Peptide sequencing facilitated the design of degenerate oligonucleotides from homologous sequences encoding putative fungal proline aminopeptidases, enabling subsequent cloning of the gene. TePAP was shown to be relatively uninhibited by classical serine peptidase inhibitors and to be sensitive to selected cysteine- and histidine-modifying reagents, yet gene sequence analysis identified the protein as a serine peptidase with an alpha/beta hydrolase fold. Northern analysis indicated that Tepap mRNA levels were regulated by the composition of the growth medium. Highest Tepap transcript levels were observed when the fungus was grown in medium containing glucose and the protein hydrolysate casitone. Interestingly, both the induction profile and substrate preference of this enzyme suggest potential co-operativity between extracellular and intracellular proteolysis in this organism. Gel filtration chromatography suggested that the enzyme exists as a 270 kDa homo-hexamer, whereas most bacterial prolyl aminopeptidases (PAPs) are monomers. Phylogenetic analysis of known PAPs revealed two diverse subfamilies that are distinguishable on the basis of primary and secondary structure and appear to correlate with the subunit composition of the native enzymes. Sequence comparisons revealed that PAPs with key conserved topological features are widespread in bacterial and fungal kingdoms, and this study identified many putative PAP candidates within sequenced genomes. This work represents, to our knowledge, the first detailed biochemical and molecular analysis of an inducible PAP from a eukaryote and the first intracellular peptidase isolated from the thermophilic fungus T. emersonii.",
        sentences: [],
        annotations: [
          {
            id: "17",
            infons: { identifier: "68825", type: "Species" },
            text: "Talaromyces emersonii",
            locations: [{ offset: 420, length: 21 }],
          },
          {
            id: "18",
            infons: { identifier: "-", type: "Chemical" },
            text: "p-nitroanilide",
            locations: [{ offset: 558, length: 14 }],
          },
          {
            id: "19",
            infons: { identifier: "-", type: "Chemical" },
            text: "Suc-AAPF-pNA",
            locations: [{ offset: 588, length: 12 }],
          },
          {
            id: "20",
            infons: { identifier: "-", type: "Chemical" },
            text: "Suc-AAA-pNA",
            locations: [{ offset: 602, length: 11 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D011392", type: "Chemical" },
            text: "proline",
            locations: [{ offset: 715, length: 7 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D000645", type: "Chemical" },
            text: "ammonium sulfate",
            locations: [{ offset: 763, length: 16 }],
          },
          {
            id: "23",
            infons: { identifier: "68825", type: "Species" },
            text: "Talaromyces emersonii",
            locations: [{ offset: 864, length: 21 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D000409", type: "Chemical" },
            text: "Ala",
            locations: [{ offset: 1031, length: 3 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:D000082", type: "Chemical" },
            text: "X",
            locations: [{ offset: 1035, length: 1 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D014633", type: "Chemical" },
            text: "Val",
            locations: [{ offset: 1040, length: 3 }],
          },
          {
            id: "27",
            infons: { identifier: "MESH:D009841", type: "Chemical" },
            text: "oligonucleotides",
            locations: [{ offset: 1192, length: 16 }],
          },
          {
            id: "28",
            infons: { identifier: "MESH:D003545", type: "Chemical" },
            text: "cysteine",
            locations: [{ offset: 1444, length: 8 }],
          },
          {
            id: "29",
            infons: { identifier: "MESH:D006639", type: "Chemical" },
            text: "histidine",
            locations: [{ offset: 1458, length: 9 }],
          },
          {
            id: "30",
            infons: { identifier: "MESH:D005947", type: "Chemical" },
            text: "glucose",
            locations: [{ offset: 1795, length: 7 }],
          },
          {
            id: "31",
            infons: { identifier: "68825", type: "Species" },
            text: "T. emersonii",
            locations: [{ offset: 2818, length: 12 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 19556294,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:D000409",
      "species@68825",
      "chemical@-",
      "chemical@MESH:D014633",
      "chemical@MESH:D003545",
      "chemical@MESH:D006639",
      "chemical@MESH:D011392",
      "chemical@MESH:D009841",
      "chemical@MESH:D000082",
      "chemical@MESH:D000645",
      "chemical@MESH:D005947",
    ],
    journal: "Microbiology(Reading, Engl.)",
    year: 2009,
    authors: [
      "Mahon CS",
      "O Donoghue AJ",
      "Goetz DH",
      "Murray PG",
      "Craik CS",
      "Tuohy MG",
    ],
  },
  {
    _id: "18701454|None",
    id: "18701454",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "J. Biol. Chem.; 2008 Oct 24; 283(43) 28934-43. doi:10.1074/jbc.M805824200",
          year: "2008",
          "article-id_pmc": "PMC2570886",
          type: "title",
          authors:
            "O Brien TC, Mackey ZB, Fetter RD, Choe Y, O Donoghue AJ, Zhou M, Craik CS, Caffrey CR, McKerrow JH",
          section: "Title",
        },
        offset: 0,
        text:
          "A parasite cysteine protease is key to host protein degradation and iron acquisition.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "MESH:D007501", type: "Chemical" },
            text: "iron",
            locations: [{ offset: 68, length: 4 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 86,
        text:
          "Cysteine proteases of the Clan CA (papain) family are the predominant protease group in primitive invertebrates. Cysteine protease inhibitors arrest infection by the protozoan parasite, Trypanosoma brucei. RNA interference studies implicated a cathepsin B-like protease, tbcatB, as a key inhibitor target. Utilizing parasites in which one of the two alleles of tbcatb has been deleted, the key role of this protease in degradation of endocytosed host proteins is delineated. TbcatB deficiency results in a decreased growth rate and dysmorphism of the flagellar pocket and the subjacent endocytic compartment. Western blot and microscopic analysis indicate that deficiency in tbcatB results in accumulation of both host and parasite proteins, including the lysosomal marker p67. A critical function for parasitism is the degradation of host transferrin, which is necessary for iron acquisition. Substrate specificity analysis of recombinant tbcatB revealed the optimal peptide cleavage sequences for the enzyme and these were confirmed experimentally using FRET-based substrates. Degradation of transferrin was validated by SDS-PAGE and the specific cleavage sites identified by N-terminal sequencing. Because even a modest deficiency in tbcatB is lethal for the parasite, tbcatB is a logical target for the development of new anti-trypanosomal chemotherapy.",
        sentences: [],
        annotations: [
          {
            id: "11",
            infons: { identifier: "MESH:D006323", type: "Disease" },
            text: "arrest infection",
            locations: [{ offset: 228, length: 16 }],
          },
          {
            id: "12",
            infons: { identifier: "5691", type: "Species" },
            text: "Trypanosoma brucei",
            locations: [{ offset: 272, length: 18 }],
          },
          {
            id: "13",
            infons: { identifier: "MESH:D007153", type: "Disease" },
            text: "TbcatB deficiency",
            locations: [{ offset: 561, length: 17 }],
          },
          {
            id: "14",
            infons: { type: "Disease", identifier: null, db: null },
            text: "dysmorphism of the flagellar",
            locations: [{ offset: 618, length: 28 }],
          },
          {
            id: "15",
            infons: { identifier: "MESH:D010272", type: "Disease" },
            text: "parasitism",
            locations: [{ offset: 888, length: 10 }],
          },
          {
            id: "16",
            infons: { identifier: "MESH:D007501", type: "Chemical" },
            text: "iron",
            locations: [{ offset: 962, length: 4 }],
          },
          {
            id: "17",
            infons: { identifier: "MESH:D012967", type: "Chemical" },
            text: "SDS",
            locations: [{ offset: 1209, length: 3 }],
          },
          {
            id: "18",
            infons: { identifier: "-", type: "Chemical" },
            text: "tbcatB",
            locations: [{ offset: 1323, length: 6 }],
          },
          {
            id: "19",
            infons: { identifier: "-", type: "Chemical" },
            text: "tbcatB",
            locations: [{ offset: 1358, length: 6 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 18701454,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "disease@MESH:D006323",
      "chemical@MESH:D012967",
      "disease@MESH:D007153",
      "chemical@-",
      "species@5691",
      "disease@MESH:D010272",
      "chemical@MESH:D007501",
    ],
    journal: "J. Biol. Chem.",
    year: 2008,
    authors: [
      "O Brien TC",
      "Mackey ZB",
      "Fetter RD",
      "Choe Y",
      "O Donoghue AJ",
      "Zhou M",
      "Craik CS",
      "Caffrey CR",
      "McKerrow JH",
    ],
  },
  {
    _id: "18687686|None",
    id: "18687686",
    infons: {},
    passages: [
      {
        infons: {
          journal:
            "J. Biol. Chem.; 2008 Oct 24; 283(43) 29186-95. doi:10.1074/jbc.M802366200",
          year: "2008",
          "article-id_pmc": "PMC2570871",
          type: "title",
          authors:
            "O Donoghue AJ, Mahon CS, Goetz DH, O Malley JM, Gallagher DM, Zhou M, Murray PG, Craik CS, Tuohy MG",
          section: "Title",
        },
        offset: 0,
        text:
          "Inhibition of a secreted glutamic peptidase prevents growth of the fungus Talaromyces emersonii.",
        sentences: [],
        annotations: [
          {
            id: "1",
            infons: { identifier: "68825", type: "Species" },
            text: "Talaromyces emersonii",
            locations: [{ offset: 74, length: 21 }],
          },
        ],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 97,
        text:
          "The thermophilic filamentous fungus Talaromyces emersonii secretes a variety of hydrolytic enzymes that are of interest for processing of biomass into fuel. Many carbohydrases have been isolated and characterized from this fungus, but no studies had been performed on peptidases. In this study, two acid-acting endopeptidases were isolated and characterized from the culture filtrate of T. emersonii. One of these enzymes was identified as a member of the recently classified glutamic peptidase family and was subsequently named T. emersonii glutamic peptidase 1 (TGP1). The second enzyme was identified as an aspartyl peptidase (PEP1). TGP1 was cloned and sequenced and shown to exhibit 64 and 47% protein identity to peptidases from Aspergillus niger and Scytalidium lignocolum, respectively. Substrate profiling of 16 peptides determined that TGP1 has broad specificity with a preference for large residues in the P1 site, particularly Met, Gln, Phe, Lys, Glu, and small amino acids at P1' such as Ala, Gly, Ser, or Thr. This enzyme efficiently cleaves an internally quenched fluorescent substrate containing the zymogen activation sequence (k(cat)/K(m)=2 x 10(5) m(-1) s(-1)). Maximum hydrolysis occurs at pH 3.4 and 50 degrees C. The reaction is strongly inhibited by a transition state peptide analog, TA1 (K(i)=1.5 nM), as well as a portion of the propeptide sequence, PT1 (K(i)=32 nM). Ex vivo studies show that hyphal extension of T. emersonii in complex media is unaffected by the aspartyl peptidase inhibitor pepstatin but is inhibited by TA1 and PT1. This study provides insight into the functional role of the glutamic peptidase TGP1 for growth of T. emersonii.",
        sentences: [],
        annotations: [
          {
            id: "17",
            infons: { identifier: "68825", type: "Species" },
            text: "Talaromyces emersonii",
            locations: [{ offset: 133, length: 21 }],
          },
          {
            id: "18",
            infons: { identifier: "68825", type: "Species" },
            text: "T. emersonii",
            locations: [{ offset: 484, length: 12 }],
          },
          {
            id: "19",
            infons: { identifier: "68825", type: "Species" },
            text: "T. emersonii",
            locations: [{ offset: 626, length: 12 }],
          },
          {
            id: "20",
            infons: { identifier: "5061", type: "Species" },
            text: "Aspergillus niger",
            locations: [{ offset: 832, length: 17 }],
          },
          {
            id: "21",
            infons: { identifier: "MESH:D005973", type: "Chemical" },
            text: "Gln",
            locations: [{ offset: 1041, length: 3 }],
          },
          {
            id: "22",
            infons: { identifier: "MESH:D010649", type: "Chemical" },
            text: "Phe",
            locations: [{ offset: 1046, length: 3 }],
          },
          {
            id: "23",
            infons: { identifier: "MESH:D008239", type: "Chemical" },
            text: "Lys",
            locations: [{ offset: 1051, length: 3 }],
          },
          {
            id: "24",
            infons: { identifier: "MESH:D018698", type: "Chemical" },
            text: "Glu",
            locations: [{ offset: 1056, length: 3 }],
          },
          {
            id: "25",
            infons: { identifier: "MESH:D000409", type: "Chemical" },
            text: "Ala",
            locations: [{ offset: 1098, length: 3 }],
          },
          {
            id: "26",
            infons: { identifier: "MESH:D005998", type: "Chemical" },
            text: "Gly",
            locations: [{ offset: 1103, length: 3 }],
          },
          {
            id: "27",
            infons: { identifier: "MESH:D012694", type: "Chemical" },
            text: "Ser",
            locations: [{ offset: 1108, length: 3 }],
          },
          {
            id: "28",
            infons: { identifier: "MESH:D013912", type: "Chemical" },
            text: "Thr",
            locations: [{ offset: 1116, length: 3 }],
          },
          {
            id: "29",
            infons: { identifier: "-", type: "Chemical" },
            text: "propeptide",
            locations: [{ offset: 1452, length: 10 }],
          },
          {
            id: "30",
            infons: { identifier: "68825", type: "Species" },
            text: "T. emersonii",
            locations: [{ offset: 1537, length: 12 }],
          },
          {
            id: "31",
            infons: { identifier: "68825", type: "Species" },
            text: "T. emersonii",
            locations: [{ offset: 1758, length: 12 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 18687686,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: [
      "chemical@MESH:D000409",
      "chemical@MESH:D005973",
      "species@68825",
      "chemical@MESH:D012694",
      "chemical@-",
      "chemical@MESH:D013912",
      "chemical@MESH:D005998",
      "species@5061",
      "chemical@MESH:D010649",
      "chemical@MESH:D018698",
      "chemical@MESH:D008239",
    ],
    journal: "J. Biol. Chem.",
    year: 2008,
    authors: [
      "O Donoghue AJ",
      "Mahon CS",
      "Goetz DH",
      "O Malley JM",
      "Gallagher DM",
      "Zhou M",
      "Murray PG",
      "Craik CS",
      "Tuohy MG",
    ],
  },
  {
    _id: "16948499|None",
    id: "16948499",
    infons: {},
    passages: [
      {
        infons: {
          journal: "In Vitro Cell. Dev. Biol. Anim.; 2006 Jul-Aug; 42(7) 182-8",
          year: "2006",
          "article-id_pmc": "PMC2883882",
          type: "title",
          authors: "Cuetara BL, Crotti TN, O Donoghue AJ, McHugh KP",
          section: "Title",
        },
        offset: 0,
        text:
          "Cloning and characterization of osteoclast precursors from the RAW264.7 cell line.",
        sentences: [],
        annotations: [],
        relations: [],
      },
      {
        infons: { type: "abstract", section: "Abstract" },
        offset: 83,
        text:
          "Osteoclasts are bone-resorbing cells that differentiate from macrophage precursors in response to receptor activator of NF-kappaB ligand (RANKL). In vitro models of osteoclast differentiation are principally based on primary cell cultures, which are poorly suited to molecular and transgene studies because of the limitations associated with the use of primary macrophage. RAW264.7 is a transfectable macrophage cell line with the capacity to form osteoclast-like cells. In the present study, we have identified osteoclast precursors among clones of RAW264.7 cells. RAW264.7 cell were cloned by limiting dilution and induced to osteoclast differentiation by treatment with recombinant RANKL. Individual RAW264.7 cell clones formed tartrate resistant acid phosphatase (TRAP)-positive multinuclear cells to various degrees with RANKL treatment. All clones tested expressed the RANKL receptor RANK. Each of the clones expressed the osteoclast marker genes TRAP and cathepsin-K mRNA with RANKL treatment. However, we noted that only select clones were able to form large, well-spread, TRAP-positive multinuclear cells. Clones capable of forming large TRAP-positive multinuclear cells also expressed beta3 integrin and calcitonin receptor mRNAs and were capable of resorbing a mineralized matrix. All clones tested activated NF-kappaB with RANKL treatment. cDNA expression profiling of osteoclast precursor RAW264.7 cell clones demonstrates appropriate expression of a large number of genes before and after osteoclastic differentiation. These osteoclast precursor RAW264.7 cell clones provide a valuable model for dissecting the cellular and molecular regulation of osteoclast differentiation and activation.",
        sentences: [],
        annotations: [
          {
            id: "13",
            infons: {
              identifier: "21943",
              type: "Gene",
              ncbi_homologene: "2744",
            },
            text: "receptor activator of NF-kappaB ligand",
            locations: [{ offset: 181, length: 38 }],
          },
          {
            id: "14",
            infons: {
              identifier: "21943",
              type: "Gene",
              ncbi_homologene: "2744",
            },
            text: "RANKL",
            locations: [{ offset: 221, length: 5 }],
          },
          {
            id: "15",
            infons: {
              identifier: "21943",
              type: "Gene",
              ncbi_homologene: "2744",
            },
            text: "RANKL",
            locations: [{ offset: 768, length: 5 }],
          },
          {
            id: "16",
            infons: {
              identifier: "11433",
              type: "Gene",
              ncbi_homologene: "137203",
            },
            text: "tartrate resistant acid phosphatase",
            locations: [{ offset: 814, length: 35 }],
          },
          {
            id: "17",
            infons: {
              identifier: "11433",
              type: "Gene",
              ncbi_homologene: "137203",
            },
            text: "TRAP",
            locations: [{ offset: 851, length: 4 }],
          },
          {
            id: "18",
            infons: {
              identifier: "21943",
              type: "Gene",
              ncbi_homologene: "2744",
            },
            text: "RANKL",
            locations: [{ offset: 909, length: 5 }],
          },
          {
            id: "19",
            infons: {
              identifier: "21943",
              type: "Gene",
              ncbi_homologene: "2744",
            },
            text: "RANKL",
            locations: [{ offset: 958, length: 5 }],
          },
          {
            id: "20",
            infons: {
              identifier: "11433",
              type: "Gene",
              ncbi_homologene: "137203",
            },
            text: "TRAP",
            locations: [{ offset: 1036, length: 4 }],
          },
          {
            id: "21",
            infons: {
              identifier: "13038",
              type: "Gene",
              ncbi_homologene: "68053",
            },
            text: "cathepsin-K",
            locations: [{ offset: 1045, length: 11 }],
          },
          {
            id: "22",
            infons: {
              identifier: "21943",
              type: "Gene",
              ncbi_homologene: "2744",
            },
            text: "RANKL",
            locations: [{ offset: 1067, length: 5 }],
          },
          {
            id: "23",
            infons: {
              identifier: "11433",
              type: "Gene",
              ncbi_homologene: "137203",
            },
            text: "TRAP",
            locations: [{ offset: 1164, length: 4 }],
          },
          {
            id: "24",
            infons: {
              identifier: "11433",
              type: "Gene",
              ncbi_homologene: "137203",
            },
            text: "TRAP",
            locations: [{ offset: 1230, length: 4 }],
          },
          {
            id: "25",
            infons: {
              identifier: "21943",
              type: "Gene",
              ncbi_homologene: "2744",
            },
            text: "RANKL",
            locations: [{ offset: 1418, length: 5 }],
          },
        ],
        relations: [],
      },
    ],
    relations: [],
    pmid: 16948499,
    pmcid: null,
    created: { $date: 1592598838791 },
    accessions: ["gene@11433", "gene@21943", "gene@13038"],
    journal: "In Vitro Cell. Dev. Biol. Anim.",
    year: 2006,
    authors: ["Cuetara BL", "Crotti TN", "O Donoghue AJ", "McHugh KP"],
  },
];
